--- runtime/cmake/modules/FindPulseAudio.cmake	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/cmake/modules/FindPulseAudio.cmake	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -30,8 +30,8 @@
 
 FIND_PATH(PULSEAUDIO_INCLUDE_DIR pulse/pulseaudio.h
    HINTS
-   ${PC_PULSEADUIO_INCLUDEDIR}
-   ${PC_PULSEADUIO_INCLUDE_DIRS}
+   ${PC_PULSEAUDIO_INCLUDEDIR}
+   ${PC_PULSEAUDIO_INCLUDE_DIRS}
    )
 
 FIND_LIBRARY(PULSEAUDIO_LIBRARY NAMES pulse libpulse 
--- runtime/knotify/kde.notifyrc	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/knotify/kde.notifyrc	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -752,7 +752,7 @@
 Comment[pt]=Ocorreu um erro sério que fez com que o programa saísse
 Comment[pt_BR]=Houve um erro sério, que fez com que o programa fosse fechado
 Comment[ro]=A avut loc o eroare gravă ce a determinat ieșirea programului
-Comment[ru]=Произошла серьезная ошибка, приведшая к завершению работы программы
+Comment[ru]=Произошла серьёзная ошибка, приведшая к завершению работы программы
 Comment[se]=Duođalaš meattáhus bissehii prográmma
 Comment[sk]=Stala sa vážna chyba, ktorá zapríčinila ukončenie programu
 Comment[sl]=Prišlo je do resne napake, zaradi katere se je program končal
@@ -1181,7 +1181,7 @@
 Comment[pt]=Ocorreu um erro muito sério, que fez pelo menos com que o programa saísse
 Comment[pt_BR]=Um erro muito sério aconteceu, que causou (no mínimo) o fechamento do programa
 Comment[ro]=A intervenit o eroare foarte gravă, cauzînd cel puțin ieșirea programului
-Comment[ru]=Произошла очень серьезная ошибка, как минимум, вызвавшая завершение работы программы
+Comment[ru]=Произошла очень серьёзная ошибка, как минимум, вызвавшая завершение работы программы
 Comment[sk]=Stala sa veľmi vážna chyba, ktorá zapríčinila minimálne ukončenie programu
 Comment[sl]=Prišlo je do zelo resne napake, zaradi katere se je program končal
 Comment[sr]=Дошло је до врло озбиљне грешке, због које је бар програм обустављен
--- runtime/desktoptheme/metadata.desktop	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/desktoptheme/metadata.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -50,7 +50,7 @@
 Comment[gu]=થીમ ઓક્સિજન શૈલીમાં બનાવાઇ છે
 Comment[he]=ערכת נושא שמשתמשת בסגנון Oxygen
 Comment[hi]=प्रसंग को ऑक्सीजन शैली में किया गया है
-Comment[hne]=प्रसंग ल ओक्सीजन सैली मं कर दे गे हे
+Comment[hne]=प्रसंग ल आक्सीजन सैली मं कर दे गे हे
 Comment[hu]=Oxygen-stílusú téma
 Comment[is]=Þema sem gert er í 'Oxygen' stíl
 Comment[it]=Tema in stile Oxygen
--- runtime/khelpcenter/kcmhelpcenter.desktop	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/khelpcenter/kcmhelpcenter.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -120,7 +120,7 @@
 Comment[gu]=મદદ કેન્દ્ર શોધ અનુક્રમણિકા રૂપરેખાંકન અને બનાવટ
 Comment[he]=שינוי הגדרות ויצירת אינדקס החיפוש של מרכז העזרה
 Comment[hi]=मदद केंद्र खोज तालिका कॉन्फ़िगरेशन तथा ज़ेनरेशन
-Comment[hne]=मदद केंद्र खोज तालिका कान्फिगरेसन अउ जेनरेसन
+Comment[hne]=मदद केंद्र खोज टेबल कान्फिगरेसन अउ जेनरेसन
 Comment[hr]=Konfiguriranje i generiranje indeksa za pretraživanje pomoći
 Comment[hsb]=Připrawjenje a stworjenje pytanskeho indeksa za pomhanišćo
 Comment[hu]=A dokumentáció keresési indexének beállításai
--- runtime/doc/kcontrol/bookmarks/index.docbook	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/doc/kcontrol/bookmarks/index.docbook	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -61,7 +61,7 @@
 <varlistentry>
 <term><guilabel>Flatten bookmarks tree</guilabel></term>
 <listitem>
-<para>Sub-folders are show within their parent by default. If you activate this option, sub-folders are displayed on their own. It looks less nice but it may help if you have a very big folder you want to spread in two columns.</para>
+<para>Sub-folders are shown within their parent by default. If you activate this option, sub-folders are displayed on their own. It looks less nice but it may help if you have a very big folder you want to spread in two columns.</para>
 </listitem>
 </varlistentry>
 
--- runtime/doc/kcontrol/kcmlaunch/index.docbook	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/doc/kcontrol/kcmlaunch/index.docbook	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -13,8 +13,8 @@
 <!-- TRANS:ROLES_OF_TRANSLATORS -->
 </authorgroup>
 
-<date>2006-06-24</date>
-<releaseinfo>3.5.2</releaseinfo>
+<date>2009-02-10</date>
+<releaseinfo>&kde; 4.2</releaseinfo>
 
 <keywordset>
 <keyword>KDE</keyword>
@@ -54,17 +54,18 @@
 replaced by a spinning hourglass to let you know something is
 happening.  You can toggle this behavior on and off with the
 <guilabel>Enable taskbar notification</guilabel> checkbox, and when it's
-enabled, you can set a time in the <guilabel>Startup indication timeout
-:</guilabel> spinbox.</para>
+enabled, you can set a time in the 
+<guilabel>Startup indication timeout</guilabel> spinbox.</para>
 
 <para>Not all applications that you start will eventually show a
 window, or an entry in the taskbar.  Some of them, for example, are
 docked into the &kde; system tray. Alternatively, it might be that you
-sent it off to a different virtual desktop, and <guilabel>Show windows
-from all desktops</guilabel> is unchecked in the &kcontrolcenter; module
-<guimenuitem>Taskbar</guimenuitem>.  Setting a timeout ensures that,
+sent it off to a different virtual desktop, and in the <guilabel>Filters</guilabel> 
+section of the <ulink url="help:/plasma/index.html#panel">
+<guilabel>Task Manager Settings</guilabel></ulink> the option <guilabel>Only show
+tasks from the current desktop</guilabel> is checked.  Setting a timeout ensures that,
 even in these cases, you can still get launch feedback, but also that
 it will go away when the job is done.</para>
 
 </sect1>
-</article>
\ No newline at end of file
+</article>
--- runtime/doc/userguide/kde-for-admins.docbook	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/doc/userguide/kde-for-admins.docbook	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -828,7 +828,7 @@
 
 <para>The &kde; startup sequence starts with the
 <filename>startkde</filename> script. In most cases this script gets called
-from the display manager (&kdm;) once the user has been authenticated. Their
+from the display manager (&kdm;) once the user has been authenticated. There
 are two very important lines in the <filename>startkde</filename>
 script:</para>
 
@@ -2029,7 +2029,7 @@
 <variablelist>
 <varlistentry>
 <term><option>action/options_configure</option></term>
-<listitem><para>The <guimenuitem>Configure</guimenuitem> option form the
+<listitem><para>The <guimenuitem>Configure</guimenuitem> option from the
 <guimenu>Settings</guimenu> menu</para></listitem>
 </varlistentry>
 <varlistentry>
--- runtime/doc/userguide/standard-menu-entries.docbook	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/doc/userguide/standard-menu-entries.docbook	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -117,8 +117,8 @@
 </shortcut><guimenuitem>Print...</guimenuitem>
 </menuchoice></term>
 <listitem><para>This menu entry displays the &kde;
-<quote>Print</quote> dialog, read to print the current
-document. Youcan find more information about the options available in
+<quote>Print</quote> dialog, ready to print the current
+document. You can find more information about the options available in
 this dialog in <xref linkend="printing-from-apps"/>.</para>
 </listitem>
 </varlistentry>
--- runtime/doc/userguide/under-the-hood.docbook	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/doc/userguide/under-the-hood.docbook	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -67,7 +67,7 @@
 (or the corresponding $<envar>KDEHOME</envar> directory). Backups are anyway
 a good idea in case of a major failure of &kde; that would
 destroy important configuration files (for example your &kmail;  settings,
-which are in in the file <filename>kmailrc</filename>).
+which are in the file <filename>kmailrc</filename>).
 (Such a major failure should not happen but it still can happen.)</para>
 </sect2>
 
--- runtime/doc/userguide/groupware-kontact.docbook	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/doc/userguide/groupware-kontact.docbook	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -497,11 +497,6 @@
 <para>Now enable &kmail; groupware functionality, and in <guilabel>Resource
 folders are subfolders of</guilabel> put the
 <guimenuitem>inbox</guimenuitem> that is subfolder of
-<quote>office_gwdata</quote>.</para>
-
-<para>Now enable &kmail; groupware functionality, and in <guilabel>Resource
-folders are subfolders of</guilabel> put the
-<guimenuitem>inbox</guimenuitem> that is subfolder of
 <guisubmenu>office_gwdata</guisubmenu>.</para>
   
 <para>Now you have two branches of folder under
--- runtime/kioslave/trash/trash.protocol	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/kioslave/trash/trash.protocol	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -49,7 +49,7 @@
 ExtraNames[gu]=મૂળભૂત પાથ,દૂર કરવાની તારીખ
 ExtraNames[he]=נתיב מקורי,תאריך מחיקה
 ExtraNames[hi]=मूल पथ, मिटाने की तिथि
-ExtraNames[hne]=मूल पथ, मेटारव के तारीक
+ExtraNames[hne]=मूल पथ, मेटाव के तारीक
 ExtraNames[hr]=Izvorna putanja,Datum brisanja
 ExtraNames[hsb]=Originalny puć, datum zničenja
 ExtraNames[hu]=Eredeti elérési út,Törlési dátum
--- runtime/kioslave/desktop/desktop.protocol	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/kioslave/desktop/desktop.protocol	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -1,6 +1,7 @@
 [Protocol]
 exec=kio_desktop
 protocol=desktop
+Icon=user-desktop
 input=none
 output=filesystem
 listing=Name,Type,Size,Date,AccessDate,Access,Owner,Group,Link
--- runtime/kioslave/sftp/sftpfileattr.cpp	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/kioslave/sftp/sftpfileattr.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -230,7 +230,7 @@
 }
 
 /** No descriptions */
-kdbgstream& operator<< (kdbgstream& s, sftpFileAttr& a) {
+QDebug operator<<(QDebug s, sftpFileAttr& a) {
     s << "Filename: " << a.mFilename
       << ", Uid: " << a.mUid
       << ", Gid: " << a.mGid
@@ -250,11 +250,6 @@
     return s;
 }
 
-/** Make sure it builds with NDEBUG */
-kndbgstream& operator<< (kndbgstream& s, sftpFileAttr& ) {
-    return s;
-}
-
 /** Clear all attributes and flags. */
 void sftpFileAttr::clear(){
     clearAtime();
--- runtime/kioslave/sftp/ksshprocess.cpp	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/kioslave/sftp/ksshprocess.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -232,13 +232,12 @@
 #else
     mSshPath = KStandardDirs::findExe(QString::fromLatin1("plink"));
 #endif
-    kDebug(KSSHPROC) << "KSshProcess::KSshProcess(): ssh path [" << 
-		mSshPath << "]" << endl;
+    kDebug(KSSHPROC) << "ssh path [" <<  mSshPath << "]";
         
 	installSignalHandlers();
 }
 
-KSshProcess::KSshProcess(QString pathToSsh)
+KSshProcess::KSshProcess(const QString& pathToSsh)
             : mSshPath(pathToSsh), mVersion(UNKNOWN_VER), mConnected(false),
     mRunning(false), mConnectState(0)  {
 	installSignalHandlers();
@@ -252,7 +251,7 @@
     }
 }
 
-bool KSshProcess::setSshPath(QString pathToSsh) {
+bool KSshProcess::setSshPath(const QString& pathToSsh) {
     mSshPath = pathToSsh;
     version();
     if( mVersion == UNKNOWN_VER )
@@ -268,8 +267,7 @@
     // Get version string from ssh client.
     FILE *p;
     if( (p = popen(cmd.toLatin1(), "r")) == NULL ) {
-        kDebug(KSSHPROC) << "KSshProcess::version(): "
-            "failed to start ssh: " << strerror(errno) << endl;
+        kDebug(KSSHPROC) << "failed to start ssh: " << strerror(errno);
         return UNKNOWN_VER;
     }
 
@@ -277,20 +275,17 @@
     size_t len;
     char buf[128];
     if( (len = fread(buf, sizeof(char), sizeof(buf)-1, p)) == 0 ) {
-        kDebug(KSSHPROC) << "KSshProcess::version(): "
-            "Read of ssh version string failed " << 
-             strerror(ferror(p)) << endl;
+        kDebug(KSSHPROC) << "read of ssh version string failed " << strerror(ferror(p));
         return UNKNOWN_VER;
     }
     if( pclose(p) == -1 ) {
-        kError(KSSHPROC) << "KSshProcess::version(): pclose failed." << endl;
+        kError(KSSHPROC) << "pclose failed.";
     }
     buf[len] = '\0';
-    QString ver;
-    ver = buf;
-    kDebug(KSSHPROC) << "KSshProcess::version(): "
-        "got version string [" << ver << "]" << endl;
 
+    QString ver = QString::fromLocal8Bit(buf).simplified();
+    kDebug(KSSHPROC) << "got version string [" << ver << "]";
+
     mVersion = UNKNOWN_VER;
     for(int i = 0; i < SSH_VER_MAX; i++) {
         if( ver.indexOf(versionStrs[i]) != -1 ) {
@@ -299,12 +294,10 @@
         }
     }
 
-    kDebug(KSSHPROC) << "KSshPRocess::version(): version number = "
-	    << mVersion << endl;
+    kDebug(KSSHPROC) << "version number = " << mVersion;
     
     if( mVersion == UNKNOWN_VER ) {
-        kDebug(KSSHPROC) << "KSshProcess::version(): "
-            "Sorry, I don't know about this version of ssh" << endl;
+        kDebug(KSSHPROC) << "sorry, I don't know about this version of ssh";
         mError = ERR_UNKNOWN_VERSION;
         return UNKNOWN_VER;
     }
@@ -324,8 +317,6 @@
 */
 
 bool KSshProcess::setOptions(const SshOptList& opts) {
-    kDebug(KSSHPROC) << "KSshProcess::setOptions()";
-
 	if( mVersion == UNKNOWN_VER ) {
         // we don't know the ssh version yet, so find out
         version();
@@ -448,14 +439,12 @@
             break;
 
         default:
-            kDebug(KSSHPROC) << "KSshProcess::setOptions(): "
-                "unrecognized ssh opt " << (*it).opt << endl;
+            kDebug(KSSHPROC) << "unrecognized ssh opt " << (*it).opt;
         }
     }
 
     if( !subsystem.isEmpty() && !cmd.isEmpty() ) {
-        kDebug(KSSHPROC) << "KSshProcess::setOptions(): "
-            "cannot use a subsystem and command at the same time" << endl;
+        kDebug(KSSHPROC) << "cannot use a subsystem and command at the same time";
         mError = ERR_CMD_SUBSYS_CONFLICT;
         mErrorMsg = i18n("Cannot specify a subsystem and command at the same time.");
         return false;
@@ -476,8 +465,7 @@
     }
 
     if( mHost.isEmpty() ) {
-        kDebug(KSSHPROC) << "KSshProcess::setOptions(): "
-            "a host name must be supplied" << endl;
+        kDebug(KSSHPROC) << "a host name must be supplied";
         return false;
     }
     else {
@@ -497,15 +485,12 @@
 }
 
 void KSshProcess::printArgs() {
-    QList<QByteArray>::Iterator it;
-    for( it = mArgs.begin(); it != mArgs.end(); ++it) {
-        kDebug(KSSHPROC) << "arg: " << *it;
-    }
+    foreach(const QByteArray& arg, mArgs)
+        kDebug(KSSHPROC) << "arg: " << arg;
 }
 
 
 int KSshProcess::error(QString& msg) {
-    kDebug(KSSHPROC) << "KSshProcess::error()";
     kDebug() << mErrorMsg;
     msg = mErrorMsg;
     return mError;
@@ -514,31 +499,27 @@
 void KSshProcess::kill(int signal) {
     int pid = ssh.pid();
     
-    kDebug(KSSHPROC) << "KSshProcess::kill(signal:" << signal 
-                      << "): ssh pid is " << pid << endl;
-    kDebug(KSSHPROC) << "KSshPRocess::kill(): we are " 
-                      << (mConnected ? "" : "not ") << "connected" << endl;
-    kDebug(KSSHPROC) << "KSshProcess::kill(): we are " 
-                      << (mRunning ? "" : "not ") << "running a ssh process" << endl;
+    kDebug(KSSHPROC) << "signal:" << signal << "): ssh pid is " << pid;
+    kDebug(KSSHPROC) << "we are " << (mConnected ? "" : "not ") << "connected";
+    kDebug(KSSHPROC) << "we are " << (mRunning ? "" : "not ") << "running a ssh process";
 
     if( mRunning && pid > 1 ) {
-            // Kill the child process...
-            if ( ::kill(pid, signal) == 0 ) {
-                // clean up if we tried to kill the process
-                if( signal == SIGTERM || signal == SIGKILL ) {
-                    while(waitpid(-1, NULL, WNOHANG) > 0) {
-                        ;
-                    }
-                    mConnected = false;
-                    mRunning = false;
+        // Kill the child process...
+        if ( ::kill(pid, signal) == 0 ) {
+            // clean up if we tried to kill the process
+            if( signal == SIGTERM || signal == SIGKILL ) {
+                while(waitpid(-1, NULL, WNOHANG) > 0) {
+                    ;
                 }
+                mConnected = false;
+                mRunning = false;
             }
-            else
-                kDebug(KSSHPROC) << "KSshProcess::kill(): kill failed";
+        }
+        else
+            kDebug(KSSHPROC) << "kill failed";
     }
     else
-        kDebug(KSSHPROC) << "KSshProcess::kill(): "
-            "Refusing to kill ssh process" << endl;
+        kDebug(KSSHPROC) << "refusing to kill ssh process";
 }
 
 
@@ -588,13 +569,12 @@
 
 
 void KSshProcess::acceptHostKey(bool accept) {
-    kDebug(KSSHPROC) << "KSshProcess::acceptHostKey(accept:"
-        << accept << ")" << endl;
+    kDebug(KSSHPROC) << "accept host key ? " << accept;
     mAcceptHostKey = accept;
 }
 
-void KSshProcess::setPassword(QString password) {
-    kDebug(KSSHPROC) << "KSshProcess::setPassword(password:xxxxxxxx)";
+void KSshProcess::setPassword(const QString& password) {
+    kDebug(KSSHPROC) << "setPassword(password:xxxxxxxx)";
     mPassword = password;
 }
 
@@ -627,8 +607,8 @@
         // not be anything on the pty or stderr. Setup a select()
         // to wait for some data from SSH.
         if( buffer.empty() ) {
-            //kDebug(KSSHPROC) << "KSshProcess::getLine(): " <<
-            //    "Line buffer empty, calling select() to wait for data." << endl;
+            //kDebug(KSSHPROC) << "getLine(): " <<
+            //    "Line buffer empty, calling select() to wait for data.";
             int errfd = ssh.stderrFd();
             int ptyfd = ssh.fd();
             fd_set rfds;
@@ -656,14 +636,12 @@
     
             // Handle any errors from select
             if( ret == 0 ) {
-                kDebug(KSSHPROC) << "KSshProcess::connect(): " <<
-                    "timed out waiting for a response" << endl;
+                kDebug(KSSHPROC) << "timed out waiting for a response";
                 mError = ERR_TIMED_OUT;
                 return QString();
             }
             else if( ret == -1 ) {
-                kDebug(KSSHPROC) << "KSshProcess::connect(): "
-                    << "select error: " << strerror(errno) << endl;
+                kDebug(KSSHPROC) << "select error: " << strerror(errno);
                 mError = ERR_INTERNAL;
                 return QString();
             }
@@ -674,25 +652,21 @@
             if( FD_ISSET(ptyfd, &rfds) ) {
                 ptyLine = ssh.readLineFromPty(false);
                 buffer.prepend(QString(ptyLine));
-                //kDebug(KSSHPROC) << "KSshProcess::getLine(): "
-                //    "line from pty -" << ptyLine  << endl;
+                //kDebug(KSSHPROC) << "line from pty -" << ptyLine;
             }
             
             if( FD_ISSET(errfd, &rfds) ) {
                 errLine = ssh.readLineFromStderr(false);
                 buffer.prepend(QString(errLine));
-                //kDebug(KSSHPROC) << "KSshProcess::getLine(): "
-                //    "line from err -" << errLine << endl;
+                //kDebug(KSSHPROC) << "line from err -" << errLine;
             }
 
             if( FD_ISSET(ptyfd, &efds) ) {
-                kDebug(KSSHPROC) << "KSshProcess::getLine(): "
-                    "Exception on pty file descriptor." << endl;
+                kDebug(KSSHPROC) << "exception on pty file descriptor.";
             }
 
             if( FD_ISSET(errfd, &efds) ) {
-                kDebug(KSSHPROC) << "KSshProcess::getLine(): "
-                    "Exception on std err file descriptor." << endl;
+                kDebug(KSSHPROC) << "exception on std err file descriptor.";
             }
             
         }
@@ -712,11 +686,8 @@
         buffer.pop_back();
     }
     
-//    kDebug(KSSHPROC) << "KSshProcess::getLine(): " << 
-//        buffer.count() << " lines in buffer" << endl;
-    kDebug(KSSHPROC) << "KSshProcess::getLine(): "
-        "ssh: " << line << endl;
-    
+//    kDebug(KSSHPROC) << buffer.count() << " lines in buffer";
+    kDebug(KSSHPROC) << line.simplified();
 
     return line;
 #endif
@@ -775,8 +746,7 @@
     int transitionLimit = 500;
 
     while(--transitionLimit) {
-        kDebug(KSSHPROC) << "KSshProcess::connect(): "
-            << "Connect state " << stateStr(mConnectState) << endl;
+        kDebug(KSSHPROC) << "connect state " << stateStr(mConnectState);
         
         QString line;      // a line from ssh
         QString msgBuf;    // buffer for important messages from ssh
@@ -795,22 +765,20 @@
             mKnownHostsFile.clear();
             
             if( mArgs.isEmpty() ) {
-                kDebug(KSSHPROC) << "KSshProcess::connect(): ssh options "
-                    "need to be set first using setArgs()" << endl;
+                kDebug(KSSHPROC) << "ssh options need to be set first using setArgs()";
                 mError = ERR_NO_OPTIONS;
                 mErrorMsg = i18n("No options provided for ssh execution.");
                 return false;
             }
 
             if( ssh.exec(mSshPath.toLatin1(), mArgs) ) {
-                kDebug(KSSHPROC) << 
-                    "KSshProcess::connect(): ssh exec failed" << endl;
+                kDebug(KSSHPROC) << "ssh exec failed";
                 mError = ERR_CANNOT_LAUNCH;
                 mErrorMsg = i18n("Failed to execute ssh process.");
                 return false;
             }
            
-            kDebug(KSSHPROC) << "KSshPRocess::connect(): ssh pid = " << ssh.pid();
+            kDebug(KSSHPROC) << "ssh pid = " << ssh.pid();
 	    
             // set flag to indicate what have started a ssh process
             mRunning = true;
@@ -827,8 +795,7 @@
         case STATE_WAIT_PROMPT:
             line = getLine();
             if( line.isNull() ) {
-                kDebug(KSSHPROC) << "KSshProcess::connect(): "
-                    "Got null line in STATE_WAIT_PROMPT." << endl;
+                kDebug(KSSHPROC) << "Got null line in STATE_WAIT_PROMPT.";
                 mError = ERR_INTERACT;
                 mErrorMsg =
                     i18n("Error encountered while talking to ssh.");
@@ -855,9 +822,7 @@
             }
             else if( line.indexOf(continuePrompt[mVersion]) != -1 ) {
                 //mConnectState = STATE_SEND_CONTINUE;
-                kDebug(KSSHPROC) << "KSshProcess:connect(): "
-                    "Got continue prompt where we shouldn't (STATE_WAIT_PROMPT)"
-                    << endl;
+                kDebug(KSSHPROC) << "KSshProcess:connect(): Got continue prompt where we shouldn't (STATE_WAIT_PROMPT)";
                 mError = ERR_INTERACT;
                 mErrorMsg =
                     i18n("Error encountered while talking to ssh.");
@@ -906,8 +871,7 @@
                 mConnectState = STATE_WAIT_PROMPT;
             }
             else {
-                kDebug(KSSHPROC) << "KSshProcess::connect() "
-                    "Need password from caller." << endl;
+                kDebug(KSSHPROC) << "need password from caller.";
                 // The caller needs to supply a password before
                 // connecting can continue.
                 mError = ERR_NEED_PASSWD;
@@ -945,8 +909,7 @@
                 mConnectState = STATE_WAIT_PROMPT;
             }
             else {
-                kDebug(KSSHPROC) << "KSshProcess::connect() "
-                    "Need passphrase from caller." << endl;
+                kDebug(KSSHPROC) << "need passphrase from caller.";
                 // The caller needs to supply a passphrase before
                 // connecting can continue.
                 mError = ERR_NEED_PASSPHRASE;
@@ -975,8 +938,7 @@
         case STATE_NEW_KEY_WAIT_CONTINUE:
             line = getLine();
             if( line.isNull() ) {
-                kDebug(KSSHPROC) << "KSshProcess::connect(): "
-                    "Got null line in STATE_NEW_KEY_WAIT_CONTINUE." << endl;
+                kDebug(KSSHPROC) << "Got null line in STATE_NEW_KEY_WAIT_CONTINUE.";
                 mError = ERR_INTERACT;
                 mErrorMsg =
                     i18n("Error encountered while talking to ssh.");
@@ -1047,8 +1009,7 @@
         case STATE_DIFF_KEY_WAIT_CONTINUE:
             line = getLine();
             if( line.isNull() ) {
-                kDebug(KSSHPROC) << "KSshProcess::connect(): "
-                    "Got null line in STATE_DIFF_KEY_WAIT_CONTINUE." << endl;
+                kDebug(KSSHPROC) << "Got null line in STATE_DIFF_KEY_WAIT_CONTINUE.";
                 mError = ERR_INTERACT;
                 mErrorMsg =
                     i18n("Error encountered while talking to ssh.");
@@ -1108,14 +1069,12 @@
         // We found a continue prompt.  Send our answer.
         case STATE_SEND_CONTINUE:
             if( mAcceptHostKey ) {
-                kDebug(KSSHPROC) << "KSshProcess::connect(): "
-                    "host key accepted" << endl;
+                kDebug(KSSHPROC) << "host key accepted";
                 ssh.writeLine("yes");
                 mConnectState = STATE_WAIT_PROMPT;
             }
             else {
-                kDebug(KSSHPROC) << "KSshProcess::connect(): "
-                    "host key rejected" << endl;
+                kDebug(KSSHPROC) << "host key rejected";
                 ssh.writeLine("no");
                 mError = ERR_HOST_KEY_REJECTED;
                 mErrorMsg = i18n("Host key was rejected.");
@@ -1139,16 +1098,14 @@
             return false;
         
         default:
-            kDebug(KSSHPROC) << "KSshProcess::connect(): "
-                "Invalid state number - " << mConnectState << endl;
+            kDebug(KSSHPROC) << "Invalid state number - " << mConnectState;
             mError = ERR_INVALID_STATE;
             mConnectState = STATE_FATAL;
         }
     }
 
     // we should never get here
-    kDebug(KSSHPROC) << "KSshProcess::connect(): " <<
-        "After switch(). We shouldn't be here." << endl;
+    kDebug(KSSHPROC) << "After switch(). We shouldn't be here.";
     mError = ERR_INTERNAL;
     return false;
 }
--- runtime/kioslave/sftp/sftpfileattr.h	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/kioslave/sftp/sftpfileattr.h	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -251,16 +251,15 @@
     void setDirAttrsFlag(bool flag){ mDirAttrs = flag; }
 
     /** Gets the DirAttrs flag. */
-    bool getDirAttrsFlag() const { return mDirAttrs; }
-
-    friend kdbgstream& operator<< (kdbgstream& s, sftpFileAttr& a);
-    friend kndbgstream& operator<< (kndbgstream& s, sftpFileAttr& a);
-
+    bool getDirAttrsFlag() const { return mDirAttrs; }    
+    
     /** Returns the file type as determined from the file permissions */
     mode_t fileType() const;
 
     /** Set the encoding of the remote file system */
     void setEncoding( KRemoteEncoding* encoding );
+
+    friend QDebug operator<<(QDebug s, sftpFileAttr& a);
 };
 
 #endif
--- runtime/kioslave/sftp/ksshprocess.h	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/kioslave/sftp/ksshprocess.h	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -321,7 +321,7 @@
      * @param pathToSsh The fully qualified path name of the ssh binary
      *                  KSshProcess should use to setup a SSH connection.
      */
-    KSshProcess(QString pathToSsh);
+    KSshProcess(const QString& pathToSsh);
     ~KSshProcess();
 
     /**
@@ -334,7 +334,7 @@
      *         recognizes the version.
      *
      */
-     bool setSshPath(QString pathToSsh);
+     bool setSshPath(const QString& pathToSsh);
 
     /**
      * Get the ssh version.
@@ -483,7 +483,7 @@
      *
      * @param password The user password to give ssh.
      */
-    void setPassword(QString password);
+    void setPassword(const QString& password);
      
     /**
      * Access to standard in and out of the ssh process.
--- runtime/kioslave/sftp/kio_sftp.cpp	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/kioslave/sftp/kio_sftp.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -133,14 +133,14 @@
              : SlaveBase("kio_sftp", pool_socket, app_socket),
                   mConnected(false), mPort(-1), mMsgId(0) {
 #ifndef Q_WS_WIN
-  kDebug(KIO_SFTP_DB) << "sftpProtocol(): pid = " << getpid();
+  kDebug(KIO_SFTP_DB) << "pid = " << getpid();
 #endif
 }
 
 
 sftpProtocol::~sftpProtocol() {
 #ifndef Q_WS_WIN
-    kDebug(KIO_SFTP_DB) << "~sftpProtocol(): pid = " << getpid();
+    kDebug(KIO_SFTP_DB) << "pid = " << getpid();
 #endif
     closeConnection();
 }
@@ -186,7 +186,7 @@
       return sftpVersion >= 3 ? true : false;
     default:
       kDebug(KIO_SFTP_DB) << "isSupportedOperation(type:"
-                            << type << "): unrecognized operation type" << endl;
+                            << type << "): unrecognized operation type";
       break;
   }
 
@@ -195,7 +195,7 @@
 
 void sftpProtocol::copy(const KUrl &src, const KUrl &dest, int permissions, KIO::JobFlags flags)
 {
-    kDebug(KIO_SFTP_DB) << "copy(): " << src << " -> " << dest;
+    kDebug(KIO_SFTP_DB) << src << " -> " << dest;
 
     bool srcLocal = src.isLocalFile();
     bool destLocal = dest.isLocalFile();
@@ -210,7 +210,7 @@
 
 void sftpProtocol::sftpCopyGet(const KUrl& dest, const KUrl& src, int mode, KIO::JobFlags flags)
 {
-    kDebug(KIO_SFTP_DB) << "sftpCopyGet(): " << src << " -> " << dest;
+    kDebug(KIO_SFTP_DB) << src << " -> " << dest;
 
     // Attempt to establish a connection...
     openConnection();
@@ -253,7 +253,7 @@
             if (canResume( buff_part.st_size ))
             {
                 offset = buff_part.st_size;
-                kDebug(KIO_SFTP_DB) << "sftpCopyGet: Resuming @ " << offset;
+                kDebug(KIO_SFTP_DB) << "Resuming @ " << offset;
             }
         }
 
@@ -285,7 +285,7 @@
 
     if(fd == -1)
     {
-      kDebug(KIO_SFTP_DB) << "sftpCopyGet: Unable to open (" << fd << ") for writing.";
+      kDebug(KIO_SFTP_DB) << "Unable to open (" << fd << ") for writing.";
       if (errno == EACCES)
         error (ERR_WRITE_ACCESS_DENIED, dest.prettyUrl());
       else
@@ -322,7 +322,7 @@
     }
 
     data(QByteArray());
-    kDebug(KIO_SFTP_DB) << "sftpCopyGet(): emit finished()";
+    kDebug(KIO_SFTP_DB) << "emit finished()";
     finished();
 }
 
@@ -335,7 +335,7 @@
     res.code = 0;
     res.size = 0;
 
-    kDebug(KIO_SFTP_DB) << "sftpGet(): " << src;
+    kDebug(KIO_SFTP_DB) << src;
 
     // stat the file first to get its size
     if( (code = sftpStat(src, attr)) != SSH2_FX_OK ) {
@@ -373,7 +373,7 @@
     quint32 len = 60*1024;
     code = SSH2_FX_OK;
 
-    kDebug(KIO_SFTP_DB) << "sftpGet(): offset = " << offset;
+    kDebug(KIO_SFTP_DB) << "offset = " << offset;
     while( code == SSH2_FX_OK ) {
         if( (code = sftpRead(handle, offset, len, buff)) == SSH2_FX_OK ) {
             offset += buff.size();
@@ -387,8 +387,7 @@
                 if( mimeBuffer.size() > 1024 ||  offset == fileSize ) {
                     // determine mimetype
                     KMimeType::Ptr mime = KMimeType::findByNameAndContent(src.fileName(), mimeBuffer);
-                    kDebug(KIO_SFTP_DB) << "sftpGet(): mimetype is " <<
-                                      mime->name() << endl;
+                    kDebug(KIO_SFTP_DB) << "mimetype is " << mime->name();
                     mimeType(mime->name());
 
                     if (abortAfterMimeType)
@@ -508,8 +507,7 @@
     if(mConnected)
       return;
 
-    kDebug(KIO_SFTP_DB) << "openConnection(): " << mUsername << "@"
-                         << mHost << ":" << mPort << endl;
+    kDebug(KIO_SFTP_DB) << "username=" << mUsername << ", host=" << mHost << ", port=" << mPort;
 
     infoMessage( i18n("Opening SFTP connection to host %1:%2", mHost, mPort));
 
@@ -519,7 +517,6 @@
         return;
     }
 
-    ////////////////////////////////////////////////////////////////////////////
     // Setup AuthInfo for use with password caching and the
     // password dialog box.
     AuthInfo info;
@@ -533,10 +530,8 @@
     info.username = mUsername;
     info.keepPassword = true;
 
-    ///////////////////////////////////////////////////////////////////////////
-    // Check for cached authentication info if a username AND password were
-    // not specified in setHost().
-    if( mUsername.isEmpty() && mPassword.isEmpty() ) {
+    // Check for cached authentication info if no password is specified...
+    if( mPassword.isEmpty() ) {
         kDebug(KIO_SFTP_DB) << "openConnection(): checking cache "
                              << "info.username = " << info.username
                              << ", info.url = " << info.url.prettyUrl() << endl;
@@ -596,14 +591,14 @@
     if( !mUsername.isEmpty()  ) {
         opt.opt = KSshProcess::SSH_USERNAME;
         opt.str = mUsername;
-		opts.append(opt);
+        opts.append(opt);
         usernameIt = opts.end()-1;
     }
 
     if( !mPassword.isEmpty() ) {
         opt.opt = KSshProcess::SSH_PASSWD;
         opt.str = mPassword;
-		opts.append(opt);
+        opts.append(opt);
         passwdIt = opts.end()-1;
     }
 
@@ -622,8 +617,7 @@
 
     while( !(mConnected = ssh.connect()) ) {
         err = ssh.error();
-        kDebug(KIO_SFTP_DB) << "openConnection(): "
-            "Got " << err << " from KSshProcess::connect()" << endl;
+        kDebug(KIO_SFTP_DB) << "Got " << err << " from KSshProcess::connect()";
 
         switch(err) {
         case KSshProcess::ERR_NEED_PASSWD:
@@ -637,23 +631,16 @@
             else
                 info.prompt = i18n("Please enter your username and password.");
 
-            kDebug(KIO_SFTP_DB) << "openConnection(): info.username = " << info.username
-                                 << ", info.url = " << info.url.prettyUrl() << endl;
+            kDebug(KIO_SFTP_DB) << "info.username = " << info.username
+                                 << ", info.url = " << info.url.prettyUrl();
 
             if( firstTime )
                 dlgResult = openPasswordDialog(info);
             else
                 dlgResult = openPasswordDialog(info, i18n("Incorrect username or password"));
 
-            if( dlgResult ) {
-               if( info.username.isEmpty() || info.password.isEmpty() ) {
-                    error(ERR_COULD_NOT_AUTHENTICATE,
-                      i18n("Please enter a username and password"));
-                    continue;
-                }
-            }
-            else {
-                // user canceled or dialog failed to open
+            // handle user canceled or dialog failed to open...
+            if( !dlgResult ) {
                 error(ERR_USER_CANCELED, QString());
                 kDebug(KIO_SFTP_DB) << "openConnection(): user canceled, dlgResult = " << dlgResult;
                 closeConnection();
@@ -669,8 +656,8 @@
             // the password option so the user is not prompted for
             // it again.
             if( mUsername != info.username ) {
-                kDebug(KIO_SFTP_DB) << "openConnection(): Username changed from "
-                                     << mUsername << " to " << info.username << endl;
+                kDebug(KIO_SFTP_DB) << "Username changed from " << mUsername 
+                                    << " to " << info.username;
 
                 ssh.disconnect();
 
@@ -679,18 +666,16 @@
                 // the iterators will be equal to the empty iterator.
                 // Create the opts now and add them to the opt list.
                 if( usernameIt == KSshProcess::SshOptListIterator() ) {
-                    kDebug(KIO_SFTP_DB) << "openConnection(): "
-                        "Adding username to options list" << endl;
+                    kDebug(KIO_SFTP_DB) << "Adding username to options list";
                     opt.opt = KSshProcess::SSH_USERNAME;
-					opts.append(opt);
+                    opts.append(opt);
                     usernameIt = opts.end()-1;
                 }
 
                 if( passwdIt == KSshProcess::SshOptListIterator() ) {
-                    kDebug(KIO_SFTP_DB) << "openConnection(): "
-                        "Adding password to options list" << endl;
+                    kDebug(KIO_SFTP_DB) << "Adding password to options list";
                     opt.opt = KSshProcess::SSH_PASSWD;
-					opts.append(opt);
+                    opts.append(opt);
                     passwdIt = opts.end()-1;
                 }
 
@@ -766,7 +751,7 @@
         default:
             infoMessage(i18n("Connection failed."));
             // Don't call messageBox! Leave GUI handling to the apps (#108812)
-            caption = i18n("Unexpected SFTP error: %1", err);
+            caption = i18n("unexpected SFTP error: %1", err);
             msg = ssh.errorMsg();
             if (!msg.isEmpty()) {
                 caption += '\n';
@@ -832,8 +817,8 @@
     info.url.setUser(mUsername);
     info.username = mUsername;
     info.password = mPassword;
-    kDebug(KIO_SFTP_DB) << "sftpProtocol(): caching info.username = " << info.username <<
-        ", info.url = " << info.url.prettyUrl() << endl;
+    kDebug(KIO_SFTP_DB) << "Caching info.username = " << info.username 
+                        << ", info.url = " << info.url.prettyUrl();
     cacheAuthentication(info);
     mConnected = true;
     connected();
@@ -848,7 +833,7 @@
 
 void sftpProtocol::open(const KUrl &url, QIODevice::OpenMode mode)
 {
-    _DEBUG << url.url() << endl;
+    _DEBUG << url;
     openConnection();
     if (!mConnected) {
         error(KIO::ERR_CONNECTION_BROKEN, url.prettyUrl());
@@ -859,19 +844,19 @@
     sftpFileAttr attr(remoteEncoding());
 
     if ((code = sftpStat(url, attr)) != SSH2_FX_OK) {
-        _DEBUG << "stat error" << endl;
+        _DEBUG << "stat error";
         processStatus(code, url.prettyUrl());
         return;
     }
 
     // don't open a directory
     if (attr.fileType() == S_IFDIR) {
-        _DEBUG << "a directory" << endl;
+        _DEBUG << "a directory";
         error(KIO::ERR_IS_DIRECTORY, url.prettyUrl());
         return;
     }
     if (attr.fileType() != S_IFREG) {
-        _DEBUG << "not a regular file" << endl;
+        _DEBUG << "not a regular file";
         error(KIO::ERR_CANNOT_OPEN_FOR_READING, url.prettyUrl());
         return;
     }
@@ -899,7 +884,7 @@
 
     code = sftpOpen(url, pflags, attr, openHandle);
     if (code != SSH2_FX_OK) {
-        _DEBUG << "sftpOpen error" << endl;
+        _DEBUG << "Got error code " << code;
         processStatus(code, url.prettyUrl());
         return;
     }
@@ -912,7 +897,7 @@
         QByteArray buffer;
         code = sftpRead(openHandle, 0, 1024, buffer);
         if ((code != SSH2_FX_OK) && (code != SSH2_FX_EOF)){
-            _DEBUG << "error on mime type detection" << endl;
+            _DEBUG << "error on mime type detection";
             processStatus(code, url.prettyUrl());
             close();
             return;
@@ -930,7 +915,7 @@
 
 void sftpProtocol::read(KIO::filesize_t bytes)
 {
-    _DEBUG << "read, offset = " << openOffset << ", bytes = " << bytes << endl;
+    _DEBUG << "read, offset = " << openOffset << ", bytes = " << bytes;
     QByteArray buffer;
     int code = sftpRead(openHandle, openOffset, bytes, buffer);
     if ((code == SSH2_FX_OK) || (code == SSH2_FX_EOF)) {
@@ -944,7 +929,7 @@
 
 void sftpProtocol::write(const QByteArray &data)
 {
-    _DEBUG << "write" << endl;
+    _DEBUG << "write";
     int code = sftpWrite(openHandle, openOffset, data);
     if (code == SSH2_FX_OK) {
         openOffset += data.size();
@@ -957,7 +942,7 @@
 
 void sftpProtocol::seek(KIO::filesize_t offset)
 {
-    _DEBUG << "seek, offset = " << offset << endl;
+    _DEBUG << "seek, offset = " << offset;
     openOffset = offset;
     position(offset);
 }
@@ -965,7 +950,7 @@
 void sftpProtocol::close()
 {
     sftpClose(openHandle);
-    _DEBUG << "emitting finished" << endl;
+    _DEBUG << "emitting finished";
     finished();
 }
 #undef _DEBUG
@@ -1010,9 +995,7 @@
     if( !mConnected )
         return;
 
-    kDebug(KIO_SFTP_DB) << "sftpPut(): " << dest
-                         << ", resume=" << (flags & KIO::Resume)
-                         << ", overwrite=" << (flags & KIO::Overwrite) << endl;
+    kDebug(KIO_SFTP_DB) << dest << ", resume=" << (flags & KIO::Resume) << ", overwrite=" << (flags & KIO::Overwrite);
 
     KUrl origUrl( dest );
     sftpFileAttr origAttr(remoteEncoding());
@@ -1022,7 +1005,7 @@
     int code = sftpStat(origUrl, origAttr);
 
     if( code == SSH2_FX_OK ) {
-        kDebug(KIO_SFTP_DB) << "sftpPut(): <file> already exists";
+        kDebug(KIO_SFTP_DB) << qPrintable(origUrl.url()) << " already exists!";
 
         // Delete remote file if its size is zero
         if( origAttr.fileSize() == 0 ) {
@@ -1061,7 +1044,7 @@
         code = sftpStat(partUrl, partAttr);
 
         if( code == SSH2_FX_OK ) {
-            kDebug(KIO_SFTP_DB) << "sftpPut(): .part file already exists";
+            kDebug(KIO_SFTP_DB) << ".part file already exists";
             partExists = true;
             offset = partAttr.fileSize();
 
@@ -1088,7 +1071,7 @@
                     else
                         flags |= canResume( offset ) ? KIO::Resume : KIO::DefaultFlags;
 
-                    kDebug(KIO_SFTP_DB) << "sftpPut(): can resume = " << (flags & KIO::Resume)
+                    kDebug(KIO_SFTP_DB) << "can resume = " << (flags & KIO::Resume)
                                          << ", offset = " << offset;
 
                     if( !(flags & KIO::Resume) ) {
@@ -1229,13 +1212,13 @@
 
 void sftpProtocol::put ( const KUrl& url, int permissions, KIO::JobFlags flags ) {
     kDebug(KIO_SFTP_DB) << "put(): " << url << ", overwrite = " << (flags & KIO::Overwrite)
-                         << ", resume = " << (flags & KIO::Resume) << endl;
+                         << ", resume = " << (flags & KIO::Resume);
 
     sftpPut( url, permissions, flags );
 }
 
 void sftpProtocol::stat ( const KUrl& url ){
-    kDebug(KIO_SFTP_DB) << "stat(): " << url;
+    kDebug(KIO_SFTP_DB) << url;
 
     openConnection();
     if( !mConnected )
@@ -1267,19 +1250,19 @@
     }
     else {
         //kDebug() << "We sent and received stat packet ok";
-        attr.setFilename(url.fileName());
+        //attr.setFilename(url.fileName());
         statEntry(attr.entry());
     }
 
     finished();
 
-    kDebug(KIO_SFTP_DB) << "stat: END";
+    //kDebug(KIO_SFTP_DB) << "END";
     return;
 }
 
 
 void sftpProtocol::mimetype ( const KUrl& url ){
-    kDebug(KIO_SFTP_DB) << "mimetype(): " << url;
+    kDebug(KIO_SFTP_DB) << url;
 
     openConnection();
     if( !mConnected )
@@ -1298,7 +1281,7 @@
 
 
 void sftpProtocol::listDir(const KUrl& url) {
-    kDebug(KIO_SFTP_DB) << "listDir(): " << url;
+    kDebug(KIO_SFTP_DB) << url;
 
     openConnection();
     if( !mConnected )
@@ -1318,7 +1301,7 @@
     QByteArray handle;
 
     if( (code = sftpOpenDirectory(url, handle)) != SSH2_FX_OK ) {
-        kError(KIO_SFTP_DB) << "listDir(): open directory failed" << endl;
+        kError(KIO_SFTP_DB) << "listDir(): open directory failed";
         processStatus(code, url.prettyUrl());
         return;
     }
@@ -1333,7 +1316,7 @@
     }
 
     if( (code = sftpClose(handle)) != SSH2_FX_OK ) {
-        kError(KIO_SFTP_DB) << "listdir(): closing of directory failed" << endl;
+        kError(KIO_SFTP_DB) << "listdir(): closing of directory failed";
         processStatus(code, url.prettyUrl());
         return;
     }
@@ -1351,7 +1334,7 @@
  */
 void sftpProtocol::mkdir(const KUrl&url, int permissions){
 
-    kDebug(KIO_SFTP_DB) << "mkdir() creating dir: " << url.path();
+    kDebug(KIO_SFTP_DB) << "create directory: " << url.path();
 
     openConnection();
     if( !mConnected )
@@ -1386,14 +1369,14 @@
 
     r >> type >> id;
     if( id != expectedId ) {
-        kError(KIO_SFTP_DB) << "mkdir: sftp packet id mismatch" << endl;
+        kError(KIO_SFTP_DB) << "mkdir: sftp packet id mismatch";
         error(ERR_COULD_NOT_MKDIR, path);
         finished();
         return;
     }
 
     if( type != SSH2_FXP_STATUS ) {
-        kError(KIO_SFTP_DB) << "mkdir(): unexpected packet type of " << type << endl;
+        kError(KIO_SFTP_DB) << "mkdir(): unexpected packet type of " << type;
         error(ERR_COULD_NOT_MKDIR, path);
         finished();
         return;
@@ -1402,7 +1385,7 @@
     int code;
     r >> code;
     if( code != SSH2_FX_OK ) {
-        kError(KIO_SFTP_DB) << "mkdir(): failed with code " << code << endl;
+        kError(KIO_SFTP_DB) << "mkdir(): failed with code " << code;
 
         // Check if mkdir failed because the directory already exists so that
         // we can return the appropriate message...
@@ -1420,7 +1403,7 @@
 }
 
 void sftpProtocol::rename(const KUrl& src, const KUrl& dest, KIO::JobFlags flags) {
-    kDebug(KIO_SFTP_DB) << "rename(" << src << " -> " << dest << ")";
+    kDebug(KIO_SFTP_DB) << src << " -> " << dest;
 
     if (!isSupportedOperation(SSH2_FXP_RENAME)) {
       error(ERR_UNSUPPORTED_ACTION,
@@ -1465,11 +1448,11 @@
     }
 
     finished();
-    kDebug(KIO_SFTP_DB) << "rename(): END";
+    kDebug(KIO_SFTP_DB) << "END";
 }
 
 void sftpProtocol::symlink(const QString& target, const KUrl& dest, KIO::JobFlags flags) {
-    kDebug(KIO_SFTP_DB) << "symlink()";
+    kDebug(KIO_SFTP_DB) << "link " << target << "->" << dest;
 
     if (!isSupportedOperation(SSH2_FXP_SYMLINK)) {
       error(ERR_UNSUPPORTED_ACTION,
@@ -1521,8 +1504,9 @@
 void sftpProtocol::chmod(const KUrl& url, int permissions){
     QString perms;
     perms.setNum(permissions, 8);
-    kDebug(KIO_SFTP_DB) << "chmod(" << url << ", " << perms << ")";
 
+    kDebug(KIO_SFTP_DB) << "change permission of " << url << " to " << perms;
+
     openConnection();
     if( !mConnected )
         return;
@@ -1534,7 +1518,7 @@
 
     int code;
     if( (code = sftpSetStat(url, attr)) != SSH2_FX_OK ) {
-        kError(KIO_SFTP_DB) << "chmod(): sftpSetStat failed with error " << code << endl;
+        kError(KIO_SFTP_DB) << "cannot stat failed with error " << code;
         if( code == SSH2_FX_FAILURE )
             error(ERR_CANNOT_CHMOD, QString());
         else
@@ -1545,7 +1529,7 @@
 
 
 void sftpProtocol::del(const KUrl &url, bool isfile){
-    kDebug(KIO_SFTP_DB) << "del(" << url << ", " << (isfile?"file":"dir") << ")";
+    kDebug(KIO_SFTP_DB) << "delete " << (isfile ? "file: " : "directory: ") << url;
 
     openConnection();
     if( !mConnected )
@@ -1553,16 +1537,14 @@
 
     int code;
     if( (code = sftpRemove(url, isfile)) != SSH2_FX_OK ) {
-        kError(KIO_SFTP_DB) << "del(): sftpRemove failed with error code " << code << endl;
+        kError(KIO_SFTP_DB) << "delete failed with error code " << code;
         processStatus(code, url.prettyUrl());
     }
     finished();
 }
 
 void sftpProtocol::slave_status() {
-    kDebug(KIO_SFTP_DB) << "slave_status(): connected to "
-                         <<  mHost << "? " << mConnected << endl;
-
+    kDebug(KIO_SFTP_DB) << "connected to " <<  mHost << "? " << mConnected;
     slaveStatus ((mConnected ? mHost : QString()), mConnected);
 }
 
@@ -1579,8 +1561,8 @@
     ssize_t len = atomicio(ssh.stdioFd(), buf.data(), 4, true /*read*/);
 #endif
     if( len == 0 || len == -1 ) {
-        kDebug(KIO_SFTP_DB) << "getPacket(): read of packet length failed, ret = "
-                             << len << ", error =" << strerror(errno) << endl;
+        kDebug(KIO_SFTP_DB) << "read of packet length failed, ret = "
+                             << len << ", error =" << strerror(errno);
         closeConnection();
         error( ERR_CONNECTION_BROKEN, mHost);
         msg.resize(0);
@@ -1591,7 +1573,7 @@
     QDataStream s(buf);
     s >> msgLen;
 
-    //kDebug(KIO_SFTP_DB) << "getPacket(): Message size = " << msgLen;
+    //kDebug(KIO_SFTP_DB) << "message size = " << msgLen;
 
     msg.resize(0);
 
@@ -1611,8 +1593,7 @@
               errmsg = i18n("Connection closed");
             else
               errmsg = i18n("Could not read SFTP packet");
-            kDebug(KIO_SFTP_DB) << "getPacket(): nothing to read, ret = " <<
-                len << ", error =" << strerror(errno) << endl;
+            kDebug(KIO_SFTP_DB) << "nothing to read, ret = " << len << ", error =" << strerror(errno);
             closeConnection();
             error(ERR_CONNECTION_BROKEN, errmsg);
             b.close();
@@ -1621,8 +1602,8 @@
 
         b.write(buf.data(), len);
 
-        //kDebug(KIO_SFTP_DB) << "getPacket(): Read Message size = " << len;
-        //kDebug(KIO_SFTP_DB) << "getPacket(): Copy Message size = " << msg.size();
+        //kDebug(KIO_SFTP_DB) << "read Message size = " << len;
+        //kDebug(KIO_SFTP_DB) << "copy Message size = " << msg.size();
 
         msgLen -= len;
     }
@@ -1642,8 +1623,7 @@
     ret = atomicio(ssh.stdioFd(), p.data(), p.size(), false /*write*/);
 #endif
     if( ret <= 0 ) {
-        kDebug(KIO_SFTP_DB) << "putPacket(): write failed, ret =" << ret <<
-            ", error = " << strerror(errno) << endl;
+        kDebug(KIO_SFTP_DB) << "write failed, ret =" << ret << ", error = " << strerror(errno);
         return false;
     }
 
@@ -1656,7 +1636,7 @@
 Returns the canonicalized url. */
 int sftpProtocol::sftpRealPath(const KUrl& url, KUrl& newUrl){
 
-    kDebug(KIO_SFTP_DB) << "sftpRealPath(" << url << ", newUrl)";
+    kDebug(KIO_SFTP_DB) << "get the real path of " << url;
 
     QByteArray path = remoteEncoding()->encode(url.path());
     uint len = path.length();
@@ -1679,7 +1659,7 @@
 
     r >> type >> id;
     if( id != expectedId ) {
-        kError(KIO_SFTP_DB) << "sftpRealPath: sftp packet id mismatch" << endl;
+        kError(KIO_SFTP_DB) << "sftp packet id mismatch";
         return -1;
     }
 
@@ -1690,14 +1670,14 @@
     }
 
     if( type != SSH2_FXP_NAME ) {
-        kError(KIO_SFTP_DB) << "sftpRealPath(): unexpected packet type of " << type << endl;
+        kError(KIO_SFTP_DB) << "unexpected packet type of " << type;
         return -1;
     }
 
     quint32 count;
     r >> count;
     if( count != 1 ) {
-        kError(KIO_SFTP_DB) << "sftpRealPath(): Bad number of file attributes for realpath command" << endl;
+        kError(KIO_SFTP_DB) << "bad number of file attributes for realpath command";
         return -1;
     }
 
@@ -1707,8 +1687,9 @@
     newPath.truncate(newPath.size());
     if (newPath.isEmpty())
       newPath = "/";
-    newUrl.setPath(newPath);
+    newUrl.setPath(remoteEncoding()->decode(newPath));
 
+    kDebug(KIO_SFTP_DB) << "real path is " << newUrl;
     return SSH2_FX_OK;
 }
 
@@ -1762,7 +1743,7 @@
 /** Opens a directory handle for url.path. Returns true if succeeds. */
 int sftpProtocol::sftpOpenDirectory(const KUrl& url, QByteArray& handle){
 
-    kDebug(KIO_SFTP_DB) << "sftpOpenDirectory(" << url << ", handle)";
+    kDebug(KIO_SFTP_DB) << " open directory " << url;
 
     QByteArray path = remoteEncoding()->encode(url.path());
     uint len = path.length();
@@ -1785,8 +1766,8 @@
 
     r >> type >> id;
     if( id != expectedId ) {
-        kError(KIO_SFTP_DB) << "sftpOpenDirectory: sftp packet id mismatch: " <<
-            "expected " << expectedId << ", got " << id << endl;
+        kError(KIO_SFTP_DB) << "sftp packet id mismatch: " <<
+            "expected " << expectedId << ", got " << id;
         return -1;
     }
 
@@ -1797,25 +1778,23 @@
     }
 
     if( type != SSH2_FXP_HANDLE ) {
-        kError(KIO_SFTP_DB) << "sftpOpenDirectory: unexpected message type of " << type << endl;
+        kError(KIO_SFTP_DB) << "unexpected message type of " << type;
         return -1;
     }
 
     r >> handle;
     if( handle.size() > 256 ) {
-        kError(KIO_SFTP_DB) << "sftpOpenDirectory: handle exceeds max length" << endl;
+        kError(KIO_SFTP_DB) << "Handle exceeds max length";
         return -1;
     }
 
-    kDebug(KIO_SFTP_DB) << "sftpOpenDirectory: handle (" << handle.size() << "): [" << handle << "]";
+    kDebug(KIO_SFTP_DB) << "directory handle (" << handle.size() << "): [" << handle << "]";
     return SSH2_FX_OK;
 }
 
 /** Closes a directory or file handle. */
 int sftpProtocol::sftpClose(const QByteArray& handle){
 
-    kDebug(KIO_SFTP_DB) << "sftpClose()";
-
     quint32 id, expectedId;
     id = expectedId = mMsgId++;
 
@@ -1834,19 +1813,19 @@
 
     r >> type >> id;
     if( id != expectedId ) {
-        kError(KIO_SFTP_DB) << "sftpClose: sftp packet id mismatch" << endl;
+        kError(KIO_SFTP_DB) << "sftp packet id mismatch";
         return -1;
     }
 
     if( type != SSH2_FXP_STATUS ) {
-        kError(KIO_SFTP_DB) << "sftpClose: unexpected message type of " << type << endl;
+        kError(KIO_SFTP_DB) << "unexpected message type of " << type;
         return -1;
     }
 
     quint32 code;
     r >> code;
     if( code != SSH2_FX_OK ) {
-        kError(KIO_SFTP_DB) << "sftpClose: close failed with err code " << code << endl;
+        kError(KIO_SFTP_DB) << "close failed with err code " << code;
     }
 
     return code;
@@ -1855,7 +1834,7 @@
 /** Set a files attributes. */
 int sftpProtocol::sftpSetStat(const KUrl& url, const sftpFileAttr& attr){
 
-    kDebug(KIO_SFTP_DB) << "sftpSetStat(" << url << ", attr)";
+    kDebug(KIO_SFTP_DB) << "stating url " << url;
 
     QByteArray path = remoteEncoding()->encode(url.path());
     uint len = path.length();
@@ -1879,20 +1858,20 @@
 
     r >> type >> id;
     if( id != expectedId ) {
-        kError(KIO_SFTP_DB) << "sftpSetStat(): sftp packet id mismatch" << endl;
+        kError(KIO_SFTP_DB) << "sftp packet id mismatch";
         return -1;
         // XXX How do we do a fatal error?
     }
 
     if( type != SSH2_FXP_STATUS ) {
-        kError(KIO_SFTP_DB) << "sftpSetStat(): unexpected message type of " << type << endl;
+        kError(KIO_SFTP_DB) << "unexpected message type of " << type;
         return -1;
     }
 
     quint32 code;
     r >> code;
     if( code != SSH2_FX_OK ) {
-        kError(KIO_SFTP_DB) << "sftpSetStat(): set stat failed with err code " << code << endl;
+        kError(KIO_SFTP_DB) << "set stat failed with err code " << code;
     }
 
     return code;
@@ -1901,7 +1880,7 @@
 /** Sends a sftp command to remove a file or directory. */
 int sftpProtocol::sftpRemove(const KUrl& url, bool isfile){
 
-    kDebug(KIO_SFTP_DB) << "sftpRemove(): " << url << ", isFile ? " << isfile;
+    kDebug(KIO_SFTP_DB) << "deleting " << (isfile ? "file " : "directory ") << url;
 
     QByteArray path = remoteEncoding()->encode(url.path());
     uint len = path.length();
@@ -1924,19 +1903,19 @@
 
     r >> type >> id;
     if( id != expectedId ) {
-        kError(KIO_SFTP_DB) << "del(): sftp packet id mismatch" << endl;
+        kError(KIO_SFTP_DB) << "sftp packet id mismatch";
         return -1;
     }
 
     if( type != SSH2_FXP_STATUS ) {
-        kError(KIO_SFTP_DB) << "del(): unexpected message type of " << type << endl;
+        kError(KIO_SFTP_DB) << "unexpected message type of " << type;
         return -1;
     }
 
     quint32 code;
     r >> code;
     if( code != SSH2_FX_OK ) {
-        kError(KIO_SFTP_DB) << "del(): del failed with err code " << code << endl;
+        kError(KIO_SFTP_DB) << "delete failed with error code " << code;
     }
 
     return code;
@@ -1945,7 +1924,7 @@
 /** Send a sftp command to rename a file or directory. */
 int sftpProtocol::sftpRename(const KUrl& src, const KUrl& dest){
 
-    kDebug(KIO_SFTP_DB) << "sftpRename(" << src << " -> " << dest << ")";
+    kDebug(KIO_SFTP_DB) << src << " -> " << dest;
 
     QByteArray srcPath = remoteEncoding()->encode(src.path());
     QByteArray destPath = remoteEncoding()->encode(dest.path());
@@ -1974,19 +1953,19 @@
 
     r >> type >> id;
     if( id != expectedId ) {
-        kError(KIO_SFTP_DB) << "sftpRename(): sftp packet id mismatch" << endl;
+        kError(KIO_SFTP_DB) << "sftp packet id mismatch";
         return -1;
     }
 
     if( type != SSH2_FXP_STATUS ) {
-        kError(KIO_SFTP_DB) << "sftpRename(): unexpected message type of " << type << endl;
+        kError(KIO_SFTP_DB) << "unexpected message type of " << type;
         return -1;
     }
 
     int code;
     r >> code;
     if( code != SSH2_FX_OK ) {
-        kError(KIO_SFTP_DB) << "sftpRename(): rename failed with err code " << code << endl;
+        kError(KIO_SFTP_DB) << "rename failed with err code " << code;
     }
 
     return code;
@@ -1994,7 +1973,7 @@
 /** Get directory listings. */
 int sftpProtocol::sftpReadDir(const QByteArray& handle, const KUrl& url){
     // url is needed so we can lookup the link destination
-    kDebug(KIO_SFTP_DB) << "sftpReadDir(): " << url;
+    kDebug(KIO_SFTP_DB) << url;
 
     quint32 id, expectedId, count;
     quint8 type;
@@ -2017,7 +1996,7 @@
     r >> type >> id;
 
     if( id != expectedId ) {
-        kError(KIO_SFTP_DB) << "sftpReadDir(): sftp packet id mismatch" << endl;
+        kError(KIO_SFTP_DB) << "sftp packet id mismatch";
         return -1;
     }
 
@@ -2028,12 +2007,12 @@
     }
 
     if( type != SSH2_FXP_NAME ) {
-        kError(KIO_SFTP_DB) << "kio_sftpProtocol::sftpReadDir(): Unexpected message" << endl;
+        kError(KIO_SFTP_DB) << "unexpected message";
         return -1;
     }
 
     r >> count;
-    kDebug(KIO_SFTP_DB) << "sftpReadDir(): got " << count << " entries";
+    kDebug(KIO_SFTP_DB) << "got " << count << " entries";
 
     while(count--) {
         r >> attr;
@@ -2060,13 +2039,13 @@
 
 int sftpProtocol::sftpReadLink(const KUrl& url, QString& target){
 
-    kDebug(KIO_SFTP_DB) << "sftpReadLink(): " << url;
+    kDebug(KIO_SFTP_DB) << url;
 
     QByteArray path = remoteEncoding()->encode(url.path());
     uint len = path.length();
 
-    //kDebug(KIO_SFTP_DB) << "sftpReadLink(): Encoded Path: " << path;
-    //kDebug(KIO_SFTP_DB) << "sftpReadLink(): Encoded Size: " << len;
+    //kDebug(KIO_SFTP_DB) << "Encoded Path: " << path;
+    //kDebug(KIO_SFTP_DB) << "Encoded Size: " << len;
 
     quint32 id, expectedId;
     id = expectedId = mMsgId++;
@@ -2087,26 +2066,26 @@
 
     r >> type >> id;
     if( id != expectedId ) {
-        kError(KIO_SFTP_DB) << "sftpReadLink(): sftp packet id mismatch" << endl;
+        kError(KIO_SFTP_DB) << "sftp packet id mismatch";
         return -1;
     }
 
     if( type == SSH2_FXP_STATUS ) {
         quint32 code;
         r >> code;
-        kDebug(KIO_SFTP_DB) << "sftpReadLink(): read link failed with code " << code;
+        kDebug(KIO_SFTP_DB) << "read link failed with code " << code;
         return code;
     }
 
     if( type != SSH2_FXP_NAME ) {
-        kError(KIO_SFTP_DB) << "sftpReadLink(): unexpected packet type of " << type << endl;
+        kError(KIO_SFTP_DB) << "unexpected packet type of " << type;
         return -1;
     }
 
     quint32 count;
     r >> count;
     if( count != 1 ) {
-        kError(KIO_SFTP_DB) << "sftpReadLink(): Bad number of file attributes for realpath command" << endl;
+        kError(KIO_SFTP_DB) << "bad number of file attributes for realpath command";
         return -1;
     }
 
@@ -2114,7 +2093,7 @@
     r >> linkAddress;
 
     linkAddress.truncate(linkAddress.size());
-    kDebug(KIO_SFTP_DB) << "sftpReadLink(): Link address: " << linkAddress;
+    kDebug(KIO_SFTP_DB) << "link address: " << linkAddress;
 
     target = remoteEncoding()->decode(linkAddress);
 
@@ -2128,7 +2107,7 @@
     uint dlen = destPath.length();
     uint tlen = target.length();
 
-    kDebug(KIO_SFTP_DB) << "sftpSymLink(" << target << " -> " << destPath << ")";
+    kDebug(KIO_SFTP_DB) << "(" << target << " -> " << destPath << ")";
 
     quint32 id, expectedId;
     id = expectedId = mMsgId++;
@@ -2151,19 +2130,19 @@
 
     r >> type >> id;
     if( id != expectedId ) {
-        kError(KIO_SFTP_DB) << "sftpSymLink(): sftp packet id mismatch" << endl;
+        kError(KIO_SFTP_DB) << "sftp packet id mismatch";
         return -1;
     }
 
     if( type != SSH2_FXP_STATUS ) {
-        kError(KIO_SFTP_DB) << "sftpSymLink(): unexpected message type of " << type << endl;
+        kError(KIO_SFTP_DB) << "unexpected message type of " << type;
         return -1;
     }
 
     quint32 code;
     r >> code;
     if( code != SSH2_FX_OK ) {
-        kError(KIO_SFTP_DB) << "sftpSymLink(): rename failed with err code " << code << endl;
+        kError(KIO_SFTP_DB) << "rename failed with err code " << code;
     }
 
     return code;
@@ -2172,7 +2151,7 @@
 /** Stats a file. */
 int sftpProtocol::sftpStat(const KUrl& url, sftpFileAttr& attr) {
 
-    kDebug(KIO_SFTP_DB) << "sftpStat(): " << url;
+    kDebug(KIO_SFTP_DB) << url;
 
     QByteArray path = remoteEncoding()->encode(url.path());
     uint len = path.length();
@@ -2195,25 +2174,25 @@
 
     r >> type >> id;
     if( id != expectedId ) {
-        kError(KIO_SFTP_DB) << "sftpStat(): sftp packet id mismatch" << endl;
+        kError(KIO_SFTP_DB) << "sftp packet id mismatch";
         return -1;
     }
 
     if( type == SSH2_FXP_STATUS ) {
         quint32 errCode;
         r >> errCode;
-        kError(KIO_SFTP_DB) << "sftpStat(): stat failed with code " << errCode << endl;
+        kError(KIO_SFTP_DB) << "stat failed with code " << errCode;
         return errCode;
     }
 
     if( type != SSH2_FXP_ATTRS ) {
-        kError(KIO_SFTP_DB) << "sftpStat(): unexpected message type of " << type << endl;
+        kError(KIO_SFTP_DB) << "unexpected message type of " << type;
         return -1;
     }
 
     r >> attr;
     attr.setFilename(url.fileName());
-    kDebug(KIO_SFTP_DB) << "sftpStat(): " << attr;
+    kDebug(KIO_SFTP_DB) << attr;
 
     // If the stat'ed resource is a symlink, perform a recursive stat
     // to determine the actual destination's type (file/dir).
@@ -2223,11 +2202,11 @@
         int code = sftpReadLink( url, target );
 
         if ( code != SSH2_FX_OK ) {
-            kError(KIO_SFTP_DB) << "sftpStat(): Unable to stat symlink destination" << endl;
+            kError(KIO_SFTP_DB) << "unable to stat symlink destination";
             return -1;
         }
 
-        kDebug(KIO_SFTP_DB) << "sftpStat(): Resource is a symlink to -> " << target;
+        kDebug(KIO_SFTP_DB) << "resource is a symlink that points to " << target;
 
         KUrl dest( url );
         if( target[0] == '/' )
@@ -2249,8 +2228,7 @@
                 attr.setLinkType(attr2.linkType());
 
             attr.setLinkDestination(target);
-
-            kDebug(KIO_SFTP_DB) << "sftpStat(): File type: " << attr.fileType();
+            attr.setPermissions(attr2.permissions());
         }
     }
 
@@ -2260,7 +2238,7 @@
 
 int sftpProtocol::sftpOpen(const KUrl& url, const quint32 pflags,
                            const sftpFileAttr& attr, QByteArray& handle) {
-    kDebug(KIO_SFTP_DB) << "sftpOpen(" << url << ", handle";
+    kDebug(KIO_SFTP_DB) << url;
 
     QByteArray path = remoteEncoding()->encode(url.path());
     uint len = path.length();
@@ -2287,7 +2265,7 @@
 
     r >> type >> id;
     if( id != expectedId ) {
-        kError(KIO_SFTP_DB) << "sftpOpen(): sftp packet id mismatch" << endl;
+        kError(KIO_SFTP_DB) << "sftp packet id mismatch";
         return -1;
     }
 
@@ -2298,24 +2276,24 @@
     }
 
     if( type != SSH2_FXP_HANDLE ) {
-        kError(KIO_SFTP_DB) << "sftpOpen(): unexpected message type of " << type << endl;
+        kError(KIO_SFTP_DB) << "unexpected message type of " << type;
         return -1;
     }
 
     r >> handle;
     if( handle.size() > 256 ) {
-        kError(KIO_SFTP_DB) << "sftpOpen(): handle exceeds max length" << endl;
+        kError(KIO_SFTP_DB) << "handle exceeds max length";
         return -1;
     }
 
-    kDebug(KIO_SFTP_DB) << "sftpOpen(): handle (" << handle.size() << "): [" << handle << "]";
+    kDebug(KIO_SFTP_DB) << "URL handle (" << handle.size() << "): [" << handle << "]";
     return SSH2_FX_OK;
 }
 
 
 int sftpProtocol::sftpRead(const QByteArray& handle, KIO::filesize_t offset, quint32 len, QByteArray& data)
 {
- //   kDebug(KIO_SFTP_DB) << "sftpRead( offset = " << offset << ", len = " << len << ")";
+ //   kDebug(KIO_SFTP_DB) << "( offset = " << offset << ", len = " << len << ")";
     QByteArray p;
     QDataStream s(&p, QIODevice::WriteOnly);
 
@@ -2338,19 +2316,19 @@
 
     r >> type >> id;
     if( id != expectedId ) {
-        kError(KIO_SFTP_DB) << "sftpRead: sftp packet id mismatch" << endl;
+        kError(KIO_SFTP_DB) << "sftp packet id mismatch";
         return -1;
     }
 
     if( type == SSH2_FXP_STATUS ) {
         quint32 errCode;
         r >> errCode;
-        kError(KIO_SFTP_DB) << "sftpRead: read failed with code " << errCode << endl;
+        kError(KIO_SFTP_DB) << "Read failed with code " << errCode;
         return errCode;
     }
 
     if( type != SSH2_FXP_DATA ) {
-        kError(KIO_SFTP_DB) << "sftpRead: unexpected message type of " << type << endl;
+        kError(KIO_SFTP_DB) << "unexpected message type of " << type;
         return -1;
     }
 
@@ -2362,7 +2340,7 @@
 
 int sftpProtocol::sftpWrite(const QByteArray& handle, KIO::filesize_t offset, const QByteArray& data){
 //    kDebug(KIO_SFTP_DB) << "sftpWrite( offset = " << offset <<
-//        ", data sz = " << data.size() << ")" << endl;
+//        ", data sz = " << data.size() << ")";
     QByteArray p;
     QDataStream s(&p, QIODevice::WriteOnly);
 
@@ -2378,28 +2356,28 @@
     s << offset; // we don't have a convienient 64 bit int so set upper int to zero
     s << data;
 
-//    kDebug(KIO_SFTP_DB) << "sftpWrite(): SSH2_FXP_WRITE, id:"
-//        << id << ", handle:" << handle << ", offset:" << offset << ", some data" << endl;
+//    kDebug(KIO_SFTP_DB) << "SSH2_FXP_WRITE, id:"
+//        << id << ", handle:" << handle << ", offset:" << offset << ", some data";
 
-//    kDebug(KIO_SFTP_DB) << "sftpWrite(): send packet [" << p << "]";
+//    kDebug(KIO_SFTP_DB) << "Send packet [" << p << "]";
 
     putPacket(p);
     getPacket(p);
 
-//    kDebug(KIO_SFTP_DB) << "sftpWrite(): received packet [" << p << "]";
+//    kDebug(KIO_SFTP_DB) << "Received packet [" << p << "]";
 
     QDataStream r(p);
     quint8 type;
 
     r >> type >> id;
     if( id != expectedId ) {
-        kError(KIO_SFTP_DB) << "sftpWrite(): sftp packet id mismatch, got "
-            << id << ", expected " << expectedId << endl;
+        kError(KIO_SFTP_DB) << "sftp packet id mismatch, got "
+            << id << ", expected " << expectedId;
         return -1;
     }
 
     if( type != SSH2_FXP_STATUS ) {
-        kError(KIO_SFTP_DB) << "sftpWrite(): unexpected message type of " << type << endl;
+        kError(KIO_SFTP_DB) << "unexpected message type of " << type;
         return -1;
     }
 
@@ -2407,5 +2385,3 @@
     r >> code;
     return code;
 }
-
-
--- runtime/kioslave/sftp/process.cpp	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/kioslave/sftp/process.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -69,7 +69,7 @@
     m_pPTY = new KPty();
     if (!m_pPTY->open())
     {
-        kError(PTYPROC) << k_lineinfo << "Master setup failed.\n" << endl;
+        kError(PTYPROC) << k_lineinfo << "Master setup failed.";
         return -1;
     }
 #else
@@ -148,7 +148,7 @@
     int flags = fcntl(fd, F_GETFL);
     if (flags < 0) 
     {
-        kError(PTYPROC) << k_lineinfo << "fcntl(F_GETFL): " << perror << "\n";
+        kError(PTYPROC) << k_lineinfo << "fcntl(F_GETFL): " << perror;
         return ret;
     }
     if (block)
@@ -157,7 +157,7 @@
         flags |= O_NONBLOCK;
     if (fcntl(fd, F_SETFL, flags) < 0)
     {
-        kError(PTYPROC) << k_lineinfo << "fcntl(F_SETFL): " << perror << "\n";
+        kError(PTYPROC) << k_lineinfo << "fcntl(F_SETFL): " << perror;
         return ret;
     }
 
@@ -244,7 +244,7 @@
     int slave = open(m_pPTY->ttyName(), O_RDWR);
     if (slave < 0) 
     {
-        kError(PTYPROC) << k_lineinfo << "Could not open slave pty.\n";
+        kError(PTYPROC) << k_lineinfo << "Could not open slave pty.";
         return -1;
     } 
 
@@ -265,7 +265,7 @@
 
     if ((m_Pid = fork()) == -1) 
     {
-        kError(PTYPROC) << k_lineinfo << "fork(): " << perror << "\n";
+        kError(PTYPROC) << k_lineinfo << "fork(): " << perror;
         return -1;
     } 
 
@@ -287,7 +287,7 @@
 
     if( !ok )
     {
-        kError(PTYPROC) << "dup of socket descriptor failed" << endl;
+        kError(PTYPROC) << "dup of socket descriptor failed";
         _exit(1);
     }
 
@@ -308,7 +308,7 @@
 	    QString file = KStandardDirs::findExe(command);
     	if (file.isEmpty())
 	    {
-	        kError(PTYPROC) << k_lineinfo << command << " not found\n";
+	        kError(PTYPROC) << k_lineinfo << command << " not found";
     	    _exit(1);
 	    }
     	path = QFile::encodeName(file);
@@ -324,7 +324,7 @@
     }
     argp[i] = 0L;
     execv(path, (char * const *)argp);
-    kError(PTYPROC) << k_lineinfo << "execv(\"" << path << "\"): " << perror << "\n";
+    kError(PTYPROC) << k_lineinfo << "execv(\"" << path << "\"): " << perror;
     _exit(1);
     return -1; // Shut up compiler. Never reached.
 #endif
@@ -346,7 +346,7 @@
     int slave = open(m_pPTY->ttyName(), O_RDWR);
     if (slave < 0) 
     {
-	kError(PTYPROC) << k_lineinfo << "Could not open slave tty.\n";
+	kError(PTYPROC) << k_lineinfo << "Could not open slave tty.";
 	return -1;
     }
 
@@ -356,7 +356,7 @@
     {
 	if (tcgetattr(slave, &tio) < 0) 
 	{
-	    kError(PTYPROC) << k_lineinfo << "tcgetattr(): " << perror << "\n";
+	    kError(PTYPROC) << k_lineinfo << "tcgetattr(): " << perror;
 	    close(slave);
 	    return -1;
 	}
@@ -417,7 +417,7 @@
 	    if (errno == EINTR) continue;
 	    else 
 	    {
-		kError(PTYPROC) << k_lineinfo << "select(): " << perror << "\n";
+		kError(PTYPROC) << k_lineinfo << "select(): " << perror;
 		return -1;
 	    }
 	}
@@ -445,7 +445,7 @@
 	    if (errno == ECHILD)
 		retval = 0;
 	    else
-		kError(PTYPROC) << k_lineinfo << "waitpid(): " << perror << "\n";
+		kError(PTYPROC) << k_lineinfo << "waitpid(): " << perror;
 	    break;
 	}
 	if (ret == m_Pid) 
@@ -488,13 +488,13 @@
     struct ::termios tio;
     if (m_pPTY->tcGetAttr(&tio) < 0)
     {
-        kError(PTYPROC) << k_lineinfo << "tcgetattr(): " << perror << "\n";
+        kError(PTYPROC) << k_lineinfo << "tcgetattr(): " << perror;
         return -1;
     }
     tio.c_oflag &= ~OPOST;
     if (m_pPTY->tcSetAttr(&tio) < 0)
     {
-        kError(PTYPROC) << k_lineinfo << "tcsetattr(): " << perror << "\n";
+        kError(PTYPROC) << k_lineinfo << "tcsetattr(): " << perror;
         return -1;
     }
 #endif
--- runtime/nepomuk/services/strigi/nepomukstrigiservice.notifyrc	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/nepomuk/services/strigi/nepomukstrigiservice.notifyrc	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -239,7 +239,7 @@
 Comment[hne]=तेज डेस्कटाप खोज बर स्ट्रिगि सुरु के इंडेक्सिंग लोकल फाइल बर पूरा कर लिस
 Comment[hsb]=Strigi je prěnje tworjenje indeksa za lokalne dataje zakónčił, za spěšne pytanje na dźěłowym powjerchu
 Comment[is]=Strigi hefur lokið gerð upphafsyfirlits um skrár á tölvunni til að styðja við hraðar skjáborðsleitir
-Comment[it]=Strigi ha completato l'indicizzazione iniziale dei file locali per velocizzare le ricerch desktop
+Comment[it]=Strigi ha completato l'indicizzazione iniziale dei file locali per velocizzare le ricerche desktop
 Comment[ja]=Strigi が、デスクトップ検索を高速化するためのローカルファイルの最初のインデックス作成を完了しました
 Comment[kk]=Іздеуді тездету үшін Strigi жергілікті файлдарды индекстеуін аяқтады
 Comment[km]=បានបញ្ចប់​ក​ការ​បង្កើត​លិបិក្រម​របស់ Strigi នៃ​ឯកសារ​មូលដ្ឋាន​សម្រាប់​ការ​ស្វែងរក​ផ្ទៃតុ​រហ័ស
--- runtime/kstyles/oxygen/oxygen.cpp	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/kstyles/oxygen/oxygen.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -3034,6 +3034,16 @@
 {
     switch(type)
     {
+        case CT_GroupBox:
+        {
+            // adjust groupbox width to bold label font
+            if (const QStyleOptionGroupBox* gbOpt = qstyleoption_cast<const QStyleOptionGroupBox*>(option)) {
+                QSize size = KStyle::sizeFromContents(type, option, contentsSize, widget);
+                int labelWidth = subControlRect(CC_GroupBox, gbOpt, SC_GroupBoxLabel, widget).width();
+                size.setWidth(qMax(size.width(), labelWidth));
+                return size;
+            }
+        }
         case CT_ToolButton:
         {
             QSize size = contentsSize;
@@ -3154,12 +3164,12 @@
                         cr = subElementRect(SE_CheckBoxIndicator, option, widget);
                         QRect gcr((gbOpt->rect.width() - tw -cr.width())/2 , (h-cr.height())/2+r.y(), cr.width(), cr.height());
                         if(subControl == SC_GroupBoxCheckBox)
-			{
-			    if (!isFlat)
-				return visualRect(option->direction, option->rect, gcr);
-			    else
-				return visualRect(option->direction, option->rect, QRect(0,0,cr.width(),cr.height()));
-			}
+                        {
+                            if (!isFlat)
+                                return visualRect(option->direction, option->rect, gcr);
+                            else
+                                return visualRect(option->direction, option->rect, QRect(0,0,cr.width(),cr.height()));
+                        }
                     }
 
                     // left align labels in flat group boxes, center align labels in framed group boxes
--- runtime/l10n/ae/entry.desktop	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/l10n/ae/entry.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -60,7 +60,7 @@
 Name[pt]=Emiratos Árabes Unidos
 Name[pt_BR]=Emirados Árabes Unidos
 Name[ro]=Emiratele Arabe Unite
-Name[ru]=Объединенные Арабские Эмираты
+Name[ru]=Объединённые Арабские Эмираты
 Name[se]=Ovttastuvvon arábalaš emiráhtat
 Name[si]=එක්සත් අරාඹි එමීර් රාජ්‍ය
 Name[sk]=Spojené arabské emiráty
--- runtime/l10n/au/entry.desktop	(.../tags/KDE/4.2.1/kdebase)	(wersja 938088)
+++ runtime/l10n/au/entry.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 938088)
@@ -25,7 +25,7 @@
 Name[gu]=ઓસ્ટ્રેલિઆ
 Name[he]=אוסטרליה
 Name[hi]=आस्ट्रेलिया
-Name[hne]=आस्ट्रेले
+Name[hne]=आस्ट्रेलिया
 Name[hr]=Australija
 Name[hsb]=Awstralska
 Name[hu]=Ausztrália
