--- runtime/cmake/modules/FindPulseAudio.cmake	(.../tags/KDE/4.2.1/kdebase)	(wersja 936422)
+++ runtime/cmake/modules/FindPulseAudio.cmake	(.../branches/KDE/4.2/kdebase)	(wersja 936422)
@@ -30,8 +30,8 @@
 
 FIND_PATH(PULSEAUDIO_INCLUDE_DIR pulse/pulseaudio.h
    HINTS
-   ${PC_PULSEADUIO_INCLUDEDIR}
-   ${PC_PULSEADUIO_INCLUDE_DIRS}
+   ${PC_PULSEAUDIO_INCLUDEDIR}
+   ${PC_PULSEAUDIO_INCLUDE_DIRS}
    )
 
 FIND_LIBRARY(PULSEAUDIO_LIBRARY NAMES pulse libpulse 
--- runtime/knotify/kde.notifyrc	(.../tags/KDE/4.2.1/kdebase)	(wersja 936422)
+++ runtime/knotify/kde.notifyrc	(.../branches/KDE/4.2/kdebase)	(wersja 936422)
@@ -752,7 +752,7 @@
 Comment[pt]=Ocorreu um erro sério que fez com que o programa saísse
 Comment[pt_BR]=Houve um erro sério, que fez com que o programa fosse fechado
 Comment[ro]=A avut loc o eroare gravă ce a determinat ieșirea programului
-Comment[ru]=Произошла серьезная ошибка, приведшая к завершению работы программы
+Comment[ru]=Произошла серьёзная ошибка, приведшая к завершению работы программы
 Comment[se]=Duođalaš meattáhus bissehii prográmma
 Comment[sk]=Stala sa vážna chyba, ktorá zapríčinila ukončenie programu
 Comment[sl]=Prišlo je do resne napake, zaradi katere se je program končal
@@ -1181,7 +1181,7 @@
 Comment[pt]=Ocorreu um erro muito sério, que fez pelo menos com que o programa saísse
 Comment[pt_BR]=Um erro muito sério aconteceu, que causou (no mínimo) o fechamento do programa
 Comment[ro]=A intervenit o eroare foarte gravă, cauzînd cel puțin ieșirea programului
-Comment[ru]=Произошла очень серьезная ошибка, как минимум, вызвавшая завершение работы программы
+Comment[ru]=Произошла очень серьёзная ошибка, как минимум, вызвавшая завершение работы программы
 Comment[sk]=Stala sa veľmi vážna chyba, ktorá zapríčinila minimálne ukončenie programu
 Comment[sl]=Prišlo je do zelo resne napake, zaradi katere se je program končal
 Comment[sr]=Дошло је до врло озбиљне грешке, због које је бар програм обустављен
--- runtime/desktoptheme/metadata.desktop	(.../tags/KDE/4.2.1/kdebase)	(wersja 936422)
+++ runtime/desktoptheme/metadata.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 936422)
@@ -50,7 +50,7 @@
 Comment[gu]=થીમ ઓક્સિજન શૈલીમાં બનાવાઇ છે
 Comment[he]=ערכת נושא שמשתמשת בסגנון Oxygen
 Comment[hi]=प्रसंग को ऑक्सीजन शैली में किया गया है
-Comment[hne]=प्रसंग ल ओक्सीजन सैली मं कर दे गे हे
+Comment[hne]=प्रसंग ल आक्सीजन सैली मं कर दे गे हे
 Comment[hu]=Oxygen-stílusú téma
 Comment[is]=Þema sem gert er í 'Oxygen' stíl
 Comment[it]=Tema in stile Oxygen
--- runtime/khelpcenter/kcmhelpcenter.desktop	(.../tags/KDE/4.2.1/kdebase)	(wersja 936422)
+++ runtime/khelpcenter/kcmhelpcenter.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 936422)
@@ -120,7 +120,7 @@
 Comment[gu]=મદદ કેન્દ્ર શોધ અનુક્રમણિકા રૂપરેખાંકન અને બનાવટ
 Comment[he]=שינוי הגדרות ויצירת אינדקס החיפוש של מרכז העזרה
 Comment[hi]=मदद केंद्र खोज तालिका कॉन्फ़िगरेशन तथा ज़ेनरेशन
-Comment[hne]=मदद केंद्र खोज तालिका कान्फिगरेसन अउ जेनरेसन
+Comment[hne]=मदद केंद्र खोज टेबल कान्फिगरेसन अउ जेनरेसन
 Comment[hr]=Konfiguriranje i generiranje indeksa za pretraživanje pomoći
 Comment[hsb]=Připrawjenje a stworjenje pytanskeho indeksa za pomhanišćo
 Comment[hu]=A dokumentáció keresési indexének beállításai
--- runtime/kioslave/trash/trash.protocol	(.../tags/KDE/4.2.1/kdebase)	(wersja 936422)
+++ runtime/kioslave/trash/trash.protocol	(.../branches/KDE/4.2/kdebase)	(wersja 936422)
@@ -49,7 +49,7 @@
 ExtraNames[gu]=મૂળભૂત પાથ,દૂર કરવાની તારીખ
 ExtraNames[he]=נתיב מקורי,תאריך מחיקה
 ExtraNames[hi]=मूल पथ, मिटाने की तिथि
-ExtraNames[hne]=मूल पथ, मेटारव के तारीक
+ExtraNames[hne]=मूल पथ, मेटाव के तारीक
 ExtraNames[hr]=Izvorna putanja,Datum brisanja
 ExtraNames[hsb]=Originalny puć, datum zničenja
 ExtraNames[hu]=Eredeti elérési út,Törlési dátum
--- runtime/l10n/ae/entry.desktop	(.../tags/KDE/4.2.1/kdebase)	(wersja 936422)
+++ runtime/l10n/ae/entry.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 936422)
@@ -60,7 +60,7 @@
 Name[pt]=Emiratos Árabes Unidos
 Name[pt_BR]=Emirados Árabes Unidos
 Name[ro]=Emiratele Arabe Unite
-Name[ru]=Объединенные Арабские Эмираты
+Name[ru]=Объединённые Арабские Эмираты
 Name[se]=Ovttastuvvon arábalaš emiráhtat
 Name[si]=එක්සත් අරාඹි එමීර් රාජ්‍ය
 Name[sk]=Spojené arabské emiráty
--- runtime/l10n/au/entry.desktop	(.../tags/KDE/4.2.1/kdebase)	(wersja 936422)
+++ runtime/l10n/au/entry.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 936422)
@@ -25,7 +25,7 @@
 Name[gu]=ઓસ્ટ્રેલિઆ
 Name[he]=אוסטרליה
 Name[hi]=आस्ट्रेलिया
-Name[hne]=आस्ट्रेले
+Name[hne]=आस्ट्रेलिया
 Name[hr]=Australija
 Name[hsb]=Awstralska
 Name[hu]=Ausztrália
