--- kdebase-runtime-4.6.4/kioslave/nfs/CMakeLists.txt~	2011-02-25 22:54:12.000000000 +0100
+++ kdebase-runtime-4.6.4/kioslave/nfs/CMakeLists.txt	2011-06-12 15:56:39.101708948 +0200
@@ -4,7 +4,7 @@
 kde4_add_plugin(kio_nfs ${kio_nfs_PART_SRCS})
 
 
-target_link_libraries(kio_nfs   ${KDE4_KIO_LIBS})
+target_link_libraries(kio_nfs   ${KDE4_KIO_LIBS} tirpc)
 
 install(TARGETS kio_nfs  DESTINATION ${PLUGIN_INSTALL_DIR} )
 
--- kde-runtime-4.14.3/kioslave/CMakeLists.txt.orig	2020-09-04 17:38:19.378728096 +0200
+++ kde-runtime-4.14.3/kioslave/CMakeLists.txt	2020-09-04 21:16:53.496620249 +0200
@@ -50,6 +50,7 @@
    add_subdirectory( floppy )
    add_subdirectory( finger )
    add_subdirectory( man )
+   set(CMAKE_REQUIRED_FLAGS "-I/usr/include/tirpc")
    check_include_files(rpc/rpc.h HAVE_RPC_RPC_H)
    add_feature_info("NFS kioslave" HAVE_RPC_RPC_H "The RPC library is needed to build the NFS kioslave")
    if(HAVE_RPC_RPC_H)
