--- kde-runtime-4.7.4/activitymanager/CMakeLists.txt.orig	2011-12-02 00:25:05.000000000 +0100
+++ kde-runtime-4.7.4/activitymanager/CMakeLists.txt	2011-12-30 02:27:41.851186489 +0100
@@ -25,12 +25,12 @@
 macro_optional_find_package(QZeitgeist)
 
 if (QZEITGEIST_INCLUDE_DIR)
-    if (QZEITGEIST_VERSION VERSION_LESS 0.8.0)
+    if (QZEITGEIST_VERSION VERSION_LESS 0.9.0)
         set(QZeitgeist_FOUND TRUE)
-    else (QZEITGEIST_VERSION VERSION_LESS 0.8.0)
-        message(STATUS "QZeitgeist 0.8.0 and greater are not supported")
+    else (QZEITGEIST_VERSION VERSION_LESS 0.9.0)
+        message(STATUS "QZeitgeist 0.9.0 and greater are not supported")
         set(QZeitgeist_FOUND FALSE)
-    endif (QZEITGEIST_VERSION VERSION_LESS 0.8.0)
+    endif (QZEITGEIST_VERSION VERSION_LESS 0.9.0)
 endif(QZEITGEIST_INCLUDE_DIR)
 
 macro_log_feature(
--- kde-runtime-4.7.4/activitymanager/ZeitgeistEventBackend.cpp.orig	2011-06-22 16:30:08.000000000 +0200
+++ kde-runtime-4.7.4/activitymanager/ZeitgeistEventBackend.cpp	2011-12-30 02:27:41.853186489 +0100
@@ -21,31 +21,31 @@
 
 #ifndef HAVE_QZEITGEIST
 #ifdef __GNUC__
-    #warning "No QtZeitgeist, disabling desktop events processing"
+    #warning "No QZeitgeist, disabling desktop events processing"
 #endif
 
 #else // HAVE_QZEITGEIST
 
 #include "ZeitgeistEventBackend.h"
 
-#include <QtZeitgeist/QtZeitgeist>
-#include <QtZeitgeist/Interpretation>
-#include <QtZeitgeist/Manifestation>
+#include <QZeitgeist/QZeitgeist>
+#include <QZeitgeist/Interpretation>
+#include <QZeitgeist/Manifestation>
 
 static QString eventInterpretation(Event::Type type)
 {
     switch (type) {
         case Event::Accessed:
-            return QtZeitgeist::Interpretation::Event::ZGAccessEvent;
+            return QZeitgeist::Interpretation::Event::ZGAccessEvent;
 
         case Event::Opened:
-            return QtZeitgeist::Interpretation::Event::ZGAccessEvent;
+            return QZeitgeist::Interpretation::Event::ZGAccessEvent;
 
         case Event::Modified:
-            return QtZeitgeist::Interpretation::Event::ZGModifyEvent;
+            return QZeitgeist::Interpretation::Event::ZGModifyEvent;
 
         case Event::Closed:
-            return QtZeitgeist::Interpretation::Event::ZGLeaveEvent;
+            return QZeitgeist::Interpretation::Event::ZGLeaveEvent;
 
     }
 
@@ -57,23 +57,23 @@
 {
     switch (reason) {
         case Event::User:
-            return QtZeitgeist::Manifestation::Event::ZGUserActivity;
+            return QZeitgeist::Manifestation::Event::ZGUserActivity;
 
         case Event::Heuristic:
-            return QtZeitgeist::Manifestation::Event::ZGHeuristicActivity;
+            return QZeitgeist::Manifestation::Event::ZGHeuristicActivity;
 
         case Event::Scheduled:
-            return QtZeitgeist::Manifestation::Event::ZGScheduledActivity;
+            return QZeitgeist::Manifestation::Event::ZGScheduledActivity;
 
         case Event::System:
-            return QtZeitgeist::Manifestation::Event::ZGSystemNotification;
+            return QZeitgeist::Manifestation::Event::ZGSystemNotification;
 
         case Event::World:
-            return QtZeitgeist::Manifestation::Event::ZGWorldActivity;
+            return QZeitgeist::Manifestation::Event::ZGWorldActivity;
     }
 
     // shut up GCC
-    return QtZeitgeist::Manifestation::Event::ZGUserActivity;
+    return QZeitgeist::Manifestation::Event::ZGUserActivity;
 }
 
 static QString applicationUri(const QString & application)
@@ -84,7 +84,7 @@
 
 ZeitgeistEventBackend::ZeitgeistEventBackend()
 {
-    QtZeitgeist::init();
+    
 }
 
 void ZeitgeistEventBackend::addEvents(const EventList & events)
