--- runtime/knotify/kde.notifyrc	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/knotify/kde.notifyrc	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -13,6 +13,7 @@
 Comment[en_GB]=KDE Workspace
 Comment[es]=Espacio de trabajo KDE
 Comment[et]=KDE töötsoon
+Comment[eu]=KDE langunea
 Comment[fi]=KDE-työtila
 Comment[fr]=Espace de travail de KDE
 Comment[fy]=KDE wurkromte
--- runtime/knotify/hardwarenotifications.notifyrc	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/knotify/hardwarenotifications.notifyrc	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -11,6 +11,7 @@
 Name[en_GB]=Hardware notifications
 Name[es]=Notificaciones de hardware
 Name[et]=Riistvara märguanded
+Name[eu]=Hardware jakinarazpenak
 Name[fi]=Laitteistohuomautukset
 Name[fr]=Notifications du matériel
 Name[fy]=Systeem notifikaasjes
@@ -61,6 +62,7 @@
 Comment[en_GB]=Notifications triggered by solid
 Comment[es]=Notificaciones activadas por solid
 Comment[et]=Solidi käivitatud märguanded
+Comment[eu]=Solid-ek eragindako jakinarazpenak
 Comment[fi]=Solidin laukaisemat huomautukset
 Comment[fr]=Notifications déclenchées par Solid
 Comment[fy]=Meldingen oanstjoerd troch solid
@@ -111,6 +113,7 @@
 Name[en_GB]=Mount or unmount error
 Name[es]=Error al montar o desmontar
 Name[et]=Ühendamise või lahutamise viga
+Name[eu]=Muntaketa edo desmuntaketa akatsa
 Name[fi]=Liittämisvirhe
 Name[fr]=Erreur de montage ou de démontage
 Name[fy]=Keppel of ûntkeppel flater
@@ -159,6 +162,7 @@
 Comment[en_GB]=There has been a problem mounting or unmounting a device
 Comment[es]=Ha ocurrido un error al montar o desmontar un dispositivo
 Comment[et]=Seadme ühendamisel või lahutamisel tekkis probleem
+Comment[eu]=Akats bat gertatu da gailu bat muntatu edo desmuntatzen
 Comment[fi]=Laitteen liittämisessä tai liitoksen poistamisessa oli ongelma.
 Comment[fr]=Il y a eu un problème lors du montage ou du démontage d'un périphérique
 Comment[fy]=Der wie swierrigens by it oankeppeljen of ûntkeppeljen fan in apparaat
--- runtime/solid-device-automounter/kded/device_automounter.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/solid-device-automounter/kded/device_automounter.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -70,6 +70,7 @@
 Comment[en_GB]=Automatically mounts devices as needed
 Comment[es]=Monta automáticamente dispositivos cuando se necesitan
 Comment[et]=Seadmete automaatne ühendamine vajaduse korral
+Comment[eu]=Gailuak automatikoki muntatzen ditu beharrezko denean
 Comment[fi]=Liittää laitteet automaattisesti tarpeen mukaan
 Comment[fr]=Monte automatiquement les périphériques à la demande
 Comment[fy]=Keppelt automatysk apparaten oan as it nedich is
--- runtime/kcontrol/knotify/kcmnotify.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kcontrol/knotify/kcmnotify.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -24,6 +24,7 @@
 Name[en_GB]=Manage Notifications
 Name[es]=Gestionar notificaciones
 Name[et]=Märguannete haldamine
+Name[eu]=Jakinarazpenak kudeatu
 Name[fi]=Huomautusten asetukset
 Name[fr]=Gérer les notifications
 Name[fy]=Systeemoankundigingen beheare
--- runtime/phonon/platform_kde/phonon.notifyrc	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/phonon/platform_kde/phonon.notifyrc	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -12,6 +12,7 @@
 Comment[en_GB]=Multimedia System
 Comment[es]=Sistema multimedia
 Comment[et]=Multimeediasüsteem
+Comment[eu]=Multimedia sistema
 Comment[fi]=Multimediajärjestelmä
 Comment[fr]=Système multimédia
 Comment[fy]=Multymedia systeem
--- runtime/phonon/kcm/kcm_phonon.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/phonon/kcm/kcm_phonon.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -22,6 +22,7 @@
 Name[en_GB]=Phonon
 Name[es]=Phonon
 Name[et]=Phonon
+Name[eu]=Phonon
 Name[fi]=Phonon
 Name[fr]=Phonon
 Name[fy]=Phonon
--- runtime/phonon/kded-module/phononserver.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/phonon/kded-module/phononserver.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -17,6 +17,7 @@
 Name[en_GB]=Sound Policy
 Name[es]=Política de sonido
 Name[et]=Helireeglid
+Name[eu]=Soinu politika
 Name[fi]=Äänikäytäntö
 Name[fr]=Politique des sons
 Name[fy]=Lûd belied
@@ -64,6 +65,7 @@
 Comment[en_GB]=Provides sound system policy to applications
 Comment[es]=Proporciona las políticas de sonido del sistema a las aplicaciones
 Comment[et]=Helisüsteemi reeglite edastamine rakendustele
+Comment[eu]=Aplikazioei soinu sistemaren politika hornitzen die
 Comment[fi]=Tarjoaa äänijärjestelmäkäytännön sovelluksille
 Comment[fr]=Fournis une politique de système sonore aux applications
 Comment[fy]=Jout lûd systeem belied oan applikaasjes
--- runtime/khelpcenter/plugins/onlinehelp.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/khelpcenter/plugins/onlinehelp.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -11,6 +11,7 @@
 Name[en_GB]=Online Help
 Name[es]=Ayuda en línea
 Name[et]=Abi
+Name[eu]=Lerroko Laguntza
 Name[fi]=Online-ohje
 Name[fr]=Aide en ligne
 Name[fy]=Online help
--- runtime/doc/kioslave/CMakeLists.txt	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/doc/kioslave/CMakeLists.txt	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -15,3 +15,4 @@
 endif(NOT WIN32)
 add_subdirectory(tar)
 add_subdirectory(thumbnail)
+add_subdirectory(xz)
--- runtime/kwalletd/kwalletd.notifyrc	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kwalletd/kwalletd.notifyrc	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -13,6 +13,7 @@
 Comment[en_GB]=Wallet
 Comment[es]=Cartera
 Comment[et]=Turvalaegas
+Comment[eu]=Zorroa
 Comment[fi]=Lompakko
 Comment[fr]=Portefeuille
 Comment[fy]=Slúf
--- runtime/ktimezoned/ktimezoned.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/ktimezoned/ktimezoned.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -20,6 +20,7 @@
 Name[en_GB]=Time Zone
 Name[es]=Zona horaria
 Name[et]=Ajavöönd
+Name[eu]=Ordu gunea
 Name[fi]=Aikavyöhyke
 Name[fr]=Zone horaire
 Name[fy]=Tiidsône
@@ -73,6 +74,7 @@
 Comment[en_GB]=Provides the system's time zone to applications
 Comment[es]=Proporciona la zona horaria del sistema a las aplicaciones
 Comment[et]=Süsteemi ajavööndi edastamine rakendustele
+Comment[eu]=Aplikazioei sistemaren ordu gunea hornitzen die
 Comment[fi]=Tarjoaa järjestelmän aikavyöhyketiedon ohjelmille
 Comment[fr]=Fourni la zone horaire du système aux applications
 Comment[fy]=Jout de tiidsône fan it systeem troch oan applikaasjes
--- runtime/kioslave/archive/zip.protocol	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kioslave/archive/zip.protocol	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -7,6 +7,6 @@
 listing=Name,Type,Size,Date,Access,Owner,Group,Link
 reading=true
 source=true
-DocPath=kioslave/zip.html
+#DocPath=kioslave/zip/index.html
 Icon=package-x-generic
 Class=:local
--- runtime/kioslave/thumbnail/imagethumbnail.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kioslave/thumbnail/imagethumbnail.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -12,6 +12,7 @@
 Name[en_GB]=Images (GIF, PNG, BMP, ...)
 Name[es]=Imágenes (GIF, PNG, BMP...)
 Name[et]=Pildid (GIF, PNG, BMP...)
+Name[eu]=Irudiak (GIF, PNG, BMP, ...)
 Name[fi]=Kuvat (GIF, PNG, BMP, ...)
 Name[fr]=Images (GIF, PNG, BMP, ...)
 Name[fy]=Ofbyldings (GIF, PNG, BMP, ...)
--- runtime/kioslave/filter/xz.protocol	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kioslave/filter/xz.protocol	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -7,5 +7,5 @@
 output=stream
 reading=true
 source=false
-DocPath=kioslave/xz.html
+DocPath=kioslave/xz/index.html
 Icon=package-x-generic
--- runtime/kioslave/filter/lzma.protocol	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kioslave/filter/lzma.protocol	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -7,5 +7,5 @@
 output=stream
 reading=true
 source=false
-DocPath=kioslave/xz.html
+DocPath=kioslave/xz/index.html
 Icon=package-x-generic
--- runtime/kioslave/network/network/builder/upnp/upnpnetworkbuilder.cpp	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kioslave/network/network/builder/upnp/upnpnetworkbuilder.cpp	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -36,6 +36,7 @@
 #include <QtDBus/QDBusConnection>
 #include <QtDBus/QDBusInterface>
 #include <QtDBus/QDBusPendingCallWatcher>
+#include <QtCore/QTimer>
 #include <QtCore/QStringList>
 
 #include <KDebug>
@@ -60,6 +61,11 @@
 
 void UpnpNetworkBuilder::start()
 {
+    QTimer::singleShot(0, this, SLOT(startBrowse()));
+}
+
+void UpnpNetworkBuilder::startBrowse()
+{
     qDBusRegisterMetaType<DeviceTypeMap>();
     qDBusRegisterMetaType<Cagibi::Device>();
 
--- runtime/kioslave/network/network/builder/upnp/upnpnetworkbuilder.h	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kioslave/network/network/builder/upnp/upnpnetworkbuilder.h	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -63,6 +63,8 @@
     void removeUPnPDevices( const QList<Cagibi::Device>& devices );
 
   private Q_SLOTS:
+    void startBrowse();
+
     void onDevicesAdded( const DeviceTypeMap& deviceTypeMap );
     void onDevicesRemoved( const DeviceTypeMap& deviceTypeMap );
     void onAddedDeviceDetails( const Cagibi::Device& device );
--- runtime/kioslave/remote/kdedmodule/remotedirnotify.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kioslave/remote/kdedmodule/remotedirnotify.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -11,6 +11,7 @@
 Name[en_GB]=Remote URL Change Notifier
 Name[es]=Notificador de cambios en URL remoto
 Name[et]=Mujalasuva URL-i muutumise teadustaja
+Name[eu]=Urruneko URL aldaketa jakinarazlea
 Name[fi]=Etäverkko-osoitteiden muutosten huomautin
 Name[fr]=Notifieur de changement d'URL distante
 Name[fy]=Eksterne URL-adres feroarings melder
@@ -58,6 +59,7 @@
 Comment[en_GB]=Provides change notification for network folders
 Comment[es]=Proporciona notificaciones visuales para carpetas de red
 Comment[et]=Võrgukataloogide muutuste märguanded
+Comment[eu]=Sareko karpeten aldaketa jakinarazpenak hornitzen ditu
 Comment[fi]=Tarjoaa huomautuksia verkkokansioiden muutoksista
 Comment[fr]=Fournis les notifications de modification pour les dossiers réseaux
 Comment[fy]=Ferskaft feroaringsmelding foar netwurkmappen
--- runtime/kioslave/desktop/desktopnotifier.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kioslave/desktop/desktopnotifier.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -12,6 +12,7 @@
 Name[en_GB]=Directory Watcher
 Name[es]=Observador de carpetas
 Name[et]=Kataloogijälgija
+Name[eu]=Karpeta begiralea
 Name[fi]=Kansion tarkkailija
 Name[fr]=Surveillance des dossiers
 Name[fy]=Triemtafel sjogger
@@ -72,6 +73,7 @@
 Comment[en_GB]=Monitors directories for changes
 Comment[es]=Monitoriza cambios en carpetas
 Comment[et]=Kataloogide muutuste jälgimine
+Comment[eu]=Direktorioak zelatatzen ditu aldaketa bila
 Comment[fi]=Tarkkailee muutoksia kansioissa
 Comment[fr]=Surveilles les modifications des dossiers
 Comment[fy]=Hâld triemtafel feroarings yn de gaten
--- runtime/nepomuk/services/strigi/nepomukstrigiservice.notifyrc	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/nepomuk/services/strigi/nepomukstrigiservice.notifyrc	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -12,6 +12,7 @@
 Comment[en_GB]=Desktop Search
 Comment[es]=Búsqueda de escritorio
 Comment[et]=Töölauaotsing
+Comment[eu]=Mahaigaineko bilaketa
 Comment[fi]=Työpöytähaku
 Comment[fr]=Recherche sur le bureau
 Comment[fy]=Buroblêd sykaksje
--- runtime/nepomuk/services/activities/nepomukactivitiesservice.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/nepomuk/services/activities/nepomukactivitiesservice.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -15,6 +15,7 @@
 Name[en_GB]=Nepomuk Activities Service
 Name[es]=Servicio de actividades de Nepomuk
 Name[et]=Nepomuki tegevuste teenus
+Name[eu]=Nepomuk jarduera zerbitzua
 Name[fr]=Service d'agencements de Nepomuk
 Name[fy]=Nepomuk aktiviteits tsjinst
 Name[ga]=Seirbhís Ghníomhaíochta Nepomuk
--- runtime/nepomuk/services/storage/nepomukstorage.notifyrc	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/nepomuk/services/storage/nepomukstorage.notifyrc	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -12,6 +12,7 @@
 Name[en_GB]=Semantic Data Storage
 Name[es]=Almacenamiento de datos semánticos
 Name[et]=Semantiline andmesalvesti
+Name[eu]=Datu semantikoen biltegiratzea
 Name[fi]=Semanttinen tietovarasto
 Name[fr]=Stockage de données sémantique
 Name[fy]=Semantic Gegevens opslach
@@ -61,6 +62,7 @@
 Comment[en_GB]=Semantic Desktop
 Comment[es]=Escritorio semántico
 Comment[et]=Semantiline töölaud
+Comment[eu]=Mahaigain semantikoa
 Comment[fi]=Semanttinen työpöytä
 Comment[fr]=Bureau sémantique
 Comment[fy]=Semantic buroblêd
--- runtime/solidautoeject/solidautoeject.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/solidautoeject/solidautoeject.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -15,6 +15,7 @@
 Name[en_GB]=Drive Ejector
 Name[es]=Eyector de unidad
 Name[et]=Plaadiväljasti
+Name[eu]=Unitate egozlea
 Name[fi]=Aseman poistaja
 Name[fr]=Éjecteur de disque
 Name[fy]=Stasjons Utsmiter
@@ -62,6 +63,7 @@
 Comment[en_GB]=Automatically releases drives when their eject button is pushed
 Comment[es]=Libera automáticamente las unidades cuando se pulsa el botón de expulsión
 Comment[et]=Võimaldab plaatidel automaatselt väljuda, kui vajutatakse väljastamisnuppu
+Comment[eu]=Unitateak automatikoki askatzen ditu beraien egozteko botoia sakatzen denean
 Comment[fi]=Vapauttaa asemat automaattisesti kun niiden poistopainiketta painetaan
 Comment[fr]=Libère automatiquement les disques quand leur bouton d'éjection est pressé
 Comment[fy]=Sil automatysk stasjons útsmite as der op de útsmyt knop drukt wurdt
--- runtime/activitymanager/activitymanager.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/activitymanager/activitymanager.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -21,6 +21,7 @@
 Name[en_GB]=Activity Manager
 Name[es]=Gestor de actividades
 Name[et]=Tegevuste haldur
+Name[eu]=Jarduera kudeatzailea
 Name[fi]=Aktiviteettienhallinta
 Name[fr]=Gestionnaire d'agencements
 Name[fy]=Aktiviteits behearder
--- runtime/kurifilter-plugins/ikws/searchproviders/facebook.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/facebook.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -15,6 +15,7 @@
 Name[en_GB]=Facebook
 Name[es]=Facebook
 Name[et]=Facebook
+Name[eu]=Facebook
 Name[fi]=Facebook
 Name[fr]=Facebook
 Name[fy]=Facebook
@@ -68,6 +69,7 @@
 Query[en_GB]=http://www.facebook.com/search/?q=\\{@}&init=quick
 Query[es]=http://www.facebook.com/search/?q=\\{@}&init=quick
 Query[et]=http://www.facebook.com/search/?q=\\{@}&init=quick
+Query[eu]=http://www.facebook.com/search/?q=\\{@}&init=quick
 Query[fi]=http://www.facebook.com/search/?q=\\{@}&init=quick
 Query[fr]=http://www.facebook.com/search/?q=\\{@}&init=quick
 Query[fy]=http://www.facebook.com/search/?q=\\{@}&init=quick
--- runtime/kurifilter-plugins/ikws/searchproviders/github.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/github.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -15,6 +15,7 @@
 Name[en_GB]=GitHub
 Name[es]=GitHub
 Name[et]=GitHub
+Name[eu]=GitHub
 Name[fi]=GitHub
 Name[fr]=GitHub
 Name[fy]=GitHub
@@ -68,6 +69,7 @@
 Query[en_GB]=http://github.com/search?q=\\{@}&x=0&y=0
 Query[es]=http://github.com/search?q=\\{@}&x=0&y=0
 Query[et]=http://github.com/search?q=\\{@}&x=0&y=0
+Query[eu]=http://github.com/search?q=\\{@}&x=0&y=0
 Query[fi]=http://github.com/search?q=\\{@}&x=0&y=0
 Query[fr]=http://github.com/search?q=\\{@}&x=0&y=0
 Query[fy]=http://github.com/search?q=\\{@}&x=0&y=0
--- runtime/kurifilter-plugins/ikws/searchproviders/qt.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/qt.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -101,6 +101,7 @@
 Query[en_GB]=http://doc.qt.nokia.com/latest/\\{@}.html
 Query[es]=http://doc.qt.nokia.com/latest/\\{@}.html
 Query[et]=http://doc.qt.nokia.com/latest/\\{@}.html
+Query[eu]=http://doc.qt.nokia.com/latest/\\{@}.html
 Query[fi]=http://doc.qt.nokia.com/latest/\\{@}.html
 Query[fr]=http://doc.qt.nokia.com/latest/\\{@}.html
 Query[fy]=http://doc.qt.nokia.com/latest/\\{@}.html
--- runtime/kurifilter-plugins/ikws/searchproviders/identica_people.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/identica_people.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -14,6 +14,7 @@
 Name[en_GB]=Identi.ca People
 Name[es]=Gente Identi.ca
 Name[et]=Identi.ca inimesed
+Name[eu]=Identi.ca Jendea
 Name[fr]=Personne Identi.ca
 Name[fy]=Identi.ca minsken
 Name[ga]=Daoine Identi.ca
@@ -63,6 +64,7 @@
 Query[en_GB]=http://identi.ca/search/people?q=\\{@}&search=Search
 Query[es]=http://identi.ca/search/people?q=\\{@}&search=Search
 Query[et]=http://identi.ca/search/people?q=\\{@}&search=Search
+Query[eu]=http://identi.ca/search/people?q=\\{@}&search=Search
 Query[fi]=http://identi.ca/search/people?q=\\{@}&search=Search
 Query[fr]=http://identi.ca/search/people?q=\\{@}&search=Search
 Query[fy]=http://identi.ca/search/people?q=\\{@}&search=Search
--- runtime/kurifilter-plugins/ikws/searchproviders/blip.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/blip.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -15,6 +15,7 @@
 Name[en_GB]=Blip.tv
 Name[es]=Blip.tv
 Name[et]=Blip.tv
+Name[eu]=Blip.tv
 Name[fi]=Blip.tv
 Name[fr]=Blip.tv
 Name[fy]=Blip.tv
@@ -68,6 +69,7 @@
 Query[en_GB]=http://blip.tv/search?q=\\{@}
 Query[es]=http://blip.tv/search?q=\\{@}
 Query[et]=http://blip.tv/search?q=\\{@}
+Query[eu]=http://blip.tv/search?q=\\{@}
 Query[fi]=http://blip.tv/search?q=\\{@}
 Query[fr]=http://blip.tv/search?q=\\{@}
 Query[fy]=http://blip.tv/search?q=\\{@}
--- runtime/kurifilter-plugins/ikws/searchproviders/beolingus.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/beolingus.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -13,6 +13,7 @@
 Name[en_GB]=Beolingus Online Dictionary
 Name[es]=Diccionario en línea Beolingus
 Name[et]=Beolinguse võrgusõnastik
+Name[eu]=Beolingus lerroko hiztegia
 Name[fi]=Beolingus-verkkosanakirja
 Name[fr]=Dictionnaire en ligne Beolingus
 Name[fy]=Beolingus Online wurdboek
@@ -67,6 +68,7 @@
 Query[en_GB]=http://dict.tu-chemnitz.de/dings.cgi?lang=de&service=deen&opterrors=0&optpro=0&query=\\{@}&iservice=&comment=&email
 Query[es]=http://dict.tu-chemnitz.de/dings.cgi?lang=de&service=deen&opterrors=0&optpro=0&query=\\{@}&iservice=&comment=&email
 Query[et]=http://dict.tu-chemnitz.de/dings.cgi?lang=de&service=deen&opterrors=0&optpro=0&query=\\{@}&iservice=&comment=&email
+Query[eu]=http://dict.tu-chemnitz.de/dings.cgi?lang=de&service=deen&opterrors=0&optpro=0&query=\\{@}&iservice=&comment=&email
 Query[fi]=http://dict.tu-chemnitz.de/dings.cgi?lang=de&service=deen&opterrors=0&optpro=0&query=\\{@}&iservice=&comment=&email
 Query[fr]=http://dict.tu-chemnitz.de/dings.cgi?lang=de&service=deen&opterrors=0&optpro=0&query=\\{@}&iservice=&comment=&email
 Query[fy]=http://dict.tu-chemnitz.de/dings.cgi?lang=de&service=deen&opterrors=0&optpro=0&query=\\{@}&iservice=&comment=&email
--- runtime/kurifilter-plugins/ikws/searchproviders/duckduckgo_shopping.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/duckduckgo_shopping.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -12,6 +12,7 @@
 Name[en_GB]=Duck Duck Go Shopping
 Name[es]=Duck Duck Go Compras
 Name[et]=Duck Duck Go Shopping
+Name[eu]=Duck Duck Go Erosketa
 Name[fr]=Duck Duck Go Shopping
 Name[fy]=Duck Duck Go Winkelje
 Name[ga]=Siopadóireacht Duck Duck Go
@@ -60,6 +61,7 @@
 Query[en_GB]=http://duckduckgo.com/?q=\\{@}&v=s
 Query[es]=http://duckduckgo.com/?q=\\{@}&v=s
 Query[et]=http://duckduckgo.com/?q=\\{@}&v=s
+Query[eu]=http://duckduckgo.com/?q=\\{@}&v=s
 Query[fi]=http://duckduckgo.com/?q=\\{@}&v=s
 Query[fr]=http://duckduckgo.com/?q=\\{@}&v=s
 Query[fy]=http://duckduckgo.com/?q=\\{@}&v=s
--- runtime/kurifilter-plugins/ikws/searchproviders/google_maps.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/google_maps.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -13,6 +13,7 @@
 Name[en_GB]=Google Maps
 Name[es]=Google Maps
 Name[et]=Google Maps
+Name[eu]=Google Mapak
 Name[fi]=Google Maps (kartat)
 Name[fr]=Google Maps
 Name[fy]=Google kaart
@@ -69,6 +70,7 @@
 Query[en_GB]=http://maps.google.com/maps?q=\\{@}&ie=UTF8&iwloc=addr
 Query[es]=http://maps.google.com/maps?q=\\{@}&ie=UTF8&iwloc=addr
 Query[et]=http://maps.google.com/maps?q=\\{@}&ie=UTF8&iwloc=addr
+Query[eu]=http://maps.google.com/maps?q=\\{@}&ie=UTF8&iwloc=addr
 Query[fi]=http://maps.google.com/maps?q=\\{@}&ie=UTF8&iwloc=addr
 Query[fr]=http://maps.google.com/maps?q=\\{@}&ie=UTF8&iwloc=addr
 Query[fy]=http://maps.google.com/maps?q=\\{@}&ie=UTF8&iwloc=addr
--- runtime/kurifilter-plugins/ikws/searchproviders/kde_forums.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/kde_forums.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -15,6 +15,7 @@
 Name[en_GB]=KDE Forums
 Name[es]=Foros de KDE
 Name[et]=KDE foorumid
+Name[eu]=Mandriva foroak
 Name[fi]=KDE:n foorumit
 Name[fr]=Forums de KDE
 Name[fy]=KDE Foarums
@@ -67,6 +68,7 @@
 Query[en_GB]=http://forum.kde.org/search.php?keywords=\\{@}&terms=all&author=&tags=&sv=0&sc=1&sf=all&sr=posts&sk=t&sd=d&st=0&ch=300&feed_type=NONE&feed_style=HTML&countlimit=100&t=0&sid=77c16bc0abb91ff1347597c69835d5bd&submit=Search
 Query[es]=http://forum.kde.org/search.php?keywords=\\{@}&terms=all&author=&tags=&sv=0&sc=1&sf=all&sr=posts&sk=t&sd=d&st=0&ch=300&feed_type=NONE&feed_style=HTML&countlimit=100&t=0&sid=77c16bc0abb91ff1347597c69835d5bd&submit=Search
 Query[et]=http://forum.kde.org/search.php?keywords=\\{@}&terms=all&author=&tags=&sv=0&sc=1&sf=all&sr=posts&sk=t&sd=d&st=0&ch=300&feed_type=NONE&feed_style=HTML&countlimit=100&t=0&sid=77c16bc0abb91ff1347597c69835d5bd&submit=Search
+Query[eu]=http://forum.kde.org/search.php?keywords=\\{@}&terms=all&author=&tags=&sv=0&sc=1&sf=all&sr=posts&sk=t&sd=d&st=0&ch=300&feed_type=NONE&feed_style=HTML&countlimit=100&t=0&sid=77c16bc0abb91ff1347597c69835d5bd&submit=Search
 Query[fi]=http://forum.kde.org/search.php?keywords=\\{@}&terms=all&author=&tags=&sv=0&sc=1&sf=all&sr=posts&sk=t&sd=d&st=0&ch=300&feed_type=NONE&feed_style=HTML&countlimit=100&t=0&sid=77c16bc0abb91ff1347597c69835d5bd&submit=Search
 Query[fr]=http://forum.kde.org/search.php?keywords=\\{@}&terms=all&author=&tags=&sv=0&sc=1&sf=all&sr=posts&sk=t&sd=d&st=0&ch=300&feed_type=NONE&feed_style=HTML&countlimit=100&t=0&sid=77c16bc0abb91ff1347597c69835d5bd&submit=Search
 Query[fy]=http://forum.kde.org/search.php?keywords=\\{@}&terms=all&author=&tags=&sv=0&sc=1&sf=all&sr=posts&sk=t&sd=d&st=0&ch=300&feed_type=NONE&feed_style=HTML&countlimit=100&t=0&sid=77c16bc0abb91ff1347597c69835d5bd&submit=Search
--- runtime/kurifilter-plugins/ikws/searchproviders/kde_techbase.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/kde_techbase.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -13,6 +13,7 @@
 Name[en_GB]=KDE TechBase
 Name[es]=KDE TechBase
 Name[et]=KDE TechBase
+Name[eu]=KDE TechBase
 Name[fi]=KDE TechBase
 Name[fr]=Centre technique de KDE
 Name[fy]=KDE TechBase
@@ -65,6 +66,7 @@
 Query[en_GB]=http://techbase.kde.org/Special:Search?search=\\{@}&fulltext=Search
 Query[es]=http://techbase.kde.org/Special:Search?search=\\{@}&fulltext=Search
 Query[et]=http://techbase.kde.org/Special:Search?search=\\{@}&fulltext=Search
+Query[eu]=http://techbase.kde.org/Special:Search?search=\\{@}&fulltext=Search
 Query[fi]=http://techbase.kde.org/Special:Search?search=\\{@}&fulltext=Search
 Query[fr]=http://techbase.kde.org/Special:Search?search=\\{@}&fulltext=Search
 Query[fy]=http://techbase.kde.org/Special:Search?search=\\{@}&fulltext=Search
--- runtime/kurifilter-plugins/ikws/searchproviders/ecosia.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/ecosia.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -12,6 +12,7 @@
 Name[en_GB]=Ecosia search engine
 Name[es]=Motor de búsqueda Ecosia
 Name[et]=Ecosia otsingumootor
+Name[eu]=Ecosia bilaketa motorea
 Name[fi]=Ecosia-hakukone
 Name[fr]=Moteur de recherche Ecosia
 Name[fy]=Ecosia sykmotor
@@ -68,6 +69,7 @@
 Query[en_GB]=http://ecosia.org/search.php?q=\\{@}&meta=normal
 Query[es]=http://ecosia.org/search.php?q=\\{@}&meta=normal
 Query[et]=http://ecosia.org/search.php?q=\\{@}&meta=normal
+Query[eu]=http://ecosia.org/search.php?q=\\{@}&meta=normal
 Query[fi]=http://ecosia.org/search.php?q=\\{@}&meta=normal
 Query[fr]=http://ecosia.org/search.php?q=\\{@}&meta=normal
 Query[fy]=http://ecosia.org/search.php?q=\\{@}&meta=normal
--- runtime/kurifilter-plugins/ikws/searchproviders/magnatune.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/magnatune.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -15,6 +15,7 @@
 Name[en_GB]=Magnatune
 Name[es]=Magnatune
 Name[et]=Magnatune
+Name[eu]=Magnatune
 Name[fi]=Magnatune
 Name[fr]=Magnatune
 Name[fy]=Magnatune
@@ -67,6 +68,7 @@
 Query[en_GB]=http://my.magnatune.com/search?w=\\{@}&t=m&x=0&y=0
 Query[es]=http://my.magnatune.com/search?w=\\{@}&t=m&x=0&y=0
 Query[et]=http://my.magnatune.com/search?w=\\{@}&t=m&x=0&y=0
+Query[eu]=http://my.magnatune.com/search?w=\\{@}&t=m&x=0&y=0
 Query[fi]=http://my.magnatune.com/search?w=\\{@}&t=m&x=0&y=0
 Query[fr]=http://my.magnatune.com/search?w=\\{@}&t=m&x=0&y=0
 Query[fy]=http://my.magnatune.com/search?w=\\{@}&t=m&x=0&y=0
--- runtime/kurifilter-plugins/ikws/searchproviders/flickrcc.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/flickrcc.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -14,6 +14,7 @@
 Name[en_GB]=Flickr Creative Commons
 Name[es]=Flickr Creative Commons
 Name[et]=Flickr Creative Commons
+Name[eu]=Flickr Creative Commons
 Name[fi]=Flickr Creative Commons
 Name[fr]=Flickr Creative Commons
 Name[fy]=Flickr Creative Commons
@@ -63,6 +64,7 @@
 Query[en_GB]=http://www.flickr.com/search/?q=\\{@}&l=cc&ct=0&mt=all&adv=1
 Query[es]=http://www.flickr.com/search/?q=\\{@}&l=cc&ct=0&mt=all&adv=1
 Query[et]=http://www.flickr.com/search/?q=\\{@}&l=cc&ct=0&mt=all&adv=1
+Query[eu]=http://www.flickr.com/search/?q=\\{@}&l=cc&ct=0&mt=all&adv=1
 Query[fi]=http://www.flickr.com/search/?q=\\{@}&l=cc&ct=0&mt=all&adv=1
 Query[fr]=http://www.flickr.com/search/?q=\\{@}&l=cc&ct=0&mt=all&adv=1
 Query[fy]=http://www.flickr.com/search/?q=\\{@}&l=cc&ct=0&mt=all&adv=1
--- runtime/kurifilter-plugins/ikws/searchproviders/yahoo_image.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/yahoo_image.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -15,6 +15,7 @@
 Name[en_GB]=Yahoo Images
 Name[es]=Yahoo Imágenes
 Name[et]=Yahoo pildid
+Name[eu]=Yahoo irudiak
 Name[fi]=Yahoon kuvahaku
 Name[fr]=Images Yahoo
 Name[fy]=Yahoo ôfbyldings
@@ -66,6 +67,7 @@
 Query[en_GB]=http://images.search.yahoo.com/search/images;_ylt=A0geuy0K6JZLsVkB4m1XNyoA?ei=UTF-8&p=\\{@}&y=0&fr2=tab-web&fr=yfp-t-701
 Query[es]=http://images.search.yahoo.com/search/images;_ylt=A0geuy0K6JZLsVkB4m1XNyoA?ei=UTF-8&p=\\{@}&y=0&fr2=tab-web&fr=yfp-t-701
 Query[et]=http://images.search.yahoo.com/search/images;_ylt=A0geuy0K6JZLsVkB4m1XNyoA?ei=UTF-8&p=\\{@}&y=0&fr2=tab-web&fr=yfp-t-701
+Query[eu]=http://images.search.yahoo.com/search/images;_ylt=A0geuy0K6JZLsVkB4m1XNyoA?ei=UTF-8&p=\\{@}&y=0&fr2=tab-web&fr=yfp-t-701
 Query[fi]=http://images.search.yahoo.com/search/images;_ylt=A0geuy0K6JZLsVkB4m1XNyoA?ei=UTF-8&p=\\{@}&y=0&fr2=tab-web&fr=yfp-t-701
 Query[fr]=http://images.search.yahoo.com/search/images;_ylt=A0geuy0K6JZLsVkB4m1XNyoA?ei=UTF-8&p=\\{@}&y=0&fr2=tab-web&fr=yfp-t-701
 Query[fy]=http://images.search.yahoo.com/search/images;_ylt=A0geuy0K6JZLsVkB4m1XNyoA?ei=UTF-8&p=\\{@}&y=0&fr2=tab-web&fr=yfp-t-701
--- runtime/kurifilter-plugins/ikws/searchproviders/yahoo_shopping.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/yahoo_shopping.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -15,6 +15,7 @@
 Name[en_GB]=Yahoo Shopping
 Name[es]=Yahoo Compras
 Name[et]=Yahoo Shopping
+Name[eu]=Yahoo Shopping
 Name[fi]=Yahoo Ostokset
 Name[fr]=Yahoo Shopping
 Name[fy]=Yahoo Winkelje
@@ -66,6 +67,7 @@
 Query[en_GB]=http://shopping.yahoo.com/search;_ylt=A0SO8ZgG65ZLyX8AsLX7w8QF;_ylu=X3oDMTBpcjIxMDllBHNlYwN0YWJzBHZ0aWQDVjEwOQ--?cop=mss&ei=UTF-8&p=\\{@}&fr2=tab-video&fr=yfp-t-701
 Query[es]=http://shopping.yahoo.com/search;_ylt=A0SO8ZgG65ZLyX8AsLX7w8QF;_ylu=X3oDMTBpcjIxMDllBHNlYwN0YWJzBHZ0aWQDVjEwOQ--?cop=mss&ei=UTF-8&p=\\{@}&fr2=tab-video&fr=yfp-t-701
 Query[et]=http://shopping.yahoo.com/search;_ylt=A0SO8ZgG65ZLyX8AsLX7w8QF;_ylu=X3oDMTBpcjIxMDllBHNlYwN0YWJzBHZ0aWQDVjEwOQ--?cop=mss&ei=UTF-8&p=\\{@}&fr2=tab-video&fr=yfp-t-701
+Query[eu]=http://shopping.yahoo.com/search;_ylt=A0SO8ZgG65ZLyX8AsLX7w8QF;_ylu=X3oDMTBpcjIxMDllBHNlYwN0YWJzBHZ0aWQDVjEwOQ--?cop=mss&ei=UTF-8&p=\\{@}&fr2=tab-video&fr=yfp-t-701
 Query[fi]=http://shopping.yahoo.com/search;_ylt=A0SO8ZgG65ZLyX8AsLX7w8QF;_ylu=X3oDMTBpcjIxMDllBHNlYwN0YWJzBHZ0aWQDVjEwOQ--?cop=mss&ei=UTF-8&p=\\{@}&fr2=tab-video&fr=yfp-t-701
 Query[fr]=http://shopping.yahoo.com/search;_ylt=A0SO8ZgG65ZLyX8AsLX7w8QF;_ylu=X3oDMTBpcjIxMDllBHNlYwN0YWJzBHZ0aWQDVjEwOQ--?cop=mss&ei=UTF-8&p=\\{@}&fr2=tab-video&fr=yfp-t-701
 Query[fy]=http://shopping.yahoo.com/search;_ylt=A0SO8ZgG65ZLyX8AsLX7w8QF;_ylu=X3oDMTBpcjIxMDllBHNlYwN0YWJzBHZ0aWQDVjEwOQ--?cop=mss&ei=UTF-8&p=\\{@}&fr2=tab-video&fr=yfp-t-701
--- runtime/kurifilter-plugins/ikws/searchproviders/kde_look.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/kde_look.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -14,6 +14,7 @@
 Name[en_GB]=KDE Look
 Name[es]=KDE Look
 Name[et]=KDE Look
+Name[eu]=KDE Look
 Name[fi]=KDE Look
 Name[fr]=KDE Look
 Name[fy]=KDE Uterlik
@@ -66,6 +67,7 @@
 Query[en_GB]=http://www.kde-look.org/content/search.php?search=Search&name=\\{@}
 Query[es]=http://www.kde-look.org/content/search.php?search=Search&name=\\{@}
 Query[et]=http://www.kde-look.org/content/search.php?search=Search&name=\\{@}
+Query[eu]=http://www.kde-look.org/content/search.php?search=Search&name=\\{@}
 Query[fi]=http://www.kde-look.org/content/search.php?search=Search&name=\\{@}
 Query[fr]=http://www.kde-look.org/content/search.php?search=Search&name=\\{@}
 Query[fy]=http://www.kde-look.org/content/search.php?search=Search&name=\\{@}
--- runtime/kurifilter-plugins/ikws/searchproviders/yahoo_local.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/yahoo_local.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -13,6 +13,7 @@
 Name[en_GB]=Yahoo Local
 Name[es]=Yahoo Local
 Name[et]=Yahoo Local
+Name[eu]=Yahoo Local
 Name[fi]=Yahoon paikallishaku
 Name[fr]=Yahoo Local
 Name[fy]=Yahoo lokaal
@@ -64,6 +65,7 @@
 Query[en_GB]=http://local.yahoo.com/results?stx=\\{@}&ei=UTF-8&fr=yfp-t-701&fr2=tab-img
 Query[es]=http://local.yahoo.com/results?stx=\\{@}&ei=UTF-8&fr=yfp-t-701&fr2=tab-img
 Query[et]=http://local.yahoo.com/results?stx=\\{@}&ei=UTF-8&fr=yfp-t-701&fr2=tab-img
+Query[eu]=http://local.yahoo.com/results?stx=\\{@}&ei=UTF-8&fr=yfp-t-701&fr2=tab-img
 Query[fi]=http://local.yahoo.com/results?stx=\\{@}&ei=UTF-8&fr=yfp-t-701&fr2=tab-img
 Query[fr]=http://local.yahoo.com/results?stx=\\{@}&ei=UTF-8&fr=yfp-t-701&fr2=tab-img
 Query[fy]=http://local.yahoo.com/results?stx=\\{@}&ei=UTF-8&fr=yfp-t-701&fr2=tab-img
--- runtime/kurifilter-plugins/ikws/searchproviders/youtube.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/youtube.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -13,6 +13,7 @@
 Name[en_GB]=YouTube
 Name[es]=YouTube
 Name[et]=YouTube
+Name[eu]=YouTube
 Name[fi]=YouTube
 Name[fr]=YouTube
 Name[fy]=YouTube
@@ -66,6 +67,7 @@
 Query[en_GB]=http://www.youtube.com/results?search_query=\\{@}&search_type=&aq=f
 Query[es]=http://www.youtube.com/results?search_query=\\{@}&search_type=&aq=f
 Query[et]=http://www.youtube.com/results?search_query=\\{@}&search_type=&aq=f
+Query[eu]=http://www.youtube.com/results?search_query=\\{@}&search_type=&aq=f
 Query[fi]=http://www.youtube.com/results?search_query=\\{@}&search_type=&aq=f
 Query[fr]=http://www.youtube.com/results?search_query=\\{@}&search_type=&aq=f
 Query[fy]=http://www.youtube.com/results?search_query=\\{@}&search_type=&aq=f
--- runtime/kurifilter-plugins/ikws/searchproviders/urbandictionary.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/urbandictionary.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -14,6 +14,7 @@
 Name[en_GB]=Urban Dictionary
 Name[es]=Urban Dictionary
 Name[et]=Urban Dictionary
+Name[eu]=Urban hiztegia
 Name[fi]=Urban Dictionary
 Name[fr]=Dictionnaire Urban
 Name[fy]=Urban Wurdboek
@@ -65,6 +66,7 @@
 Query[en_GB]=http://www.urbandictionary.com/define.php?term=\\{@}
 Query[es]=http://www.urbandictionary.com/define.php?term=\\{@}
 Query[et]=http://www.urbandictionary.com/define.php?term=\\{@}
+Query[eu]=http://www.urbandictionary.com/define.php?term=\\{@}
 Query[fi]=http://www.urbandictionary.com/define.php?term=\\{@}
 Query[fr]=http://www.urbandictionary.com/define.php?term=\\{@}
 Query[fy]=http://www.urbandictionary.com/define.php?term=\\{@}
--- runtime/kurifilter-plugins/ikws/searchproviders/amazon_mp3.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/amazon_mp3.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -15,6 +15,7 @@
 Name[en_GB]=Amazon MP3
 Name[es]=Amazon MP3
 Name[et]=Amazon MP3
+Name[eu]=Amazon MP3
 Name[fi]=Amazon MP3
 Name[fr]=Amazon MP3
 Name[fy]=Amazon MP3
@@ -68,6 +69,7 @@
 Query[en_GB]=http://www.amazon.co.uk/s/ref=nb_sb_noss?url=search-alias=digital-music&field-keywords=\\{@}&x=0&y=0
 Query[es]=http://www.amazon.com/s/ref=nb_sb_noss?url=search-alias=digital-music&field-keywords=\\{@}&x=0&y=0
 Query[et]=http://www.amazon.com/s/ref=nb_sb_noss?url=search-alias=digital-music&field-keywords=\\{@}&x=0&y=0
+Query[eu]=http://www.amazon.com/s/ref=nb_sb_noss?url=search-alias=digital-music&field-keywords=\\{@}&x=0&y=0
 Query[fi]=http://www.amazon.com/s/ref=nb_sb_noss?url=search-alias=digital-music&field-keywords=\\{@}&x=0&y=0
 Query[fr]=http://www.amazon.com/s/ref=nb_sb_noss?url=search-alias=digital-music&field-keywords=\\{@}&x=0&y=0
 Query[fy]=http://www.amazon.com/s/ref=nb_sb_noss?url=search-alias=digital-music&field-keywords=\\{@}&x=0&y=0
--- runtime/kurifilter-plugins/ikws/searchproviders/yahoo_video.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/yahoo_video.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -15,6 +15,7 @@
 Name[en_GB]=Yahoo Video
 Name[es]=Yahoo Vídeo
 Name[et]=Yahoo Video
+Name[eu]=Yahoo Video
 Name[fi]=Yahoo Video
 Name[fr]=Vidéo Yahoo
 Name[fy]=Yahoo fideo
@@ -65,6 +66,7 @@
 Query[en_GB]=http://video.search.yahoo.com/search/video;_ylt=A0WTb_3a6JZLSHwAK7.JzbkF?ei=UTF-8&p=\\{@}&y=0&fr2=tab-img&fr=yfp-t-701
 Query[es]=http://video.search.yahoo.com/search/video;_ylt=A0WTb_3a6JZLSHwAK7.JzbkF?ei=UTF-8&p=\\{@}&y=0&fr2=tab-img&fr=yfp-t-701
 Query[et]=http://video.search.yahoo.com/search/video;_ylt=A0WTb_3a6JZLSHwAK7.JzbkF?ei=UTF-8&p=\\{@}&y=0&fr2=tab-img&fr=yfp-t-701
+Query[eu]=http://video.search.yahoo.com/search/video;_ylt=A0WTb_3a6JZLSHwAK7.JzbkF?ei=UTF-8&p=\\{@}&y=0&fr2=tab-img&fr=yfp-t-701
 Query[fi]=http://video.search.yahoo.com/search/video;_ylt=A0WTb_3a6JZLSHwAK7.JzbkF?ei=UTF-8&p=\\{@}&y=0&fr2=tab-img&fr=yfp-t-701
 Query[fr]=http://video.search.yahoo.com/search/video;_ylt=A0WTb_3a6JZLSHwAK7.JzbkF?ei=UTF-8&p=\\{@}&y=0&fr2=tab-img&fr=yfp-t-701
 Query[fy]=http://video.search.yahoo.com/search/video;_ylt=A0WTb_3a6JZLSHwAK7.JzbkF?ei=UTF-8&p=\\{@}&y=0&fr2=tab-img&fr=yfp-t-701
--- runtime/kurifilter-plugins/ikws/searchproviders/google_code.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/google_code.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -15,6 +15,7 @@
 Name[en_GB]=Google Code
 Name[es]=Google Code
 Name[et]=Google Code
+Name[eu]=Google Code
 Name[fi]=Google Code
 Name[fr]=Google Code
 Name[fy]=Google koade
@@ -66,6 +67,7 @@
 Query[en_GB]=http://code.google.com/query/#q=\\{%2520}
 Query[es]=http://code.google.com/query/#q=\\{%2520}
 Query[et]=http://code.google.com/query/#q=\\{%2520}
+Query[eu]=http://code.google.com/query/#q=\\{%2520}
 Query[fi]=http://code.google.com/query/#q=\\{%2520}
 Query[fr]=http://code.google.com/query/#q=\\{%2520}
 Query[fy]=http://code.google.com/query/#q=\\{%2520}
--- runtime/kurifilter-plugins/ikws/searchproviders/flickr.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/flickr.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -15,6 +15,7 @@
 Name[en_GB]=Flickr
 Name[es]=Flickr
 Name[et]=Flickr
+Name[eu]=Flickr
 Name[fi]=Flickr
 Name[fr]=Flickr
 Name[fy]=Flickr
@@ -67,6 +68,7 @@
 Query[en_GB]=http://www.flickr.com/search/?q=\\{@}
 Query[es]=http://www.flickr.com/search/?q=\\{@}
 Query[et]=http://www.flickr.com/search/?q=\\{@}
+Query[eu]=http://www.flickr.com/search/?q=\\{@}
 Query[fi]=http://www.flickr.com/search/?q=\\{@}
 Query[fr]=http://www.flickr.com/search/?q=\\{@}
 Query[fy]=http://www.flickr.com/search/?q=\\{@}
--- runtime/kurifilter-plugins/ikws/searchproviders/imdb.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/imdb.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -102,6 +102,7 @@
 Query[en_GB]=http://imdb.com/find?s=all&q=\\{@}
 Query[es]=http://imdb.com/find?s=all&q=\\{@}
 Query[et]=http://imdb.com/find?s=all&q=\\{@}
+Query[eu]=http://imdb.com/find?s=all&q=\\{@}
 Query[fi]=http://imdb.com/find?s=all&q=\\{@}
 Query[fr]=http://imdb.com/find?s=all&q=\\{@}
 Query[fy]=http://imdb.com/find?s=all&q=\\{@}
--- runtime/kurifilter-plugins/ikws/searchproviders/identica_groups.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/identica_groups.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -15,6 +15,7 @@
 Name[en_GB]=Identi.ca Groups
 Name[es]=Grupos Identi.ca
 Name[et]=Identi.ca grupid
+Name[eu]=Identi.ca Taldeak
 Name[fi]=Identi.ca Ryhmät
 Name[fr]=Groupes Icenti.ca
 Name[fy]=Identi.ca groepen
@@ -67,6 +68,7 @@
 Query[en_GB]=http://identi.ca/search/group?q=\\{@}
 Query[es]=http://identi.ca/search/group?q=\\{@}
 Query[et]=http://identi.ca/search/group?q=\\{@}
+Query[eu]=http://identi.ca/search/group?q=\\{@}
 Query[fi]=http://identi.ca/search/group?q=\\{@}
 Query[fr]=http://identi.ca/search/group?q=\\{@}
 Query[fy]=http://identi.ca/search/group?q=\\{@}
--- runtime/kurifilter-plugins/ikws/searchproviders/duckduckgo.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/duckduckgo.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -13,6 +13,7 @@
 Name[en_GB]=Duck Duck Go
 Name[es]=Duck Duck Go
 Name[et]=Duck Duck Go
+Name[eu]=Duck Duck Go
 Name[fi]=Duck Duck Go
 Name[fr]=Duck Duck Go
 Name[fy]=Duck Duck Go
@@ -62,6 +63,7 @@
 Query[en_GB]=http://duckduckgo.com/?q=\\{@}&v=
 Query[es]=http://duckduckgo.com/?q=\\{@}&v=
 Query[et]=http://duckduckgo.com/?q=\\{@}&v=
+Query[eu]=http://duckduckgo.com/?q=\\{@}&v=
 Query[fi]=http://duckduckgo.com/?q=\\{@}&v=
 Query[fr]=http://duckduckgo.com/?q=\\{@}&v=
 Query[fy]=http://duckduckgo.com/?q=\\{@}&v=
--- runtime/kurifilter-plugins/ikws/searchproviders/php.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/php.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -102,6 +102,7 @@
 Query[en_GB]=http://www.php.net/manual-lookup.php?pattern=\\{@}
 Query[es]=http://www.php.net/manual-lookup.php?pattern=\\{@}
 Query[et]=http://www.php.net/manual-lookup.php?pattern=\\{@}
+Query[eu]=http://www.php.net/manual-lookup.php?pattern=\\{@}
 Query[fi]=http://www.php.net/manual-lookup.php?pattern=\\{@}
 Query[fr]=http://www.php.net/manual-lookup.php?pattern=\\{@}
 Query[fy]=http://www.php.net/manual-lookup.php?pattern=\\{@}
--- runtime/kurifilter-plugins/ikws/searchproviders/opendesktop.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/opendesktop.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -10,12 +10,15 @@
 Name[el]=openDesktop.org
 Name[en_GB]=openDesktop.org
 Name[es]=openDesktop.org
+Name[et]=openDesktop.org
+Name[eu]=openDesktop.org
 Name[fr]=openDesktop.org
 Name[ga]=openDesktop.org
 Name[ia]=openDesktop.org
 Name[id]=openDesktop.org
 Name[it]=openDesktop.org
 Name[km]=openDesktop.org
+Name[lt]=openDesktop.org
 Name[nds]=openDesktop.org
 Name[nl]=openDesktop.org
 Name[pa]=openDesktop.org
@@ -44,6 +47,7 @@
 Query[en_GB]=http://www.opendesktop.org/content/search.php?search=Search&name=\\{@}
 Query[es]=http://www.opendesktop.org/content/search.php?search=Search&name=\\{@}
 Query[et]=http://www.opendesktop.org/content/search.php?search=Search&name=\\{@}
+Query[eu]=http://www.opendesktop.org/content/search.php?search=Search&name=\\{@}
 Query[fi]=http://www.opendesktop.org/content/search.php?search=Search&name=\\{@}
 Query[fr]=http://www.opendesktop.org/content/search.php?search=Search&name=\\{@}
 Query[fy]=http://www.opendesktop.org/content/search.php?search=Search&name=\\{@}
--- runtime/kurifilter-plugins/ikws/searchproviders/kde_userbase.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/kde_userbase.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -13,6 +13,7 @@
 Name[en_GB]=KDE UserBase
 Name[es]=KDE UserBase
 Name[et]=KDE UserBase
+Name[eu]=KDE UserBase
 Name[fi]=KDE UserBase
 Name[fr]=Centre utilisateur de KDE
 Name[fy]=KDE UserBase
@@ -65,6 +66,7 @@
 Query[en_GB]=http://userbase.kde.org/Special:Search?search=\\{@}&fulltext=Search
 Query[es]=http://userbase.kde.org/Special:Search?search=\\{@}&fulltext=Search
 Query[et]=http://userbase.kde.org/Special:Search?search=\\{@}&fulltext=Search
+Query[eu]=http://userbase.kde.org/Special:Search?search=\\{@}&fulltext=Search
 Query[fi]=http://userbase.kde.org/Special:Search?search=\\{@}&fulltext=Search
 Query[fr]=http://userbase.kde.org/Special:Search?search=\\{@}&fulltext=Search
 Query[fy]=http://userbase.kde.org/Special:Search?search=\\{@}&fulltext=Search
--- runtime/kurifilter-plugins/ikws/searchproviders/7digital.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/7digital.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -15,6 +15,7 @@
 Name[en_GB]=7Digital
 Name[es]=7Digital
 Name[et]=7Digital
+Name[eu]=7Digital
 Name[fi]=7Digital
 Name[fr]=7 numérique
 Name[fy]=7Digital
@@ -67,6 +68,7 @@
 Query[en_GB]=http://us.7digital.com/Search?search=\\{@}&searchtype=global&submit=Search
 Query[es]=http://us.7digital.com/Search?search=\\{@}&searchtype=global&submit=Search
 Query[et]=http://us.7digital.com/Search?search=\\{@}&searchtype=global&submit=Search
+Query[eu]=http://us.7digital.com/Search?search=\\{@}&searchtype=global&submit=Search
 Query[fi]=http://us.7digital.com/Search?search=\\{@}&searchtype=global&submit=Search
 Query[fr]=http://us.7digital.com/Search?search=\\{@}&searchtype=global&submit=Search
 Query[fy]=http://us.7digital.com/Search?search=\\{@}&searchtype=global&submit=Search
--- runtime/kurifilter-plugins/ikws/searchproviders/wikia.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/wikia.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -15,6 +15,7 @@
 Name[en_GB]=Wikia
 Name[es]=Wikia
 Name[et]=Wikia
+Name[eu]=Wikia
 Name[fi]=Wikia
 Name[fr]=Wikia
 Name[fy]=Wikia
@@ -68,6 +69,7 @@
 Query[en_GB]=http://www.wikia.com/Special:Search?search=\\{@}&wikia_search_submit=Search
 Query[es]=http://www.wikia.com/Special:Search?search=\\{@}&wikia_search_submit=Search
 Query[et]=http://www.wikia.com/Special:Search?search=\\{@}&wikia_search_submit=Search
+Query[eu]=http://www.wikia.com/Special:Search?search=\\{@}&wikia_search_submit=Search
 Query[fi]=http://www.wikia.com/Special:Search?search=\\{@}&wikia_search_submit=Search
 Query[fr]=http://www.wikia.com/Special:Search?search=\\{@}&wikia_search_submit=Search
 Query[fy]=http://www.wikia.com/Special:Search?search=\\{@}&wikia_search_submit=Search
--- runtime/kurifilter-plugins/ikws/searchproviders/jamendo.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/jamendo.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -14,6 +14,7 @@
 Name[en_GB]=Jamendo
 Name[es]=Jamendo
 Name[et]=Jamendo
+Name[eu]=Jamendo
 Name[fi]=Jamendo
 Name[fr]=Jamendo
 Name[fy]=Jamendo
@@ -41,6 +42,7 @@
 Name[pt_BR]=Jamendo
 Name[ro]=Jamendo
 Name[ru]=Jamendo
+Name[se]=Gávppašeaddji
 Name[sk]=Jamendo
 Name[sr]=Џамендо
 Name[sr@ijekavian]=Џамендо
@@ -66,6 +68,7 @@
 Query[en_GB]=http://www.jamendo.com/en/search/all/\\{@}
 Query[es]=http://www.jamendo.com/en/search/all/\\{@}
 Query[et]=http://www.jamendo.com/en/search/all/\\{@}
+Query[eu]=http://www.jamendo.com/en/search/all/\\{@}
 Query[fi]=http://www.jamendo.com/en/search/all/\\{@}
 Query[fr]=http://www.jamendo.com/en/search/all/\\{@}
 Query[fy]=http://www.jamendo.com/en/search/all/\\{@}
--- runtime/kurifilter-plugins/ikws/searchproviders/vimeo.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/vimeo.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -14,6 +14,7 @@
 Name[en_GB]=Vimeo
 Name[es]=Vimeo
 Name[et]=Vimeo
+Name[eu]=Vimeo
 Name[fi]=Vimeo
 Name[fr]=Vimeo
 Name[fy]=Vimeo
@@ -66,6 +67,7 @@
 Query[en_GB]=http://vimeo.com/videos/search:\\{@}
 Query[es]=http://vimeo.com/videos/search:\\{@}
 Query[et]=http://vimeo.com/videos/search:\\{@}
+Query[eu]=http://vimeo.com/videos/search:\\{@}
 Query[fi]=http://vimeo.com/videos/search:\\{@}
 Query[fr]=http://vimeo.com/videos/search:\\{@}
 Query[fy]=http://vimeo.com/videos/search:\\{@}
--- runtime/kurifilter-plugins/ikws/searchproviders/yahoo.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/yahoo.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -16,6 +16,7 @@
 Name[eo]=Yahoo
 Name[es]=Yahoo
 Name[et]=Yahoo
+Name[eu]=Yahoo
 Name[fi]=Yahoo
 Name[fr]=Yahoo
 Name[fy]=Yahoo
@@ -42,6 +43,7 @@
 Name[pt_BR]=Yahoo
 Name[ro]=Yahoo
 Name[ru]=Yahoo
+Name[se]=Yahoo
 Name[sk]=Yahoo
 Name[sr]=Јаху
 Name[sr@ijekavian]=Јаху
@@ -67,6 +69,7 @@
 Query[en_GB]=http://search.yahoo.com/search?toggle=1&cop=mss&ei=UTF-8&fr=yfp-t-701&p=\\{@}&x=0&y=0&=Web+Search
 Query[es]=http://search.yahoo.com/search?toggle=1&cop=mss&ei=UTF-8&fr=yfp-t-701&p=\\{@}&x=0&y=0&=Web+Search
 Query[et]=http://search.yahoo.com/search?toggle=1&cop=mss&ei=UTF-8&fr=yfp-t-701&p=\\{@}&x=0&y=0&=Web+Search
+Query[eu]=http://search.yahoo.com/search?toggle=1&cop=mss&ei=UTF-8&fr=yfp-t-701&p=\\{@}&x=0&y=0&=Web+Search
 Query[fi]=http://search.yahoo.com/search?toggle=1&cop=mss&ei=UTF-8&fr=yfp-t-701&p=\\{@}&x=0&y=0&=Web+Search
 Query[fr]=http://search.yahoo.com/search?toggle=1&cop=mss&ei=UTF-8&fr=yfp-t-701&p=\\{@}&x=0&y=0&=Web+Search
 Query[fy]=http://search.yahoo.com/search?toggle=1&cop=mss&ei=UTF-8&fr=yfp-t-701&p=\\{@}&x=0&y=0&=Web+Search
--- runtime/kurifilter-plugins/ikws/searchproviders/gitorious.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/gitorious.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -15,6 +15,7 @@
 Name[en_GB]=Gitorious
 Name[es]=Gitorious
 Name[et]=Gitorious
+Name[eu]=Gitorious
 Name[fi]=Gitorious
 Name[fr]=Gitorious
 Name[fy]=Gitorious
@@ -67,6 +68,7 @@
 Query[en_GB]=http://gitorious.org/search?q=\\{@}&commit=Search
 Query[es]=http://gitorious.org/search?q=\\{@}&commit=Search
 Query[et]=http://gitorious.org/search?q=\\{@}&commit=Search
+Query[eu]=http://gitorious.org/search?q=\\{@}&commit=Search
 Query[fi]=http://gitorious.org/search?q=\\{@}&commit=Search
 Query[fr]=http://gitorious.org/search?q=\\{@}&commit=Search
 Query[fy]=http://gitorious.org/search?q=\\{@}&commit=Search
--- runtime/kurifilter-plugins/ikws/searchproviders/amazon.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/amazon.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -15,6 +15,7 @@
 Name[en_GB]=Amazon
 Name[es]=Amazon
 Name[et]=Amazon
+Name[eu]=Amazon
 Name[fi]=Amazon
 Name[fr]=Amazon
 Name[fy]=Amazon
@@ -67,6 +68,7 @@
 Query[en_GB]=http://www.amazon.co.uk/s/ref=nb_sb_noss?url=search-alias=aps&field-keywords=\\{@}&x=0&y=0
 Query[es]=http://www.amazon.com/s/ref=nb_sb_noss?url=search-alias=aps&field-keywords=\\{@}&x=0&y=0
 Query[et]=http://www.amazon.com/s/ref=nb_sb_noss?url=search-alias=aps&field-keywords=\\{@}&x=0&y=0
+Query[eu]=http://www.amazon.com/s/ref=nb_sb_noss?url=search-alias=aps&field-keywords=\\{@}&x=0&y=0
 Query[fi]=http://www.amazon.com/s/ref=nb_sb_noss?url=search-alias=aps&field-keywords=\\{@}&x=0&y=0
 Query[fr]=http://www.amazon.com/s/ref=nb_sb_noss?url=search-alias=aps&field-keywords=\\{@}&x=0&y=0
 Query[fy]=http://www.amazon.com/s/ref=nb_sb_noss?url=search-alias=aps&field-keywords=\\{@}&x=0&y=0
--- runtime/kurifilter-plugins/ikws/searchproviders/duckduckgo_info.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/duckduckgo_info.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -12,6 +12,7 @@
 Name[en_GB]=Duck Duck Go Info
 Name[es]=Duck Duck Go Info
 Name[et]=Duck Duck Go Info
+Name[eu]=Duck Duck Go Info
 Name[fr]=Duck Duck Go Info
 Name[fy]=Duck Duck Go Ynfo
 Name[ga]=Eolas Duck Duck Go
@@ -61,6 +62,7 @@
 Query[en_GB]=http://duckduckgo.com/?q=\\{@}&v=r
 Query[es]=http://duckduckgo.com/?q=\\{@}&v=r
 Query[et]=http://duckduckgo.com/?q=\\{@}&v=r
+Query[eu]=http://duckduckgo.com/?q=\\{@}&v=r
 Query[fi]=http://duckduckgo.com/?q=\\{@}&v=r
 Query[fr]=http://duckduckgo.com/?q=\\{@}&v=r
 Query[fy]=http://duckduckgo.com/?q=\\{@}&v=r
--- runtime/kurifilter-plugins/ikws/searchproviders/identica_notices.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/kurifilter-plugins/ikws/searchproviders/identica_notices.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -14,6 +14,7 @@
 Name[en_GB]=Identi.ca Notices
 Name[es]=Avisos Identi.ca
 Name[et]=Identi.ca teadaanded
+Name[eu]=Identi.ca Berriak
 Name[fr]=Notices Identi.ca
 Name[fy]=Identi.ca Notices
 Name[ga]=Fógraí Identi.ca
@@ -62,6 +63,7 @@
 Query[en_GB]=http://identi.ca/search/notice?q=\\{@}
 Query[es]=http://identi.ca/search/notice?q=\\{@}
 Query[et]=http://identi.ca/search/notice?q=\\{@}
+Query[eu]=http://identi.ca/search/notice?q=\\{@}
 Query[fi]=http://identi.ca/search/notice?q=\\{@}
 Query[fr]=http://identi.ca/search/notice?q=\\{@}
 Query[fy]=http://identi.ca/search/notice?q=\\{@}
--- runtime/soliduiserver/soliduiserver.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/soliduiserver/soliduiserver.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -17,6 +17,7 @@
 Name[en_GB]=Hardware Detection
 Name[es]=Detección del hardware
 Name[et]=Riistvara tuvastamine
+Name[eu]=Hardware detekzioa
 Name[fi]=Laitteiston tunnistus
 Name[fr]=Détection du matériel
 Name[fy]=Hurdguod deteksje
@@ -65,6 +66,7 @@
 Comment[en_GB]=Provides a user interface for hardware events
 Comment[es]=Proporciona una interfaz de usuario para eventos de hardware
 Comment[et]=Riistvarasündmuste kasutajaliides
+Comment[eu]=Hardware gertakarientzako erabiltzaile interfaze bat hornitzen du
 Comment[fi]=Tarjoaa käyttöliittymän laitteistotapahtumille
 Comment[fr]=Fourni un interface utilisateur pour les évènements matériels
 Comment[fy]=Ferskaft in brûkersomjouwing foar hurdguod barren
--- runtime/attica/kcm/kcm_attica.desktop	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/attica/kcm/kcm_attica.desktop	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -22,6 +22,7 @@
 Name[en_GB]=Social Desktop
 Name[es]=Escritorio social
 Name[et]=Sotsiaalne töölaud
+Name[eu]=Gizarte mahaigaina
 Name[fi]=Sosiaalinen työpöytä
 Name[fr]=Bureau social
 Name[fy]=Sosiale buroblêd
@@ -73,6 +74,7 @@
 Comment[en_GB]=Manage Social Desktop Providers
 Comment[es]=Gestionar proveedores del escritorio social
 Comment[et]=Sotsiaalse töölaua pakkujate haldamine
+Comment[eu]=Kudeatu Gizarte Mahaigain Hornitzaileak
 Comment[fi]=Hallinnoi sosiaalisen työpöydän tarjoajia
 Comment[fr]=Gérer les fournisseurs de bureaux sociaux
 Comment[fy]=Sosjale buroblêd ferskaffers beheare
--- runtime/plasma/tools/plasmapkg/main.cpp	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/plasma/tools/plasmapkg/main.cpp	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -26,6 +26,7 @@
 #include <KAboutData>
 #include <KCmdLineArgs>
 #include <KLocale>
+#include <KPluginInfo>
 #include <KService>
 #include <KServiceTypeTrader>
 #include <KShell>
@@ -52,12 +53,14 @@
 QStringList packages(const QStringList& types)
 {
     QStringList result;
+
     foreach (const QString& type, types) {
-        KService::List services = KServiceTypeTrader::self()->query("Plasma/" + type);
-        foreach(const KService::Ptr &service, services) {
+        const KService::List services = KServiceTypeTrader::self()->query(type);
+        foreach (const KService::Ptr &service, services) {
             result << service->property("X-KDE-PluginInfo-Name", QVariant::String).toString();
         }
     }
+
     return result;
 }
 
@@ -65,7 +68,7 @@
 {
     QStringList list = packages(types);
     list.sort();
-    foreach(const QString& package, list) {
+    foreach (const QString& package, list) {
         output(package);
     }
 }
@@ -158,9 +161,9 @@
     if (type == i18nc("package type", "plasmoid").toLower() || type == "plasmoid") {
         packageRoot = "plasma/plasmoids/";
         servicePrefix = "plasma-applet-";
-        pluginTypes << "Applet";
-        pluginTypes << "PopupApplet";
-        pluginTypes << "Containment";
+        pluginTypes << "Plasma/Applet";
+        pluginTypes << "Plasma/PopupApplet";
+        pluginTypes << "Plasma/Containment";
     } else if (type == i18nc("package type", "theme").toLower() || type == "theme") {
         packageRoot = "desktoptheme/";
     } else if (type == i18nc("package type", "wallpaper").toLower() || type == "wallpaper") {
@@ -168,24 +171,24 @@
     } else if (type == i18nc("package type", "dataengine").toLower() || type == "dataengine") {
         packageRoot = "plasma/dataengines/";
         servicePrefix = "plasma-dataengine-";
-        pluginTypes << "DataEngine";
+        pluginTypes << "Plasma/DataEngine";
     } else if (type == i18nc("package type", "runner").toLower() || type == "runner") {
         packageRoot = "plasma/runners/";
         servicePrefix = "plasma-runner-";
-        pluginTypes << "Runner";
+        pluginTypes << "Plasma/Runner";
     } else if (type == i18nc("package type", "wallpaperplugin").toLower() || type == "wallpaperplugin") {
         packageRoot = "plasma/wallpapers/";
         servicePrefix = "plasma-wallpaper-";
-        pluginTypes << "Wallpaper";
+        pluginTypes << "Plasma/Wallpaper";
     } else if (type == i18nc("package type", "layout-template").toLower() || type == "layout-template") {
         packageRoot = "plasma/layout-templates/";
         servicePrefix = "plasma-layout-";
-        pluginTypes << "LayoutTemplate";
+        pluginTypes << "Plasma/LayoutTemplate";
     } else {
-        QString constraint = QString("'%1' == [X-KDE-PluginInfo-Name]").arg(packageRoot);
+        const QString constraint = QString("[X-KDE-PluginInfo-Name] == '%1'").arg(args->getOption("type"));
         KService::List offers = KServiceTypeTrader::self()->query("Plasma/PackageStructure", constraint);
         if (offers.isEmpty()) {
-            output(i18n("Could not find a suitable installer for package of type %1", type));
+            output(i18n("Could not find a suitable installer for package of type %1", args->getOption("type")));
             return 1;
         }
 
@@ -195,9 +198,10 @@
 
         if (!installer) {
             output(i18n("Could not load installer for package of type %1. Error reported was: %2",
-                        type, error));
+                        args->getOption("type"), error));
             return 1;
         }
+
         packageRoot = installer->defaultPackageRoot();
         pluginTypes << installer->type();
     }
--- runtime/plasma/remotewidgetshelper/kcm_remotewidgets.actions	(.../tags/KDE/4.5.0/kdebase)	(revision 1161528)
+++ runtime/plasma/remotewidgetshelper/kcm_remotewidgets.actions	(.../branches/KDE/4.5/kdebase)	(revision 1161528)
@@ -9,6 +9,7 @@
 Name[en_GB]=Save remote widgets' policies
 Name[es]=Guardado de políticas de elementos gráficos remotos
 Name[et]=Välisvidinate reeglite salvestamine
+Name[eu]=Gorde urruneko tramankuluen politikak
 Name[fr]=Enregistrer la stratégie des gadgets distants
 Name[fy]=Belied foar eksterne widgets bewarje
 Name[ga]=Sábháil polasaithe um giuirléidí cianda
@@ -52,6 +53,7 @@
 Description[en_GB]=Prevents the system from saving remote plasma widgets' policies
 Description[es]=Impide que el sistema guarde políticas de componentes Plasma remotos
 Description[et]=Takistab süsteemil salvestamast Plasma välisvidinate reegleid
+Description[eu]=Sistemak urruneko plasma tramankuluen politikak gorde ditzan galarazten du
 Description[fr]=Empêche le système d'enregistrer la stratégie des gadgets distant
 Description[fy]=Foarkomt it bewarjen troch it systeem fan beliedsregels foar wigets op ôfstân
 Description[ga]=Ná lig don chóras polasaithe um giuirléidí cianda plasma a shábháil
