--- runtime/attica/kdeplugin/kdeplatformdependent.cpp	(.../tags/KDE/4.5.1/kdebase)	(wersja 1170567)
+++ runtime/attica/kdeplugin/kdeplatformdependent.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1170567)
@@ -26,8 +26,9 @@
 
 #include <QtCore/QDebug>
 
+#include <KConfigGroup>
+#include <kcomponentdata.h>
 #include <KDebug>
-#include <KConfigGroup>
 #include <KWallet/Wallet>
 #include <kcmultidialog.h>
 #include <KStringHandler>
@@ -38,6 +39,11 @@
 KdePlatformDependent::KdePlatformDependent()
     : m_config(KSharedConfig::openConfig("atticarc")), m_accessManager(0), m_wallet(0)
 {
+    // when a plain Qt application loads this plugin, it needs a valid KGlobal object
+    if (!KGlobal::hasMainComponent()) {
+        KComponentData componentData("attica_kde");
+    }
+
     KLocale* locale = KGlobal::locale();
     if (locale) {
         locale->insertCatalog("attica_kde");
