--- runtime/knotify/notifybyktts.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956244)
+++ runtime/knotify/notifybyktts.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956244)
@@ -17,16 +17,18 @@
    Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
  */
-#include "notifybyktts.h" 
+#include "notifybyktts.h"
 #include <QtDBus/QtDBus>
 #include <QHash>
-
+#include <ktoolinvocation.h>
+#include <kmessagebox.h>
 #include <kmacroexpander.h>
-
+#include <klocale.h>
 #include "knotifyconfig.h"
 
-NotifyByKTTS::NotifyByKTTS(QObject *parent) : KNotifyPlugin(parent) , kspeech("org.kde.kttsd", "/KSpeech", "org.kde.KSpeech")
+NotifyByKTTS::NotifyByKTTS(QObject *parent) : KNotifyPlugin(parent),kspeech("org.kde.kttsd", "/KSpeech", "org.kde.KSpeech"), tryToStartKttsd( false )
 {
+    if( kspeech.isValid())
 	kspeech.call("setApplicationName", "KNotify");
 }
 
@@ -37,8 +39,26 @@
 
 void NotifyByKTTS::notify( int id, KNotifyConfig * config )
 {
-	QString say = config->readEntry( "KTTS" );
-	
+        if( !kspeech.isValid())
+        {
+            if (  tryToStartKttsd ) //don't try to restart it all the time.
+                return;
+            // If KTTSD not running, start it.
+            if (!QDBusConnection::sessionBus().interface()->isServiceRegistered("org.kde.kttsd"))
+            {
+                QString error;
+                if (KToolInvocation::startServiceByDesktopName("kttsd", QStringList(), &error))
+                {
+                    KMessageBox::error(0, i18n( "Starting KTTSD Failed"), error );
+                    tryToStartKttsd = true;
+                    return;
+                }
+                kspeech.call("setApplicationName", "KNotify");
+            }
+        }
+
+        QString say = config->readEntry( "KTTS" );
+
 	if (!say.isEmpty()) {
 		QHash<QChar,QString> subst;
 		subst.insert( 'e', config->eventid );
@@ -49,10 +69,10 @@
 		subst.insert( 'm', config->text );
 		say = KMacroExpander::expandMacrosShellQuote( say, subst );
 	}
-	
+
 	if ( say.isEmpty() )
 		say = config->text; // fallback
-	
+
 	kspeech.call(QDBus::NoBlock, "say", say, 0);
 
 	finished(id);
--- runtime/knotify/notifybyktts.h	(.../tags/KDE/4.2.2/kdebase)	(wersja 956244)
+++ runtime/knotify/notifybyktts.h	(.../branches/KDE/4.2/kdebase)	(wersja 956244)
@@ -29,15 +29,17 @@
 #include <QDBusInterface>
 
 class NotifyByKTTS : public KNotifyPlugin
-{ Q_OBJECT
-	public:
-		NotifyByKTTS(QObject *parent=0l);
-		virtual ~NotifyByKTTS();
-		
-		virtual QString optionName() { return "KTTS"; }
-		virtual void notify(int id , KNotifyConfig *config);
-	private:
-		QDBusInterface kspeech;
+{
+    Q_OBJECT
+public:
+    NotifyByKTTS(QObject *parent=0l);
+    virtual ~NotifyByKTTS();
+
+    virtual QString optionName() { return "KTTS"; }
+    virtual void notify(int id , KNotifyConfig *config);
+private:
+    QDBusInterface kspeech;
+    bool tryToStartKttsd;
 };
 
 #endif
--- runtime/knotify/notifybyexecute.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956244)
+++ runtime/knotify/notifybyexecute.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956244)
@@ -21,7 +21,7 @@
 #include "notifybyexecute.h"
 
 #include <QHash>
-#include <QProcess>
+#include <KProcess>
 
 #include "knotifyconfig.h"
 
@@ -59,8 +59,10 @@
 		QString execLine = KMacroExpander::expandMacrosShellQuote( command, subst );
 		if ( execLine.isEmpty() )
 			execLine = command; // fallback
-                //TODO fixme on WIN32
-		QProcess::startDetached( "/bin/sh", QStringList() << "-c" << execLine );
+                KProcess proc;
+		proc.setShellCommand(execLine.trimmed());
+		if(!proc.startDetached())
+			kDebug(300)<<"KNotify: Could not start process!";
 	}
 	
 	finish( id );
--- runtime/knotify/notifybypopup.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956244)
+++ runtime/knotify/notifybypopup.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956244)
@@ -51,7 +51,7 @@
 	m_dbusServiceExists = interface && interface->isServiceRegistered(dbusServiceName);
 
 	if( m_dbusServiceExists )
-		kDebug(300) << "using" << dbusServiceName << "for popups";
+		slotServiceOwnerChanged(dbusServiceName, QString(), "_"); //connect signals
 
 	// to catch register/unregister events from service in runtime
 	connect(interface, SIGNAL(serviceOwnerChanged(const QString&, const QString&, const QString&)),
--- runtime/kcontrol/locale/localetime.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956244)
+++ runtime/kcontrol/locale/localetime.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956244)
@@ -285,7 +285,11 @@
     connect (m_comboWeekDayOfPray, SIGNAL(activated(int)),
              this, SLOT(slotWeekDayOfPrayChanged(int)));
 
-    lay->addRow(fGr,wGr);
+    QHBoxLayout * horizontalLayout = new QHBoxLayout();
+    horizontalLayout->addWidget( fGr );
+    horizontalLayout->addWidget( wGr );
+    
+    lay->addRow( horizontalLayout );
 
     updateWeekDayNames();
 }
--- runtime/phonon/kcm/xine/kcm_phononxine.desktop	(.../tags/KDE/4.2.2/kdebase)	(wersja 956244)
+++ runtime/phonon/kcm/xine/kcm_phononxine.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 956244)
@@ -32,6 +32,7 @@
 Comment[af]=Xine Backend konfigurasie
 Comment[be]=Настаўленне унутранага карыстання Xine
 Comment[be@latin]=Настаўленне унутранага карыстання Xine
+Comment[bg]=Настройване Xine
 Comment[bn]=জাইন ব্যাক-এন্ড কনফিগারেশন
 Comment[bn_IN]=Xine ব্যাক-এন্ডের কনফিগারেশন
 Comment[ca]=Configuració del dorsal del Xine
--- runtime/platforms/win/kwinstartmenu/winstartmenu.desktop	(.../tags/KDE/4.2.2/kdebase)	(wersja 956244)
+++ runtime/platforms/win/kwinstartmenu/winstartmenu.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 956244)
@@ -3,6 +3,7 @@
 Name=KDED Windows Start Menu Module
 Name[ar]=قائمة K نمط وندوز
 Name[be@latin]=Modul hałoŭnaha menu dla systemy „KDE”
+Name[bg]=Модул KDED Windows Start Menu
 Name[bn_IN]=KDED Windows Start Menu মডিউল
 Name[ca]=Mòdul del menú d'inici del Windows del KDED
 Name[cs]=Modul KDED pro startovní nabídku
--- runtime/kioslave/info/info.cc	(.../tags/KDE/4.2.2/kdebase)	(wersja 956244)
+++ runtime/kioslave/info/info.cc	(.../branches/KDE/4.2/kdebase)	(wersja 956244)
@@ -38,7 +38,7 @@
 	} else {
 		QString missing =m_infoScript.isNull() ?  "kio_info/kde-info2html" : "kio_info/kde-info2html.conf";
 		errorStr = "kde-info2html" + i18n( "\nUnable to locate file %1 which is necessary to run this service. "
-				"Please check your software installation" ,  missing );
+				"Please check your software installation." ,  missing );
 	}
 	error( KIO::ERR_CANNOT_LAUNCH_PROCESS, errorStr );
 	exit();
--- runtime/kioslave/info/Messages.sh	(.../tags/KDE/4.2.2/kdebase)	(wersja 956244)
+++ runtime/kioslave/info/Messages.sh	(.../branches/KDE/4.2/kdebase)	(wersja 956244)
@@ -1,2 +1,2 @@
 #! /usr/bin/env bash
-$XGETTEXT *.cpp -o $podir/kio_info.pot
+$XGETTEXT *.cc -o $podir/kio_info.pot
--- runtime/kstyles/themes/qtwindows.themerc	(.../tags/KDE/4.2.2/kdebase)	(wersja 956244)
+++ runtime/kstyles/themes/qtwindows.themerc	(.../branches/KDE/4.2/kdebase)	(wersja 956244)
@@ -18,7 +18,7 @@
 Comment[ar]=أسلوب مدمجة نمط مايكروسوفت وندوز 9x
 Comment[be]=Убудаваны стыль Windows 9x
 Comment[be@latin]=Styl systemy „Windows 9x”
-Comment[bg]=Вграден стил  Windows 9x style
+Comment[bg]=Вграден стил Windows 9x
 Comment[ca]=Estil integrat Windows 9x sense temes
 Comment[cs]=Zabudovaný styl Windows 9x bez motivu
 Comment[csb]=Wbùdowóny sztél szlachùjący za Windows 9x
--- runtime/kstyles/oxygen/oxygen.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956244)
+++ runtime/kstyles/oxygen/oxygen.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956244)
@@ -3021,6 +3021,9 @@
         case PM_DefaultLayoutSpacing:
             return 4; // qcommon is 6
 
+        case PM_ButtonMargin:
+            return 5;
+
         case PM_DefaultFrameWidth:
             if (qobject_cast<const QLineEdit*>(widget))
                 return 4;
--- runtime/kstyles/oxygen/CMakeLists.txt	(.../tags/KDE/4.2.2/kdebase)	(wersja 956244)
+++ runtime/kstyles/oxygen/CMakeLists.txt	(.../branches/KDE/4.2/kdebase)	(wersja 956244)
@@ -15,7 +15,7 @@
 
 kde4_add_plugin(oxygen ${oxygen_PART_SRCS})
 
-target_link_libraries(oxygen  ${KDE4_KDEUI_LIBS} )
+target_link_libraries(oxygen  ${KDE4_KDEUI_LIBS} ${KDE4_KIO_LIBS})
 
 install(TARGETS oxygen  DESTINATION ${PLUGIN_INSTALL_DIR}/plugins/styles/ )
 
--- runtime/kurifilter-plugins/ikws/searchproviders/technoratitags.desktop	(.../tags/KDE/4.2.2/kdebase)	(wersja 956244)
+++ runtime/kurifilter-plugins/ikws/searchproviders/technoratitags.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 956244)
@@ -34,6 +34,7 @@
 Name[kn]=ಟೆಕ್ನೋರೆಟಿ ತಲೆಚೀಟಿಗಳು (ಟಾಗ್ಸ್)
 Name[ko]=Technorati 태그
 Name[ku]=Etîketên Technorati
+Name[lt]=Technorati žymos
 Name[lv]=Technorati tagi
 Name[mai]=टेक्नोराती टैग्स
 Name[ml]=ടെക്നോരാറ്റി ടാഗുകള്‍
