--- runtime/knotify/kde.notifyrc	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/knotify/kde.notifyrc	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -803,7 +803,7 @@
 Comment[et]=Tekkis tõsine viga, mis sundis programmi tööd lõpetama
 Comment[eu]=Errore larria egon da programa ateraraziz
 Comment[fi]=Vakava virhe aiheutti ohjelman keskeytymisen
-Comment[fr]=Une erreur sérieuse a entraîné la fin le programme
+Comment[fr]=Une erreur sérieuse a entraîné la fin du programme
 Comment[fy]=Der wie in serieuse flater wertroch it programma sluten is
 Comment[ga]=Tharla earráid thromchúiseach a thug ar an gclár éirí as
 Comment[gl]=Aconteceu un erro grave que fixo que o programa se pechase inesperadamente
@@ -2617,7 +2617,7 @@
 Comment[et]=Esitatakse küsimus
 Comment[eu]=Galdera bat egiten ari da
 Comment[fi]=Esitetään kysymys
-Comment[fr]=une question vous est posée
+Comment[fr]=Une question vous est posée
 Comment[fy]=In fraach wurdt stelt
 Comment[ga]=Ceist á fiafraí
 Comment[gl]=Está a facerse unha pregunta
--- runtime/solid-device-automounter/kcm/device_automounter_kcm.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/solid-device-automounter/kcm/device_automounter_kcm.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -88,7 +88,7 @@
 Comment[et]=Eemaldatavate andmekandjate automaatse käitlemise seadistamine
 Comment[eu]=Biltegiratzeko euskarri aldagarrien maneiu automatikoaren konfiguraketa
 Comment[fi]=Muokkaa irrotettavan tallennusmedian automaattista käsittelyä
-Comment[fr]=Configure la prise en charge automatique des média de stockage amovible
+Comment[fr]=Configure la prise en charge automatique des médias de stockage amovibles
 Comment[fy]=IT automatysk ôfhanneljen fan útnimbere opslach media ynstelle
 Comment[ga]=Cumraigh láimhseáil uathoibríoch meán inbhainte stórála
 Comment[he]=הגדרת תפעול אוטומטי להתקני אחסון נשלפים.
--- runtime/localization/currency/khr.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/localization/currency/khr.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -17,7 +17,7 @@
 Name[es]=Riel camboyano
 Name[et]=Kambodža riel
 Name[eu]=Kanbodiako riel
-Name[fi]=Kambodzan riel
+Name[fi]=Kambodžan riel
 Name[fr]=Riel cambodgien
 Name[fy]=Cambodjaanske riel
 Name[ga]=Riel na Cambóide
--- runtime/localization/currency/ats.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/localization/currency/ats.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -13,7 +13,7 @@
 Name[csb]=Aùstrëjacczi Sziling
 Name[da]=Østrigsk schilling
 Name[de]=Österreichischer Schilling
-Name[el]=Αυστρία σκίλινγκ
+Name[el]=Σελίνι Αυστρίας
 Name[en_GB]=Austrian Schilling
 Name[eo]=Aŭstra ŝilingo
 Name[es]=Chelín austriaco
--- runtime/localization/currency/eek.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/localization/currency/eek.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -13,7 +13,7 @@
 Name[csb]=Estóńskô Kòróna
 Name[da]=Estisk kroon
 Name[de]=Estnische Krone
-Name[el]=Εσθονική κορώνα
+Name[el]=Εσθονική κορόνα
 Name[en_GB]=Estonian Kroon
 Name[eo]=Estona krono
 Name[es]=Corona estonia
--- runtime/localization/currency/fkp.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/localization/currency/fkp.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -17,7 +17,7 @@
 Name[es]=Libra malvinense
 Name[et]=Falklandi saarte nael
 Name[eu]=Malvinak Uharteetako (Falkland islands) libra
-Name[fi]=Falklandin saarten (Malvinassaarten) punta
+Name[fi]=Falklandinsaarten (Malvinassaarten) punta
 Name[fr]=Livre des Îles Falkland
 Name[fy]=Falklânske pûn
 Name[ga]=Punt na nOileán Fáclainne
--- runtime/localization/currency/kyd.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/localization/currency/kyd.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -17,7 +17,7 @@
 Name[es]=Dólar de las Islas Caimán
 Name[et]=Kaimanisaarte dollar
 Name[eu]=Kaiman Uharteetako dolar
-Name[fi]=Cayman-saarien dollari
+Name[fi]=Caymansaarten dollari
 Name[fr]=Dollar des Îles Caïman
 Name[fy]=Kayman eilânske dollar
 Name[ga]=Dollar na nOileáin Cayman
--- runtime/localization/currency/thb.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/localization/currency/thb.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -12,7 +12,7 @@
 Name[csb]=Tajsczi Baht
 Name[da]=Thailandsk baht
 Name[de]=Baht
-Name[el]=Μπαχτ Ταϊλανδίας
+Name[el]=Μπατ Ταϊλάνδης
 Name[en_GB]=Thai Baht
 Name[eo]=Tajlanda bahto
 Name[es]=Baht tailandés
--- runtime/localization/currency/mgf.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/localization/currency/mgf.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -12,7 +12,7 @@
 Name[csb]=Madagarsczi Frank
 Name[da]=Malagassisk franc
 Name[de]=Madagaskar-Franc
-Name[el]=Φράγκο Μαλγασίκας
+Name[el]=Φράγκο Μαδαγασκάρης
 Name[en_GB]=Malagasy Franc
 Name[eo]=Madagaskara franko
 Name[es]=Franco malgache
--- runtime/localization/currency/tjs.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/localization/currency/tjs.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -17,7 +17,7 @@
 Name[es]=Somoni tayiko
 Name[et]=Tadžikistani somoni
 Name[eu]=Tadjikistaneko somoni
-Name[fi]=Tajikistanin somoni
+Name[fi]=Tadžikistanin somoni
 Name[fr]=Somoni tadjik
 Name[fy]=Tadjikistaanske somoni
 Name[ga]=Somoni na Táidsíceastáine
--- runtime/localization/currency/azm.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/localization/currency/azm.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -18,7 +18,7 @@
 Name[es]=Manat azerí
 Name[et]=Aserbaidžaani manat
 Name[eu]=Azerbaijan Manat
-Name[fi]=Azerbaidzanin manat
+Name[fi]=Azerbaidžanin manat
 Name[fr]=Manat d'Azerbaïjan
 Name[fy]=Azerbeidjaanske manat
 Name[ga]=Manat na hAsarbainseáine
--- runtime/localization/currency/azn.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/localization/currency/azn.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -18,7 +18,7 @@
 Name[es]=Manat azerí
 Name[et]=Aserbaidžaani manat
 Name[eu]=Azerbaijan Manat
-Name[fi]=Azerbaidzanin manat
+Name[fi]=Azerbaidžanin manat
 Name[fr]=Manat d'Azerbaïjan
 Name[fy]=Azerbeidjaanske manat
 Name[ga]=Manat na hAsarbainseáine
--- runtime/localization/currency/bam.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/localization/currency/bam.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -18,7 +18,7 @@
 Name[es]=Marco convertible de Bosnia y Herzegovina
 Name[et]=Bosnia ja Hertsegovina konverteeritav mark
 Name[eu]=Bosnia eta Herzegovinako marko aldagarria
-Name[fi]=Bosnia ja Herzegovinan vaihdettava markka
+Name[fi]=Bosnia ja Hertsegovinan vaihdettava markka
 Name[fr]=Mark convertible de Bosnie-Herzégovine
 Name[fy]=Bosnië en Herzegovina convertible mark
 Name[ga]=Marc insóinseáilte Bhoisnia agus Heirseagaivéin
--- runtime/localization/currency/sgd.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/localization/currency/sgd.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -13,7 +13,7 @@
 Name[csb]=Singapùrsczi Dolar
 Name[da]=Singaporeansk dollar
 Name[de]=Singapur-Dollar
-Name[el]=Δολάριο Σινγκαπούρης
+Name[el]=Δολάριο Σιγκαπούρης
 Name[en_GB]=Singapore Dollar
 Name[eo]=Singapura dolaro
 Name[es]=Dólar de Singapur
--- runtime/localization/currency/isk.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/localization/currency/isk.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -13,7 +13,7 @@
 Name[csb]=Islandkô Kòróna
 Name[da]=Islandsk krone
 Name[de]=Isländische Krone
-Name[el]=Ισλανδική κορώνα
+Name[el]=Ισλανδική κορόνα
 Name[en_GB]=Icelandic Krona
 Name[eo]=Islanda krono
 Name[es]=Corona islandesa
--- runtime/localization/currency/byr.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/localization/currency/byr.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -19,7 +19,7 @@
 Name[es]=Rublo bielorruso
 Name[et]=Valgevene rubla
 Name[eu]=Belorrusiako Errubloa
-Name[fi]=Valkovenäjän rupla
+Name[fi]=Valko-Venäjän rupla
 Name[fr]=Rouble biélorusse
 Name[fy]=Wyt-Russyske rubbel
 Name[ga]=Rúbal na Bealarúise
--- runtime/kcontrol/locale/language.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/kcontrol/locale/language.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -36,7 +36,7 @@
 Name[eu]=Herrialdea/eskualdea eta hizkuntza
 Name[fa]=کشور/منطقه و زبان
 Name[fi]=Alue ja kieli
-Name[fr]=Pays & langue
+Name[fr]=Pays / Région & langue
 Name[fy]=Lân/regio en taal
 Name[ga]=Tír/Réigiún agus Teanga
 Name[gl]=País/Rexión e lingua
--- runtime/kcontrol/componentchooser/componentservices/kcm_kemail.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/kcontrol/componentchooser/componentservices/kcm_kemail.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -84,7 +84,7 @@
 Comment[cy]=Mae'r gwasanaeth ymayn gadael i chi ffurfweddu eich dibynnydd ebost cyffredinol. Dylai bob raglen KDE sydd angen cyrchiad i ddibynnydd ebost ddefnyddio'r gosodiad yma.
 Comment[da]=Denne service tillader dig at indstille din e-mail-klient. Alle KDE-programmer der behøver adgang til en e-mail-klient skulle respektere denne indstilling.
 Comment[de]=Hier können Sie Ihr E-Mail-Programm einrichten. Alle KDE-Anwendungen, die Zugriff auf ein solches Programm benötigen, sollten diese Einstellungen verwenden.
-Comment[el]=Αυτή η υπηρεσία σας επιτρέπει να ρυθμίσετε τον προκαθορισμένο πελάτη email σας. Όλες οι εφαρμογές του KDE που χρειάζονται πρόσβαση σε μια εφαρμογή πελάτη email θα πρέπει να χρησιμοποιούν αυτή τη ρύθμιση.
+Comment[el]=Αυτή η υπηρεσία σας επιτρέπει να ρυθμίσετε τον προκαθορισμένο πελάτη της ηλ. αλληλογραφίας σας. Όλες οι εφαρμογές του KDE που χρειάζονται πρόσβαση σε μια εφαρμογή πελάτη ηλ. αλληλογραφίας θα πρέπει να χρησιμοποιούν αυτή τη ρύθμιση.
 Comment[en_GB]=This service allows you to configure your default email client. All KDE applications which need access to an email client application should honour this setting.
 Comment[eo]=Tiu servo ebligas, ke vi elektas vian defaŭltan retpoŝtilon. Ĉiuj KDEaj aplikoj, kiuj bezonas retpoŝtilon, uzos tiun agordon.
 Comment[es]=Este servicio le permite configurar su cliente de correo predeterminado. Todas las aplicaciones KDE que precisen acceso a una aplicación de cliente de correo deberían reconocer esta opción.
--- runtime/kcontrol/componentchooser/componentservices/kcm_wm.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/kcontrol/componentchooser/componentservices/kcm_wm.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -19,7 +19,7 @@
 Name[et]=Aknahaldur
 Name[eu]=Leiho kudeatzailea
 Name[fi]=Ikkunahallinta
-Name[fr]=Gestionnaire de fenêtre
+Name[fr]=Gestionnaire de fenêtres
 Name[fy]=Finsterbehearder
 Name[ga]=Bainisteoir Fuinneog
 Name[gl]=Xestor de fiestras
@@ -87,7 +87,7 @@
 Comment[et]=Siin saab valida KDE seansis kasutatava aknahalduri.
 Comment[eu]=Hemen zure KDE saioan erabili nahi duzun leiho kudeatzailea hautatu dezakezu.
 Comment[fi]=Tästä voit valita KDE-työpöydän käyttämän ikkunaohjelman.
-Comment[fr]=Vous pouvez choisir ici le gestionnaire de fenêtre que vous souhaitez utiliser dans votre session KDE.
+Comment[fr]=Vous pouvez choisir ici le gestionnaire de fenêtres que vous souhaitez utiliser dans votre session KDE.
 Comment[fy]=Hjir kinne jo de finstersbehearder dy jo yn jo KDE-sesje brûke wolle selektearje.
 Comment[ga]=Anseo is féidir leat an bainisteoir fuinneog a rithfear i do sheisiún KDE.
 Comment[gl]=Aquí pode escoller o xestor de fiestras que desexe executar na sesión de KDE.
--- runtime/kcontrol/componentchooser/componentservices/kcm_filemanager.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/kcontrol/componentchooser/componentservices/kcm_filemanager.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -87,7 +87,7 @@
 Comment[et]=Siin saab seadistada vaikimisi kasutatavat failihaldurit. Kõik KDE rakendused, kus saab avada katalooge, peaksid siinsete seadistustega arvestama.
 Comment[eu]=Hemen zure web arakatzaile lehenetsia konfigura dezakezu. K menuk eta KDE aplikazioek hau erabiliko dute karpetak irekitzerakoan
 Comment[fi]=Tässä voit muokata käytettävää oletustiedostonhallintaohjelmaa. Kaikki KDE-sovellukset käyttävät tätä tiedostonhallintaohjelmaa.
-Comment[fr]=Vous pouvez configurer ici votre gestionnaire de fichiers par défaut. Les entrées du menu K et toutes les applications KDE avec lesquels vous pouvez ouvrir des dossiers, utiliseront ce gestionnaire de fichiers.
+Comment[fr]=Vous pouvez configurer ici votre gestionnaire de fichiers par défaut. Les entrées du menu K et toutes les applications KDE avec lesquelles vous pouvez ouvrir des dossiers, utiliseront ce gestionnaire de fichiers.
 Comment[fy]=Hjir kinne jo de standert triembehearder ynstelle. Alle ynfieren yn it K menu en alle KDE applikaasjes wêryn jo mappen iepenje kinne sille dizze triembehearder brûke.
 Comment[ga]=Anseo is féidir leat bainisteoir comhad réamhshocraithe a chumrú. Úsáidfidh iontrálacha sa roghchlár K an bainisteoir comhad seo, chomh maith le gach feidhmchlár KDE is féidir fillteáin a oscailt.
 Comment[gl]=Aquí pode configurar o xestor de ficheiros predeterminado. As entradas no menú K e todos os programas de KDE nos que poida abrir cartafoles han empregar este xestor de ficheiros.
--- runtime/phonon/platform_kde/phononbackend.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/phonon/platform_kde/phononbackend.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -21,7 +21,7 @@
 Name[et]=KDE multimeedia taustaprogramm
 Name[eu]=KDE multimedia interfazea
 Name[fi]=KDE-multimediataustaosa
-Name[fr]=Moteur de traitement multimédia de KDE
+Name[fr]=Module multimédia de KDE
 Name[fy]=KDE Multymedia efterein
 Name[ga]=Inneall Ilmheán KDE
 Name[gl]=Infraestrutura multimedia de KDE
--- runtime/kpasswdserver/kpasswdserver.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/kpasswdserver/kpasswdserver.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -75,7 +75,7 @@
 Comment[et]=Ajutine parooli puhverdamine
 Comment[eu]=Pasahitzen aldiuneko cachea
 Comment[fi]=Väliaikainen salasanavälimuisti
-Comment[fr]=Masquage du mot de passe temporaire
+Comment[fr]=Masquage temporaire du mot de passe
 Comment[ga]=Taisce Shealadach Focal Faire
 Comment[gu]=કામચલાઉ પાસવર્ડ કેશીંગ
 Comment[he]=שמירה זמנית של סיסמאות בזיכרון מטמון
--- runtime/platforms/win/kwinstartmenu/winstartmenu.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/platforms/win/kwinstartmenu/winstartmenu.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -19,7 +19,7 @@
 Name[et]=KDED Windowsi startmenüü moodul
 Name[eu]=KDED leiho abiorako menuaren modulua
 Name[fi]=KDED Windows-aloitusvalikkomoduuli
-Name[fr]=Module KDED de lancement de fenêtres
+Name[fr]=Module KDED de lancement Windows
 Name[fy]=KDED Windows Start Menu Module
 Name[ga]=Modúl Roghchláir Tosaithe Windows KDED
 Name[gl]=Módulo de menú de inicio de Windows de KDED
--- runtime/kwalletd/kwalletd.notifyrc	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/kwalletd/kwalletd.notifyrc	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -215,7 +215,7 @@
 Comment[et]=KDE turvalaeka deemon nõuab parooli
 Comment[eu]=KDE diruzorroaren deabruak pasahitza eskatzen du
 Comment[fi]=KDE:n Lompakkotaustaprosessi pyytää salasanaa
-Comment[fr]=Le démon du porte-feuille de KDE requiers un mot de passe
+Comment[fr]=Le démon du portefeuille de KDE requiert un mot de passe
 Comment[fy]=The KDE slûf daemon fereasket in wachtwurd
 Comment[ga]=Tá Deamhan Sparáin KDE ag iarraidh focal faire
 Comment[gl]=O Daemon de carteiras de KDE pide un contrasinal
--- runtime/kioslave/thumbnail/desktopthumbnail.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/kioslave/thumbnail/desktopthumbnail.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -18,7 +18,7 @@
 Name[et]=Töölauafailid
 Name[eu]=Mahaigain fitxategiak
 Name[fi]=Työpöytätiedostot
-Name[fr]=Fichier du bureau
+Name[fr]=Fichiers du bureau
 Name[fy]=Buroblêd triemmen
 Name[ga]=Comhaid Deisce
 Name[gl]=Ficheiros do escritorio
--- runtime/kioslave/remote/kdedmodule/remotedirnotify.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/kioslave/remote/kdedmodule/remotedirnotify.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -15,7 +15,7 @@
 Name[et]=Mujalasuva URL-i muutumise teadustaja
 Name[eu]=Urruneko URL aldaketa jakinarazlea
 Name[fi]=Etäverkko-osoitteiden muutosten huomautin
-Name[fr]=Notifieur de changement d'URL distante
+Name[fr]=Notificateur de changement d'URL distante
 Name[gu]=દૂરસ્થ URL ફેરફાર નોંધણી
 Name[he]=שירות הודעות על שינוי כתובות מרוחקות
 Name[hi]=रिमोट आधारित यूआरएल नोटिफ़ायर
--- runtime/nepomuk/services/strigi/nepomukstrigiservice.notifyrc	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/nepomuk/services/strigi/nepomukstrigiservice.notifyrc	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -291,7 +291,7 @@
 Comment[et]=Lõpetati kohalike failide esimene indekseerimine kiire töölauaotsingu võimaldamiseks.
 Comment[eu]=Bilaketa azkarretarako fitxategi lokalen hasierako indexaketata osatu da.
 Comment[fi]=Paikallisten tiedostojen alustava indeksointi työpöydän pikahakua varten on valmis.
-Comment[fr]=L'indexation initiale des fichiers locaux pour des recherches rapides est terminé.
+Comment[fr]=L'indexation initiale des fichiers locaux pour des recherches rapides est terminée.
 Comment[fy]=De earste yndeksearring fan lokale triemmen foar flugge buroblêd-sykopdrachten is foltôge.
 Comment[ga]=Tá an chéad innéacsú de chomhaid logánta críochnaithe.
 Comment[gl]=Completouse a indexación inicial dos ficheiros locais para facer procuras rápidas no escritorio.
--- runtime/nepomuk/services/backupsync/service/identificationset.cpp	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/nepomuk/services/backupsync/service/identificationset.cpp	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -97,10 +97,10 @@
         while( iter.hasNext() ) {
             const QUrl & uri = iter.next();
             done.insert( uri );
+            
+            uris.append( Soprano::Node::resourceToN3( uri ) );
+            
             iter.remove();
-
-            uris.append( Soprano::Node::resourceToN3( uri ) );
-
             if( uris.size() == maxIterationSize )
                 break;
         }
--- runtime/nepomuk/services/backupsync/lib/CMakeLists.txt	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/nepomuk/services/backupsync/lib/CMakeLists.txt	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -46,10 +46,10 @@
 
 install(TARGETS nepomuksync EXPORT kdelibsLibraryTargets ${INSTALL_TARGETS_DEFAULT_ARGS})
 
-install(FILES
-    resourceidentifier.h
-    resourcemerger.h
-    simpleresource.h
-    nepomuksync_export.h
-    DESTINATION ${INCLUDE_INSTALL_DIR}/nepomuk COMPONENT Devel
-)
+#install(FILES
+#    resourceidentifier.h
+#    resourcemerger.h
+#    simpleresource.h
+#    nepomuksync_export.h
+#    DESTINATION ${INCLUDE_INSTALL_DIR}/nepomuk COMPONENT Devel
+#)
--- runtime/nepomuk/services/storage/nepomukstorage.notifyrc	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/nepomuk/services/storage/nepomukstorage.notifyrc	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -15,7 +15,7 @@
 Name[et]=Semantiline andmesalvesti
 Name[eu]=Datu semantikoen biltegiratzea
 Name[fi]=Semanttinen tietovarasto
-Name[fr]=Stockage de données sémantique
+Name[fr]=Stockage de données sémantiques
 Name[ga]=Stóráil Sonraí Séimeantacha
 Name[gu]=નેપોમુક માહિતી સંગ્રહ
 Name[he]=אחסון מידע סמנטי
--- runtime/kurifilter-plugins/ikws/searchproviders/imdb.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/kurifilter-plugins/ikws/searchproviders/imdb.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -26,7 +26,7 @@
 Name[eu]=Interneteko filmeen datu-basea
 Name[fa]=دادگان فیلم اینترنت
 Name[fi]=Internet Movie Database
-Name[fr]=Base de données vidéo sur Internet
+Name[fr]=Base de données de films sur Internet
 Name[fy]=Ynternetfilm-databank
 Name[ga]=Internet Movie Database
 Name[gl]=Base de datos de filmes de Internet
--- runtime/kurifilter-plugins/ikws/searchproviders/identica_groups.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/kurifilter-plugins/ikws/searchproviders/identica_groups.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -18,7 +18,7 @@
 Name[et]=Identi.ca grupid
 Name[eu]=Identi.ca Taldeak
 Name[fi]=Identi.ca Ryhmät
-Name[fr]=Groupes Icenti.ca
+Name[fr]=Groupes Identi.ca
 Name[ga]=Grúpaí Identi.ca
 Name[gu]=આઈડેન્ટિ.કા સમૂહો
 Name[he]=Identi.ca Groups
--- runtime/kurifilter-plugins/ikws/searchproviders/rae.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/kurifilter-plugins/ikws/searchproviders/rae.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -28,7 +28,7 @@
 Name[eu]=Gazteleraren Akademiaren hiztegia (RAE)
 Name[fa]=واژه‌نامه فرهنگستان اسپانیایی (RAE)
 Name[fi]=Espanjan Akatemian sanakirja (RAE)
-Name[fr]=Dictionnaire de l'Académie Espagnole (RAE)
+Name[fr]=Dictionnaire de l'Académie Espagnole (RAE)Dictionnaire de l'Académie Espagnole (RAE)
 Name[fy]=Wurdboek fan de Spaanske akademy (RAE)
 Name[ga]=Diccionario de la lengua española (RAE)
 Name[gl]=Dicionario da real academia española (RAE)
--- runtime/kurifilter-plugins/ikws/searchproviders/kde_apps.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/kurifilter-plugins/ikws/searchproviders/kde_apps.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -25,7 +25,7 @@
 Name[eu]=KDEren aplikazio bilaketa
 Name[fa]=جستجوی کاربرد KDE
 Name[fi]=KDE-sovellusten haku
-Name[fr]=Application de recherche de KDE
+Name[fr]=Chercher dans les applications de KDE
 Name[fy]=Programma's sykje
 Name[ga]=Cuardach Feidhmchláir KDE
 Name[gl]=Programas de KDE
--- runtime/kglobalaccel/kglobalaccel.notifyrc	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/kglobalaccel/kglobalaccel.notifyrc	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -492,7 +492,7 @@
 Comment[et]=Kasutaja kasutas globaalset kiirklahvi
 Comment[eu]=Erabiltzaileak laster-tekla global bat abiarazi du
 Comment[fi]=Käyttäjän liipaisema yleispikanäppäin
-Comment[fr]=L'utilisateur a déclenché un raccourcis global
+Comment[fr]=L'utilisateur a déclenché un raccourci global
 Comment[fy]=De brûker hat in globale fluchtoets aktivearre
 Comment[ga]=Ghníomhachtaigh an t-úsáideoir aicearra comhchoiteann
 Comment[gl]=O usuario iniciou un atallo global
--- runtime/attica/kcm/kcm_attica.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/attica/kcm/kcm_attica.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -86,7 +86,7 @@
 Comment[et]=Sotsiaalse töölaua pakkujate haldamine
 Comment[eu]=Kudeatu Gizarte Mahaigain Hornitzaileak
 Comment[fi]=Hallinnoi sosiaalisen työpöydän tarjoajia
-Comment[fr]=Gérer les fournisseurs de bureaux sociaux
+Comment[fr]=Gérer les fournisseurs de bureau social
 Comment[ga]=Bainistigh Soláthraithe na Deisce Shóisialta
 Comment[he]=משמש לניהול ספקי שולחן עבודה חברתי
 Comment[hr]=Upravljanje pružateljima društvene radne površine
--- runtime/l10n/kg/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/kg/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -24,7 +24,7 @@
 Name[et]=Kõrgõzstan
 Name[eu]=Kirgizistan
 Name[fa]=قرقیزستان
-Name[fi]=Kyrgyzstan
+Name[fi]=Kirgisia
 Name[fr]=Kirghizistan
 Name[fy]=Kyrgyzstan
 Name[ga]=An Chirgeastáin
--- runtime/l10n/kh/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/kh/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -24,7 +24,7 @@
 Name[et]=Kambodža
 Name[eu]=Kanbodia
 Name[fa]=کامبوج
-Name[fi]=Kambodza
+Name[fi]=Kambodža
 Name[fr]=Cambodge
 Name[fy]=Cambodja
 Name[ga]=An Chambóid
--- runtime/l10n/ky/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/ky/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -24,7 +24,7 @@
 Name[et]=Kaimanisaared
 Name[eu]=Kaiman Uharteak
 Name[fa]=جزایر کایمن
-Name[fi]=Cayman-saaret
+Name[fi]=Caymansaaret
 Name[fr]=Îles Caïman
 Name[fy]=Kaaiman Eilannen
 Name[ga]=Oileáin na gCadhman
--- runtime/l10n/ne/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/ne/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -24,7 +24,7 @@
 Name[et]=Niger
 Name[eu]=Niger
 Name[fa]=نیجر
-Name[fi]=Nigeria
+Name[fi]=Niger
 Name[fr]=Niger
 Name[fy]=Niger
 Name[ga]=An Nígir
--- runtime/l10n/ni/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/ni/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -24,7 +24,7 @@
 Name[et]=Nikaraagua
 Name[eu]=Nikaragua
 Name[fa]=نیکاراگوئه
-Name[fi]=Nikaragua
+Name[fi]=Nicaragua
 Name[fr]=Nicaragua
 Name[fy]=Nicaragua
 Name[ga]=Nicearagua
--- runtime/l10n/nl/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/nl/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -43,7 +43,7 @@
 Name[it]=Paesi Bassi
 Name[ja]=オランダ
 Name[ka]=ჰოლანდია
-Name[kk]=Нидерланды
+Name[kk]=Нидерландия
 Name[km]=ហុល្លង់
 Name[kn]=ನೆದರ್ಲ್ಯಾಂಡ್ಸ್
 Name[ko]=네덜란드
--- runtime/l10n/ae/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/ae/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -25,7 +25,7 @@
 Name[eu]=Arabiar Emirerri Batuak
 Name[fa]=امارات متحده عربی
 Name[fi]=Yhdistyneet Arabiemiraatit
-Name[fr]=Émirats arabes unis
+Name[fr]=Émirats Arabes Unis
 Name[fy]=Ferienigde Arabyske Emiraten
 Name[ga]=Aontas na nÉimíríochtaí Arabacha
 Name[gl]=Emiratos Árabes Unidos
--- runtime/l10n/ag/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/ag/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -24,7 +24,7 @@
 Name[et]=Antigua ja Barbuda
 Name[eu]=Antigua eta Barbuda
 Name[fa]=آنتیگوا و باربودا
-Name[fi]=Antigua ja Barbados
+Name[fi]=Antigua ja Barbuda
 Name[fr]=Antigua-et-Barbuda
 Name[fy]=Antigua en Barbuda
 Name[ga]=Antigua agus Barbúda
--- runtime/l10n/az/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/az/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -24,7 +24,7 @@
 Name[et]=Aserbaidžaan
 Name[eu]=Azerbaijan
 Name[fa]=آذربایجان
-Name[fi]=Azerbaidzan
+Name[fi]=Azerbaidžan
 Name[fr]=Azerbaïjan
 Name[fy]=Azerbeidjan
 Name[ga]=An Asarbaiseáin
--- runtime/l10n/ba/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/ba/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -24,7 +24,7 @@
 Name[et]=Bosnia ja Hertsegovina
 Name[eu]=Bosnia-Herzegovina
 Name[fa]=بوسنی و هرزگوین
-Name[fi]=Bosnia ja Herzegovina
+Name[fi]=Bosnia ja Hertsegovina
 Name[fr]=Bosnie-Herzégovine
 Name[fy]=Bosnië en Herzegovina
 Name[ga]=An Bhoisnia-Heirseagaivéin
--- runtime/l10n/by/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/by/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -25,7 +25,7 @@
 Name[et]=Valgevene
 Name[eu]=Bielorrusia
 Name[fa]=بلوروس
-Name[fi]=Valkovenäjä
+Name[fi]=Valko-Venäjä
 Name[fr]=Biélorussie
 Name[fy]=Wyt-Rusland
 Name[ga]=An Bhílearúis
--- runtime/l10n/cc/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/cc/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -24,7 +24,7 @@
 Name[et]=Kookossaared
 Name[eu]=Cocos (Keeling) Uharteak
 Name[fa]=جزایر کوکوس )کیلینگ(
-Name[fi]=Cookinsaaret
+Name[fi]=Kookossaaret
 Name[fr]=Îles Cocos
 Name[fy]=Cocos (Keeling) Islands
 Name[ga]=Oileán na gCócónna (Keeling)
--- runtime/l10n/ck/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/ck/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -24,7 +24,7 @@
 Name[et]=Cooki saared
 Name[eu]=Cook Uharteak
 Name[fa]=جزایر کوک
-Name[fi]=Cooksaaret
+Name[fi]=Cookinsaaret
 Name[fr]=Îles Cook
 Name[fy]=Cook islands
 Name[ga]=Oileáin Chook
--- runtime/l10n/sb/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/sb/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -24,7 +24,7 @@
 Name[et]=Saalomoni saared
 Name[eu]=Solomon Uharteak
 Name[fa]=جزایر سلیمان
-Name[fi]=Salomon-saaret
+Name[fi]=Salomonsaaret
 Name[fr]=Îles Salomon
 Name[fy]=Solomon Eilannen
 Name[ga]=Oileáin Sholamón
--- runtime/l10n/sg/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/sg/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -17,7 +17,7 @@
 Name[cy]=Singap?r
 Name[da]=Singapore
 Name[de]=Singapur
-Name[el]=Σινγκαπούρη
+Name[el]=Σιγκαπούρη
 Name[en_GB]=Singapore
 Name[eo]=Singapuro
 Name[es]=Singapur
--- runtime/l10n/sn/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/sn/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -24,7 +24,7 @@
 Name[et]=Senegal
 Name[eu]=Senegal
 Name[fa]=سنگال
-Name[fi]=Senegali
+Name[fi]=Senegal
 Name[fr]=Sénégal
 Name[fy]=Senegal
 Name[ga]=An tSeineagáil
--- runtime/l10n/tc/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/tc/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -23,7 +23,7 @@
 Name[et]=Turks ja Caicos
 Name[eu]=Turk eta Caicoak Uharteak
 Name[fa]=جزایر ترکها و کایکوها
-Name[fi]=Turks- ja Caicos-saaret
+Name[fi]=Turks- ja Caicossaaret
 Name[fr]=Îles Turques-et-Caïques
 Name[fy]=Turks en Caicos Eilânen
 Name[ga]=Oileáin Turks agus Caicos
--- runtime/l10n/tj/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/tj/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -24,7 +24,7 @@
 Name[et]=Tadžikistan
 Name[eu]=Tadjikistan
 Name[fa]=تاجیکستان
-Name[fi]=Tajikistan
+Name[fi]=Tadžikistan
 Name[fr]=Tadjikistan
 Name[fy]=Tajikistan
 Name[ga]=An Táidsíceastáin
--- runtime/l10n/ee/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/ee/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -24,7 +24,7 @@
 Name[et]=Eesti
 Name[eu]=Estonia
 Name[fa]=استونی
-Name[fi]=Eesti
+Name[fi]=Viro
 Name[fr]=Estonie
 Name[fy]=Estland
 Name[ga]=An Eastóin
--- runtime/l10n/fj/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/fj/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -24,7 +24,7 @@
 Name[et]=Fidži
 Name[eu]=Fiji
 Name[fa]=فیجی
-Name[fi]=Fiji
+Name[fi]=Fidži
 Name[fr]=Fidji
 Name[fy]=Fiji
 Name[ga]=Fidsí
--- runtime/l10n/fk/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/fk/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -24,7 +24,7 @@
 Name[et]=Falklandi saared (Malviinid)
 Name[eu]=Malvinak Uharteak (Falkland Uharteak)
 Name[fa]=جزایر فالکلند )مالویناس(
-Name[fi]=Falklandin saaret (Malvinassaaret)
+Name[fi]=Falklandinsaaret (Malvinassaaret)
 Name[fr]=Îles Falkland (Malvinas)
 Name[fy]=Falkland Eilannen (Malvinas)
 Name[ga]=Oileáin Fháclainne (Malvinas)
--- runtime/l10n/vg/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/vg/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -23,7 +23,7 @@
 Name[et]=Briti Neitsisaared
 Name[eu]=Birjinak Uharteak, Britainiarrak
 Name[fa]=جزایر ویرجین انگلیس
-Name[fi]=Brittien Neitsytsaaret
+Name[fi]=Brittiläiset Neitsytsaaret
 Name[fr]=Îles Vierges britanniques
 Name[fy]=Virgin Eilannen, Britse
 Name[ga]=Oileáin Bhriotanacha na Maighdean
--- runtime/l10n/vi/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/vi/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -23,7 +23,7 @@
 Name[et]=USA Neitsisaared
 Name[eu]=Birjinak Uharteak, AEB
 Name[fa]=جزایر ویرجین آمریکا
-Name[fi]=USA:n Neitsytsaaret
+Name[fi]=Yhdysvaltain Neitsytsaaret
 Name[fr]=Îles Vierges américaines
 Name[fy]=Virgin Eilannen, U.S.
 Name[ga]=Oileáin na Maighdean S.A.M.
--- runtime/l10n/gf/entry.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/l10n/gf/entry.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -14,7 +14,7 @@
 Name[es]=Guyana Francesa
 Name[et]=Prantsuse Guajaana
 Name[eu]=Frantziar Guyana
-Name[fi]=Ranskan Guiana
+Name[fi]=Ranskan Guyana
 Name[fr]=Guyane française
 Name[fy]=Frânsk Guiana
 Name[ga]=Guáin na Fraince
--- runtime/plasma/declarativeimports/core/datamodel.h	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/plasma/declarativeimports/core/datamodel.h	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -132,6 +132,7 @@
     QMap<QString, QVector<QVariant> > m_items;
     QHash<int, QByteArray> m_roleNames;
     QHash<QString, int> m_roleIds;
+    int m_maxRoleId;
 };
 
 int DataModel::countItems() const
--- runtime/plasma/declarativeimports/core/datamodel.cpp	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/plasma/declarativeimports/core/datamodel.cpp	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -125,7 +125,8 @@
 
 DataModel::DataModel(QObject* parent)
     : QAbstractItemModel(parent),
-      m_dataSource(0)
+      m_dataSource(0),
+      m_maxRoleId(Qt::UserRole)
 {
     setObjectName("DataModel");
     connect(this, SIGNAL(rowsInserted(const QModelIndex &, int, int)),
@@ -217,21 +218,21 @@
     m_items[sourceName] = list.toVector();
 
     if (!list.isEmpty()) {
-        int role = Qt::UserRole;
-        m_roleNames.clear();
-        m_roleIds.clear();
-
         if (list.first().canConvert<QVariantHash>()) {
             foreach (const QString& roleName, list.first().value<QVariantHash>().keys()) {
-                ++role;
-                m_roleNames[role] = roleName.toLatin1();
-                m_roleIds[roleName] = role;
+                if (!m_roleIds.contains(roleName)) {
+                    ++m_maxRoleId;
+                    m_roleNames[m_maxRoleId] = roleName.toLatin1();
+                    m_roleIds[roleName] = m_maxRoleId;
+                }
             }
         } else {
             foreach (const QString& roleName, list.first().value<QVariantMap>().keys()) {
-                ++role;
-                m_roleNames[role] = roleName.toLatin1();
-                m_roleIds[roleName] = role;
+                if (!m_roleIds.contains(roleName)) {
+                    ++m_maxRoleId;
+                    m_roleNames[m_maxRoleId] = roleName.toLatin1();
+                    m_roleIds[roleName] = m_maxRoleId;
+                }
             }
         }
 
--- runtime/plasma/containments/newspaper/appletsview.cpp	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/plasma/containments/newspaper/appletsview.cpp	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -42,6 +42,7 @@
       m_dragTimeout(1000)
 {
     m_dragCountdown = new DragCountdown(this);
+    setAlignment(Qt::AlignCenter);
 
     connect(m_dragCountdown, SIGNAL(dragRequested()), this, SLOT(appletDragRequested()));
 
@@ -187,6 +188,12 @@
     }
 }
 
+void AppletsView::mouseReleaseEvent(QGraphicsSceneMouseEvent *event)
+{
+    Plasma::ScrollWidget::mouseReleaseEvent(event);
+    m_appletsContainer->setCurrentApplet(0);
+}
+
 void AppletsView::appletDragRequested()
 {
     if (!m_appletMoved) {
--- runtime/plasma/containments/newspaper/appletsview.h	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/plasma/containments/newspaper/appletsview.h	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -54,6 +54,7 @@
     void manageMouseMoveEvent(QGraphicsSceneMouseEvent *event);
     void manageMouseReleaseEvent(QGraphicsSceneMouseEvent *event);
     void showSpacer(const QPointF &pos);
+    void mouseReleaseEvent(QGraphicsSceneMouseEvent *event);
 
 protected Q_SLOTS:
     void appletDragRequested();
--- runtime/plasma/containments/newspaper/appletscontainer.cpp	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/plasma/containments/newspaper/appletscontainer.cpp	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -107,7 +107,26 @@
     const int margin = 4 + (m_mainLayout->count() - 1) * m_mainLayout->spacing();
 
     QSizeF viewportSize = m_scrollWidget->viewportGeometry().size();
+    QSizeF maxSize;
 
+    //try to figure out the column size from the applets size hints
+    if (m_orientation == Qt::Vertical && m_containment) {
+        foreach (Plasma::Applet *applet, m_containment->applets()) {
+            QSizeF appletSize = applet->effectiveSizeHint(Qt::PreferredSize);
+            if (appletSize.width() > maxSize.width()) {
+                maxSize.setWidth(appletSize.width());
+            }
+            if (appletSize.height() > maxSize.height()) {
+                maxSize.setHeight(appletSize.height());
+            }
+        }
+        const QSizeF sizeFromHints = maxSize * m_mainLayout->count();
+        //a bit of snap to avoid contents just too large
+        if (qAbs(sizeFromHints.width() - viewportSize.width()) > 128) {
+            viewportSize = sizeFromHints;
+        }
+    }
+
     for (int i = 0; i < m_mainLayout->count(); ++i) {
         QGraphicsLinearLayout *lay = dynamic_cast<QGraphicsLinearLayout *>(m_mainLayout->itemAt(i));
 
--- runtime/plasma/scriptengines/javascript/simplebindings/size.cpp	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/plasma/scriptengines/javascript/simplebindings/size.cpp	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -52,7 +52,7 @@
     DECLARE_SELF(QSizeF, height);
     if (ctx->argumentCount() > 0) {
         qreal height = ctx->argument(0).toNumber();
-        self->setWidth(height);
+        self->setHeight(height);
     }
 
     return QScriptValue(eng, self->height());
--- runtime/plasma/scriptengines/javascript/simplebindings/rect.cpp	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/plasma/scriptengines/javascript/simplebindings/rect.cpp	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -149,7 +149,7 @@
 {
     DECLARE_SELF(QRectF, moveLeft);
     qreal left = ctx->argument(0).toNumber();
-    self->moveBottom(left);
+    self->moveLeft(left);
     return QScriptValue();
 }
 
@@ -157,7 +157,7 @@
 {
     DECLARE_SELF(QRectF, moveRight);
     qreal right = ctx->argument(0).toNumber();
-    self->moveBottom(right);
+    self->moveRight(right);
     return QScriptValue();
 }
 
--- runtime/plasma/scriptengines/javascript/plasmoid/appletinterface.cpp	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/plasma/scriptengines/javascript/plasmoid/appletinterface.cpp	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -124,25 +124,21 @@
     m_appletScriptEngine->setConfigurationRequired(needsConfiguring, reason);
 }
 
-#ifdef USE_JS_SCRIPTENGINE
-
-void AppletInterface::update(const QRectF &rect)
+void JsAppletInterface::update(const QRectF &rect)
 {
     applet()->update(rect);
 }
 
-QGraphicsLayout *AppletInterface::layout() const
+QGraphicsLayout *JsAppletInterface::layout() const
 {
     return applet()->layout();
 }
 
-void AppletInterface::setLayout(QGraphicsLayout *layout)
+void JsAppletInterface::setLayout(QGraphicsLayout *layout)
 {
     applet()->setLayout(layout);
 }
 
-#endif
-
 QString AppletInterface::activeConfig() const
 {
     return m_currentConfig.isEmpty() ? "main" : m_currentConfig;
@@ -379,7 +375,7 @@
 
 
 PopupAppletInterface::PopupAppletInterface(AbstractJsAppletScript *parent)
-    : AppletInterface(parent)
+    : POPUPAPPLETSUPERCLASS(parent)
 {
 }
 
--- runtime/plasma/scriptengines/javascript/plasmoid/simplejavascriptapplet.cpp	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/plasma/scriptengines/javascript/plasmoid/simplejavascriptapplet.cpp	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -55,6 +55,8 @@
 #include <Plasma/Svg>
 #include <Plasma/VideoWidget>
 
+#define USE_JS_SCRIPTENGINE
+
 #include "appletauthorization.h"
 #include "appletinterface.h"
 #include "scriptenv.h"
@@ -511,8 +513,8 @@
 
     // Expose applet interface
     const bool isPopupApplet = qobject_cast<Plasma::PopupApplet *>(applet());
-    m_interface = isPopupApplet ? new PopupAppletInterface(this) : new AppletInterface(this);
-    m_self = m_engine->newQObject(m_interface);
+    m_interface = isPopupApplet ? new PopupAppletInterface(this) : new JsAppletInterface(this);
+    m_self = m_engine->newQObject(m_interface, QScriptEngine::QtOwnership, QScriptEngine::ExcludeDeleteLater);
     m_env->addMainObjectProperties(m_self);
     m_self.setScope(global);
     global.setProperty("plasmoid", m_self);
--- runtime/plasma/scriptengines/javascript/plasmoid/appletinterface.h	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/plasma/scriptengines/javascript/plasmoid/appletinterface.h	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -76,10 +76,6 @@
     Q_PROPERTY(int apiVersion READ apiVersion CONSTANT)
     Q_PROPERTY(QRectF rect READ rect)
     Q_PROPERTY(QSizeF size READ size)
-#ifdef USE_JS_SCRIPTENGINE
-    Q_PROPERTY(QGraphicsLayout *layout WRITE setLayout READ layout)
-    Q_PROPERTY(QObject *sender READ sender)
-#endif
 
 public:
     AppletInterface(AbstractJsAppletScript *parent);
@@ -276,12 +272,6 @@
 
     Q_INVOKABLE Plasma::Extender *extender() const;
 
-#ifdef USE_JS_SCRIPTENGINE
-    Q_INVOKABLE void update(const QRectF &rect = QRectF());
-    QGraphicsLayout *layout() const;
-    void setLayout(QGraphicsLayout *);
-#endif
-
     Plasma::DataEngine *dataEngine(const QString &name);
 
     QList<QAction*> contextualActions() const;
@@ -311,9 +301,31 @@
     QMap<QString, Plasma::ConfigLoader*> m_configs;
 };
 
-class PopupAppletInterface : public AppletInterface
+class JsAppletInterface : public AppletInterface
 {
     Q_OBJECT
+    Q_PROPERTY(QGraphicsLayout *layout WRITE setLayout READ layout)
+    Q_PROPERTY(QObject *sender READ sender)
+
+public:
+    JsAppletInterface(AbstractJsAppletScript *parent)
+        : AppletInterface(parent)
+    {
+    }
+
+    Q_INVOKABLE void update(const QRectF &rect = QRectF());
+    QGraphicsLayout *layout() const;
+    void setLayout(QGraphicsLayout *);
+};
+
+#ifdef USE_JS_SCRIPTENGINE
+#define POPUPAPPLETSUPERCLASS JsAppletInterface
+#else
+#define POPUPAPPLETSUPERCLASS AppletInterface
+#endif
+class PopupAppletInterface : public POPUPAPPLETSUPERCLASS
+{
+    Q_OBJECT
     Q_PROPERTY(QIcon popupIcon READ popupIcon WRITE setPopupIcon)
     Q_PROPERTY(bool passivePopup READ isPassivePopup WRITE setPassivePopup)
     Q_PROPERTY(QGraphicsWidget *popupWidget READ popupWidget WRITE setPopupWidget)
--- runtime/plasma/scriptengines/javascript/data/plasma-scriptengine-dataengine-javascript.desktop	(.../tags/KDE/4.6.0/kdebase)	(wersja 1218981)
+++ runtime/plasma/scriptengines/javascript/data/plasma-scriptengine-dataengine-javascript.desktop	(.../branches/KDE/4.6/kdebase)	(wersja 1218981)
@@ -15,7 +15,7 @@
 Name[et]=JavaScripti andmemootor
 Name[eu]=JavaScript datu-motorea
 Name[fi]=JavaScript-datakone
-Name[fr]=Moteur de donnée JavaScript
+Name[fr]=Moteur de données JavaScript
 Name[fy]=JavaSkript gegevens motor
 Name[ga]=Inneall Sonraí JavaScript
 Name[gl]=Motor de datos de JavaScript
